<!DOCTYPE HTML>
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-104875636-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-104875636-1');
</script>
<html>
	<style>
		.TextWrapRight{float:right; margin: 10px;}
	</style>
	<style>
		.TextWrapLeft{float:left; margin: 10px;}
	</style>
	<head>
		<title>J.L. Steenwyk</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />
		<link rel="icon" href="images/website_flavicon_network.png">
		<noscript><link rel="stylesheet" href="assets/css/noscript.css" /></noscript>
	</head>
	<body class="is-preload">
		<article id="Top">
			<div id="page-wrapper">
			<!-- Header -->
				<header id="header">
					<nav id="nav">
						<ul>
							<li><a href="index.html#content" style="color: white">About</a></li>
							<!-- <li><a href="blog/index.html#content" style="color: white">Blog</a></li> -->
							<li><a href="publications.html" style="color: white">Publications</a></li>
							<li><a href="news.html" style="color: white">News</a></li>
							<li>
								<a href="arts.html" style="color: white">Arts</a>
								<ul>
									<li><a href="sciart.html" style="color: white">SciArt Shop</a></li>
									<li><a href="arts_visual.html" style="color: white">Visual</a></li>
									<li><a href="arts_music.html" style="color: white">Music</a></li>
									<li><a href="arts_poetry.html" style="color: white">Poetry</a></li>
								</ul>
							</li>
							<li><a href="software.html" style="color: white">Software</a></li>
							<li><a href="curriculum_vitae.html" style="color: white">Curriculum Vitae</a></li>
							<!-- <li><a href="evo_goodreads/index.html" alt="index.html" style="color: white">Evo GoodReads</a>
								<ul>
									<li><a href="evo_goodreads/evogoodreads_archive.html" style="color: white">2021 archive</a></li>
								</ul>
							</li> -->
							<li><a href="resources.html" style="color: white">Resources</a></li>
							<li><a href="contact.html" style="color: white">Contact</a></li>
						</ul>
					</nav>
				</header>

			<!-- Banner -->
				<section id="banner">
					<div class="content">
						<header>
							<p>
								<h2 style="color: white">Genome evolution &<br />software for the life sciences</h2>
								<h4 style="color: white">Howard Hughes Medical Institute Awardee<br /> of the Life Sciences Research Foundation</h4>
								<a href="https://github.com/JLSteenwyk"><img alt="GitHub followers" src="https://img.shields.io/github/followers/JLSteenwyk?style=social"></a>
								<a class="twitter-follow-button" data-size="small" href="https://twitter.com/jlsteenwyk">Follow @JLSteenwyk</a><script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
								<!-- <a href="https://patreon.com/jlsteenwyk"><img src="https://img.shields.io/endpoint.svg?url=https%3A%2F%2Fshieldsio-patreon.vercel.app%2Fapi%3Fusername%3Djlsteenwyk%26type%3Dpatrons&style=flat" alt="Support me on Patreon" /></a>-->
							</p>
						</header>
						<span class="image"><img src="images/Jacob_Steenwyk.jpg" alt="" /></span>					
					</div>
					<a href="#content" class="goto-next scrolly">Next</a>
				</section>
			</article>
		</div>
		<!-- Content -->
		<section id="content">
			<div id="page-wrapper">
				<div id="main" class="wrapper style1">
					<div class="container">
						<!-- <center>
							<div class="row gtr-50 gtr-uniform">
								<div class="col-4 col-6-xsmall"><span class="image fit"><img src="images/Jacob_Steenwyk.jpg" alt="" /></span></div>
								<div class="col-4 col-6-xsmall"><h2><span class="image fit"><img src="images/tempo_of_evolution.png" alt="" /></span></div>
								<div class="col-4 col-6-xsmall"><h2><br />HHMI Gilliam fellow studying the tempo of rapid genome evolution in fungi</h2></div>
							</div>
							
							<br />
						</center> -->
							<header class="major">
								<h2>About</h2>
							</header>
								<p>
									My <a href="#ResearchInterests">research</a> encompasses a range of topics within the fields of evolutionary biology, genomics, and bioinformatics. I primarily focus on the evolution and
									function of genomes, with a particular emphasis on animals and fungi. I aim to understand the evolutionary modes and molecular mechanisms that contribute to
									fungal pathogenicity, which has significant implications for identifying, preventing, and combating fungal infections. This work extends to understanding the
									genetic and evolutionary basis of traits across different fungal species, contributing to a broader understanding of genome function and evolution.
									<br /><br />
									I am also involved in the development of computational tools to support research in the biological sciences. I have developed several
									<a href="https://jlsteenwyk.com/software.html">software</a>, such as <a href="https://jlsteenwyk.com/ClipKIT/">ClipKIT</a> and
									<a href="https://jlsteenwyk.com/PhyKIT/">PhyKIT</a>, which are designed to improve the analysis, processing, and interpretation of phylogenomic data.
									These tools help researchers to conduct more efficient and accurate analyses in evolutionary biology and related fields.
									<br /><br />
									In addition to scientific research, I am committed to promoting diversity, equity, and inclusion in the STEM fields. I actively participates in community
									engagement and science communication efforts to increase the representation of underrepresented groups in science and to foster a dialogue between scientists
									and the public. For example, I write a broadly accessible newsletter titled <a href="https://genomely.substack.com/">Genomely: where genomics and evolution converge</a>.
									Genomely is dedicated to exploring the world of genomics, with a special focus on evolutionary and comparative studies. Subscribe for free!
									<br />
									<center>
										<iframe src="https://genomely.substack.com/embed" width="480" height="150" style="border:1px solid #EEE; background:white;" frameborder="0" scrolling="no"></iframe>
									</center>

								</p>

								<hr>

								<section id="ResearchInterests">
									<h2>Research Interests</h2>
									<h3>Genome Evolution and Function</h3>
									<p>
										<img class="TextWrapRight" src="about_images/diversity_of_yeasts.jpg" alt="about_images/diversity_of_yeasts.jpg", style="width: 40%; height: 40%">
											Determining the principles of genome function and evolution is a major goal in evolutionary biology. The Saccharomycotina subphylum of yeasts
											are a remarkably diverse group of organisms. In fact, budding yeast diversity is roughly on par with the animal and plant kingdoms (see
											figure from <a href="publication_pdfs/2018_Shen_et_al_Cell.pdf">Shen <i>et al.</i> 2018, Cell</a>). As part of the
											<a href="https://y1000plus.wei.wisc.edu/">Y1000+</a> initiative, sequencing and analysis of 1,000+ budding yeast species is underway.
											To date, numerous insights into the tempo and mode of genome evolution across approximately 400 million years have been made. 
											For example, a genetic network inferred from evolutionary information captured conserved cellular structure and genome function
											(<a href="publication_pdfs/2022_Steenwyk_etal_Science_Advances.pdf">Steenwyk <i>et al.</i> 2022, Science Advances</a>).
											Discoveries have also been made among specific lineages; for example, budding yeast from the genus
											<i>Hanseniaspora</i> have lost numerous cell cycle and DNA repair genes (<a href="publication_pdfs/2019_Steenwyk_et_al_PLoS_Biology.pdf">Steenwyk
											<i>et al.</i> 2019, PLOS Biology</a>). This discovery is in conflict with current wisdom, which suggests these genes are important to
											<i>all</i> life and therefore evolutionarily 'resistant' to change. These and other studies highlight conservation and flexibility
											in dynamics of genome evolution.
										<center>
											Representative publications:<br />
											<a href="publication_pdfs/2022_Steenwyk_etal_Science_Advances.pdf">Steenwyk <i>et al.</i> (2022) Science Advances</a><br />
											<a href="publication_pdfs/2019_Steenwyk_et_al_PLoS_Biology.pdf">Steenwyk <i>et al.</i> (2019) PLOS Biology</a><br />
											<a href="publication_pdfs/2017_Steenwyk_and_Rokas_G3.pdf">Steenwyk & Rokas (2016) G3</a><br />
										</center>
									</p>
									
									<h3>Fungal Pathogenicity</h3>
									<p>
										<img class="TextWrapLeft" src="about_images/hybrids_graphical_abstract.jpg" alt="about_images/hybrids_graphical_abstract.jpg", style="width: 32%; height: 32%">
											Filamentous fungi including species from the genera <i>Aspergillus</i> and <i>Candida</i> are of medical and technologic importance.
											Some species pose threats to human or plant health while others produce mainstay pharmaceuticals like penicillin. Studies unraveling the
											evolutionary history of these genera will help uncover the molecular processes that contribute to some microbes being harmful to humans
											while others are helpful. In one study, we investigated the evolutionary history of globally distributed clinical isolates of <i>Aspergillus
											latus</i>, which led to the discovery that <i>A. latus</i> originated from allodiploid hybridization wherein the genome content of both
											parental species is maintained (see figure from <a href="publication_pdfs/2020_Steenwyk_and_Lind_et_al_Current_Biology.pdf">Steenwyk, Lind
											<i>et al.</i> 2020, Current Biology</a>). <i>Aspergillus</i> hybrids exhibited phenotypic heterogeneity and were distinct from closely
											related species including their parents and their close relatives. These results suggest that allodiploid hybridization contributes to
											the evolution of filamentous fungal pathogens. In a separate study, genomic and phenotypic analysis revealed the genomes of nonpathogenic
											<i>Aspergillus</i> fungi encoded genes known to impact virulence and have similar phenotypic profiles compared to pathogenic species
											(<a href="publication_pdfs/2020_Steenwyk_etal_Genetics.pdf">Steenwyk <i>et al.</i> 2020, Genetics</a>).
											These findings raise the question of "what makes a pathogen?"
										<center>
											Representative publications:<br />
											<a href="publication_pdfs/2023_Steenwyk_etal_PLOS_Pathogens.pdf">Steenwyk <i>et al.</i> (2023) PLOS Pathogens</a><br />
											<a href="publication_pdfs/2020_Steenwyk_and_Lind_et_al_Current_Biology.pdf">Steenwyk, Lind <i>et al.</i> (2020) Current Biology</a><br />
											<a href="publication_pdfs/2020_Steenwyk_etal_Genetics.pdf">Steenwyk <i>et al.</i> (2020) Genetics</a><br />
										</center>
									</p>


									<h3>Phylogenomics</h3>
									<p>
										<img class="TextWrapRight" src="about_images/fungal_phylogeny.jpeg" alt="", style="width: 21.5%; height: 21.5%">
											Phylogenomics aims to elucidate the evolutionary history of species, genes, and other biological entities. Several projects aim to determine
											the evolutionary history among lineages such as fungi (e.g., see figure from <a href="publication_pdfs/2021_Li_etal_Current_Biology.pdf">Li <i>et al.</i> 2021, Current Biology</a>)
											and other projects aim to develop novel methods in phylogenomics (e.g., <a href="publication_pdfs/2021_Shen_etal_Systematic_biology.pdf">Shen <i>et al.</i> 2021, Systematic Biology</a>).
											These studies help provide the necessary backbone to study the evolution of traits, genes, and other biological entities. Moreover, divergence time analysis has also
											placed the evolutionary history of some lineages in geologic time, enabling insights to when certain lineages arose (<a href="publication_pdfs/2019_Steenwyk_et_al_mBio.pdf">Steenwyk <i>et al.</i> 2019, mBio</a>)
											These and other studies have revealed many aspects of evolutionary histories, including the prominence of incongruence — that is, conflicting phylogenetic hypotheses.
											Together with a rich history of literature, we recently synthesized incongruence, methods to ameliorate it, and identified avenues for future research
											(<a href="publication_pdfs/2023_Steenwyk_etal_Nature_Reviews_Genetics.pdf">Steenwyk <i>et al.</i> 2023, Nature Reviews Genetics</a>), capturing the maturation of the field.

										<center>
											Representative publications:<br />
											<a href="publication_pdfs/2023_Steenwyk_etal_Nature_Reviews_Genetics.pdf">Steenwyk <i>et al.</i> (2023) Nature Reviews Genetics</a><br />
											<!-- <a href="publication_pdfs/2021_Shen_etal_Systematic_biology.pdf">Shen <i>et al.</i> (2021) Systematic Biology</a><br /> -->
											<a href="publication_pdfs/2021_Li_etal_Current_Biology.pdf">Li <i>et al.</i> (2021) Current Biology</a><br />
											<a href="publication_pdfs/2019_Steenwyk_et_al_mBio.pdf">Steenwyk <i>et al.</i> (2019) mBio</a><br />
										</center>
									</p>

									<h3>Development of Computational Tools</h3>
									<p>
										<img class="TextWrapLeft" src="about_images/clipkit_performance.jpg" alt="about_images/about_images/clipkit_performance.jpg", style="width: 35%; height: 35%">
											Research in the biological sciences — such as evolutionary biology, molecular biology, and others — often relies on computational tools.
											To equip bioinformaticians with the necessary computational tools, I have developed multiple software. For example,
											<a href="https://jlsteenwyk.com/ClipKIT/">ClipKIT</a> an alignment trimming algorithm that retains phylogenetically informative sites and
											removes the rest, outperformed other alignment trimming methods across a total of ~140 thousand alignments (see figure from
											<a href="publication_pdfs/2020_Steenwyk_etal_PLOS_Biology.pdf">Steenwyk <i>et al.</i> 2020, PLOS Biology</a>; <a href="https://clipkit.genomelybio.com/">browser application</a>).
											<a href="https://jlsteenwyk.com/PhyKIT/">PhyKIT</a>, a Swiss-army knife toolkit for processing and analyzing multiple sequence alignments
											and phylogenies, facilitates determining information content in phylogenomic data matrices, conduct evolution-based screens of gene function,
											and identify putative signatures of rapid radiation events among other things
											(<a href="publication_pdfs/2021_Steenwyk_etal_Bioinformatics.pdf">Steenwyk <i>et al.</i> 2021, Bioinformatics</a>). A complete list of
											software I have developed can be found on the <a href="software.html">software page</a>.<br />
										<center>
											Representative publications:<br />
											<a href="publication_pdfs/2022_Steenwyk_etal_PLoS_Biology.pdf">Steenwyk <i>et al.</i> (2022) PLOS Biology</a><br />
											<a href="publication_pdfs/2021_Steenwyk_etal_Bioinformatics.pdf">Steenwyk <i>et al.</i> (2021) Bioinformatics</a><br />
											<a href="publication_pdfs/2020_Steenwyk_etal_PLOS_Biology.pdf">Steenwyk <i>et al.</i> (2020) PLOS Biology</a><br />
										</center>
									</p>

									<p>
										<br />
										<h4>Collaboration</h4>
										These research aims have been enabled by collaborative efforts. A non-exhaustive list of collaborator/friend labs include the following: 
										<a href="https://wp0.vanderbilt.edu/rokaslab/" target="_blank">Rokas</a>, <a href="https://xingxingshen.github.io/" target="_blank">Shen</a>, 
										<a href="https://hittinger.genetics.wisc.edu/" target="_blank">Hittinger</a>, <a href="https://chem.uncg.edu/oberlies/" target="_blank">Oberlies</a>, 
										and the <a href="https://scholar.google.com/citations?user=RuvBeIMAAAAJ&hl=en" target="_blank">Goldman</a> laboratories. I am always
										interested in collaboration because what we can achieve together is far greater than what we can achieve alone -
										please feel free to <a href="contact.html">get in touch</a>.
									</p>
									<center>
										<iframe src="https://genomely.substack.com/embed" width="480" height="320" style="border:1px solid #EEE; background:white;" frameborder="0" scrolling="no"></iframe>
									</center>
									<center><a href="#Top" class="goto-pub scrolly">Top of page</a></center>
								</section>
							</section>
						</div>
					</div>
				</div>

			<!-- Footer -->
				<footer id="footer">
					<ul class="icons">
						<li><a href="https://twitter.com/jlsteenwyk" class="icon brands alt fa-twitter"><span class="label">Twitter</span></a></li>
						<li><a href="https://scholar.google.com/citations?user=VXV2j6gAAAAJ&hl=en" class="icon brands alt fa-google"><span class="label">Google</span></a></li>
						<li><a href="https://www.researchgate.net/profile/Jacob_Steenwyk" class="icon brands alt fa-researchgate"><span class="label">ResearchGate</span></a></li>
						<li><a href="https://github.com/JLSteenwyk" class="icon brands alt fa-github"><span class="label">GitHub</span></a></li>
						<li><a href="https://www.linkedin.com/in/jlsteenwyk/" class="icon brands alt fa-linkedin-in"><span class="label">LinkedIn</span></a></li>
						<li><a href="contact.html" class="icon solid alt fa-envelope"><span class="label">Email</span></a></li>
					</ul>
					<ul class="copyright">
						<li>&copy; 2023 Jacob L. Steenwyk</li>
					</ul>
				</footer>
			</article>	
		</div>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/jquery.scrolly.min.js"></script>
			<script src="assets/js/jquery.dropotron.min.js"></script>
			<script src="assets/js/jquery.scrollex.min.js"></script>
			<script src="assets/js/browser.min.js"></script>
			<script src="assets/js/breakpoints.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>

	</body>
</html>
