 <!DOCTYPE HTML>
<!--
	Miniport by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>J.L. Steenwyk</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1" />
		<!--[if lte IE 8]><script src="assets/js/ie/html5shiv.js"></script><![endif]-->
		<link rel="stylesheet" href="assets/css/main.css" />
		<link rel="icon" href="images/website_flavicon_network.png">
		<!--[if lte IE 8]><link rel="stylesheet" href="assets/css/ie8.css" /><![endif]-->
		<!--[if lte IE 9]><link rel="stylesheet" href="assets/css/ie9.css" /><![endif]-->

		<!-- Google analytics -->
		<script>
		(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
		(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
		m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
		})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');

		ga('create', 'UA-104875636-1', 'auto');
		ga('send', 'pageview');
		</script>
	</head>

	<body>
		<!-- Nav -->
		<nav id="nav">
			<ul class="container">
					<li><a href="index.html">Home</a></li>
					<li><a href="about.html">About</a></li>
					<li><a href="publications.html">Publications</a></li>
					<li><a href="news.html">News</a></li>
					<li><a href="arts.html">Arts</a></li>
					<li><a href="software.html">Software</a></li>
					<li><a href="resources.html">Resources</a></li>
					<li><a href="contact.html">Contact</a></li>
			</ul>
		</nav>

		<!-- Publications -->
		<div class="indent">
			<article id="Publications">
			<br /><br />
			<span class="image fit"><img src="images/website_logo_transbackground_extended1.png" alt="" /></span>

			<article id="Publications">
				<header>
					<center>
					<br />
					<a href="https://scholar.google.com/citations?user=VXV2j6gAAAAJ&hl=en">
					<img src="images/GoogleScholar.png" alt="" height="40" width="40"></a>
					&emsp; 
					<a href="http://orcid.org/0000-0002-8436-595X">
					<img src="images/ORCid.png" alt="" height="40" width="40"></a>
					&emsp; 
					<a href="https://www.researchgate.net/profile/Jacob_Steenwyk">
					<img src="images/ResearchGate.png" alt="" height="40" width="40"></a>
					</center>
					<p style="font-size: 20px"><strong>Popular Science Articles</strong></p>
					<p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px;font-size: 16px">
					1. <strong>Steenwyk, J.L.</strong> & A. Rokas (2019). An outlaw yeast thrives with genetic chaos – and could provide clues for understanding cancer growth, <i>The Conversation</i>, May 21, 2019. <a href="https://theconversation.com/an-outlaw-yeast-thrives-with-genetic-chaos-and-could-provide-clues-for-understanding-cancer-growth-117359">Link</a>
					</p>
					<br />
					<p style="font-size: 20px"><strong>Preprint</strong></p>
					<p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px;font-size: 16px">
					3. <strong>J.L. Steenwyk</strong>, M.E. Mead*, S.L. Knowles*, H.A. Raja, C.D. Roberts, O. Bader, J. houbraken, G.H. Goldman, N.H. Oberlies, & A. Rokas (2020). Biosynthetic gene clusters, secondary metabolite profiles, and cards of virulence in the closest nonpathogenic relatives of <i>Aspergillus fumigatus</i>. (*Equal contributors). bioRxiv. doi: 10.1101/2020.04.09.033902.
					<a href="https://www.biorxiv.org/content/10.1101/2020.04.09.033902v1">bioRxiv preprint</a>
					<div style="padding-left: 1.275em;">
						<div data-badge-type="4" data-doi="doi: 10.1101/2020.04.09.033902" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>
					</div>
					</p>
					<p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px;font-size: 16px">
					2. Mead, M.E.*, A.T. Borowsky*, B. Joehnk, <strong>J.L Steenwyk</strong>, X.-X. Shen, A. Sil, & A. Rokas (2019). Recurrent loss of <i>abaA</i>, a master regulator of asexual development in filamentous fungi, correlates with changes in genomic and morphological traits. (*Equal contributors) bioRxiv. doi: 10.1101/829465
					<a href="https://www.biorxiv.org/content/10.1101/829465v1">bioRxiv preprint</a>
					<div style="padding-left: 1.275em;">
						<div data-badge-type="4" data-doi="doi: 10.1101/829465" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>
					</div>
					</p>
					<p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px;font-size: 16px">
					1. <strong>Steenwyk, J.</strong>, J. St. Denis, J. Dresch, D. Larochelle, & R. Drewell (2017). Whole genome bisulfite sequencing reveals a sparse, but robust pattern of DNA methylation in the <i>Dictyostelium discoideum</i> genome. <a href="http://www.biorxiv.org/content/early/2017/07/20/166033">bioRxiv preprint</a>
					<div style="padding-left: 1.275em;">
						<div data-badge-type="4" data-doi="doi: 10.1101/166033" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>
					</div>
					</p>
					<br />
				</header>
			</article>
		</div>
		<div class="indent">
			<article id="Publications">
				<header>
					<p style="font-size: 20px"><strong>Publications</strong></p>
					<p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px;font-size: 16px">
					21. Santos, R.A.C.,<strong>J.L. Steenwyk</strong>, O. Rivero-Menendez, M.E. Mead, L.P. Silva, R.W. Bastos, A. Alastruey-Izquierdo, G.H. Goldman^, & A. Rokas^ (2020). Genomic and phenotypic heterogeneity of clinical isolates of the human pathogens <i>Aspergillus fumigatus</i>, <i>Aspergillus lentulus</i> and <i>Aspergillus fumigatiaffinis</i>. (^Senior contributors). Frontiers in Microbiology. <i>in press</i>.
					<a href="https://www.biorxiv.org/content/10.1101/2020.02.28.970384v1">bioRxiv preprint</a>
					<div style="padding-left: 1.275em;">
						<div data-badge-type="4" data-doi="doi: 10.1101/2020.02.28.970384" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>
					</div>
					</p>
					<p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px;font-size: 16px">
					20. Bastos, R.W., C. Valero, L.P. Silva, T. Schoen, M. Drott, V. Brauer, R. Silva-Rocha, A. Lind, <strong>J.L. Steenwyk</strong>, A. Rokas, F. Rodrigues, A. Resendiz-Sharpe, K. Lagrou, M. Marcet-Houben, T. Gabaldon, E. McDonnell, I. Reid, A. Tsang, B.R. Oakley, F. Loures, F. Almeida, A. Huttenlocher, N.P. Keller, L. Ries, & G.H. Goldman (2020). Functional characterization of clinical isolates of the opportunistic fungal pathogen <i>Aspergillus nidulans</i>. mSphere. doi:10.1128/mSphere.00153-20. 
					<a href="publication_pdfs/2020_Bastos_et_al_mSphere.pdf">PDF</a><br />
					<div style="padding-left: 1.275em;">
						<div data-badge-type="4" data-doi="doi: 10.1128/mSphere.00153-20" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>
					</div>
					</p>
					<p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px;font-size: 16px">
					19. Rokas, A., M.E. Mead, <strong>J.L. Steenwyk</strong>, N.H. Oberlies, & G.H. Goldman (2020). Evolving moldy murderers: <i>Aspergillus</i> section <i>Fumigati</i> as a model for studying the repeated evolution of fungal pathogenicity. PLoS Pathogens. doi: 10.1371/journal.ppat.1008315. 
					<a href="publication_pdfs/2020_Rokas_et_al_PLoS_Pathogens.pdf">PDF</a><br />
					<div style="padding-left: 1.275em;">
						<div data-badge-type="4" data-doi="doi: 10.1371/journal.ppat.1008315" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>
					</div>
					</p>
					<p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px;font-size: 16px">
					18. Knowles, S.L., M.E. Mead, L.P. Silva, H.A. Raja, <strong>J.L. Steenwyk</strong>, G.H. Gustavo^, A. Rokas^, & N.H. Oberlies^ (2020). Gliotoxin, a known virulence factor in the major human pathogen <i>Aspergillus fumigatus</i>, is also biosynthesized by the non-pathogenic relative <i>A. fischeri</i>. (^Senior authors) mBio. doi: 10.1128/mBio.03361-19. 
					<a href="publication_pdfs/2020_Knowles_et_al_mBio.pdf">PDF</a><br />
					<div style="padding-left: 1.275em;">
						<div data-badge-type="4" data-doi="doi: 10.1128/mBio.03361-19" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>
					</div>
					</p>
					<p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px;font-size: 16px">
					17. Libkind, D., D. Peris, F.A. Cubillos, <strong>J.L. Steenwyk</strong>, D.A. Opulente, Q.K. Langdon, N. Bellora, A. Rokas, & C.T. Hittinger (2020). Into the wild: new yeast genomes from natural environments and new tools for their analysis. FEMS Yeast Research. doi: 10.1093/femsyr/foaa008.
					<a href="publication_pdfs/2020_Libkind_et_al_FEMS_Yeast_Research.pdf">PDF</a><br />
					<div style="padding-left: 1.275em;">
						<div data-badge-type="4" data-doi="doi: 10.1093/femsyr/foaa008" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>
					</div>
					</p>
					<p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px;font-size: 16px">
					16. Rokas, A., M.E. Mead, <strong>J.L. Steenwyk</strong>, H.A. Raja, & N.H., Oberlies (2020). Biosynthetic gene clusters and the evolution of fungal chemodiversity. Natural Product Reports. doi: 10.1039/C9NP00045C. <a href="publication_pdfs/2020_Rokas_et_al_Natural_Products_Reports.pdf">PDF</a><br />
					<div style="padding-left: 1.275em;">
						<div data-badge-type="4" data-doi="doi: 10.1039/C9NP00045C" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>
					</div>
					</p>
					<p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px;font-size: 16px">
					15. Bodinaku, I., J. Shaffer, A.B. Connors, <strong>J.L. Steenwyk</strong>, E. Kastman, A. Rokas, A. Robbat, & B. Wolfe (2019). Rapid phenotypic and metabolomic domestication of wild <i>Penicillium</i> molds on cheese. mBio. doi: 10.1128/mBio.02445-19. <a href="publication_pdfs/2019_Bodinaku_etal_mBio.pdf">PDF</a><br />
					<a href="https://mbio.asm.org/content/10/5/e02445-19">Editor's Pick</a><br />
					<div style="padding-left: 1.275em;">
						<div data-badge-type="4" data-doi="doi: 10.1128/mBio.02445-19" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>
					</div>
					</p>
					<p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px;font-size: 16px">
					14. Mead, M.E.*, H.A. Raja*, <strong>J.L. Steenwyk</strong>, S.L. Knowles, N.H. Oberlies^, & A. Rokas^ (2019). Draft genome sequence of the griseofulvin-producing fungus <i>Xylaria flabelliformis</i> strain G536. (*Equal contributors; ^Senior authors) Microbiology Resource Announcements. doi: 10.1128/MRA.00890-19. <a href="publication_pdfs/2019_Mead_and_Raja_et_al_Microbiology_Resource_Announcements.pdf">PDF</a><br />
					<div style="padding-left: 1.275em;">
						<div data-badge-type="4" data-doi="doi: 10.1128/MRA.00890-19" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>
					</div>
					</p>
					<p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px;font-size: 16px">
					13. <strong>Steenwyk, J.L.</strong> & A. Rokas (2019). treehouse: a user-friendly application to obtain subtrees from large phylogenies. BMC Research Notes. doi: 10.1186/s13104-019-4577-5. <a href="publication_pdfs/2019_Steenwyk_and_Rokas_BMC_Research_Notes.pdf">PDF</a><br />
					<a href="https://github.com/JLSteenwyk/treehouse">Download treehouse</a>
					<div style="padding-left: 1.275em;">
						<div data-badge-type="4" data-doi="doi: 10.1186/s13104-019-4577-5" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>
					</div>
					</p>
					<p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px;font-size: 16px">
					12. Labella, A.L., D.A. Opulente, <strong>J.L. Steenwyk</strong>, C.T. Hittinger, & A. Rokas (2019). Variation and selection on codon usage bias across an entire subphylum. PLoS Genetics. doi: 10.1371/journal.pgen.1008304. <a href="publication_pdfs/2019_LaBella_et_al_PLoS_Genetics.pdf">PDF</a>
					<div style="padding-left: 1.275em;">
						<div data-badge-type="4" data-doi="doi: 10.1371/journal.pgen.1008304" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>
					</div>
					</p>
					<p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px;font-size: 16px">
					11. <strong>Steenwyk, J.L.</strong>, X.-X. Shen, A.L. Lind, G.H. Goldman, & A. Rokas (2019). A robust phylogenomic timetree for biotechnologically and medically important fungi in the genera <i>Aspergillus</i> and <i>Penicillium</i>. mBio. doi: 10.1128/mBio.00925-19. <a href="publication_pdfs/2019_Steenwyk_et_al_mBio.pdf">PDF</a><br />
					<div style="padding-left: 1.275em;">
						<div data-badge-type="4" data-doi="doi: 10.1128/mBio.00925-19" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>
					</div>
					</p>
					<p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px;font-size: 16px">
					10. <strong>Steenwyk, J.L.</strong>, D. Opulente, J. Kominek, X.-X. Shen, X. Zhou, A.L. LaBella, N.P. Bradley, B.F. Eichman, N. Čadež, D. Libkind, J. DeVirgilio, A.B. Hulfachor, C.P. Kurtzman, C.T. Hittinger^, & A. Rokas^ (2019). Extensive loss of cell cycle and DNA repair genes in an ancient lineage of bipolar budding yeasts. (^Senior authors) PLoS Biology. doi: 10.1371/journal.pbio.3000255. <a href="publication_pdfs/2019_Steenwyk_et_al_PLoS_Biology.pdf">PDF</a><br />
					<a href="https://twitter.com/VanderbiltU/status/1132658907346821125">@VanderbiltU highlight</a><br />
					<div style="padding-left: 1.275em;">
						<div data-badge-type="4" data-doi="doi: https://doi.org/10.1371/journal.pbio.3000255" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>
					</div>
					</p>	
					<p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px;font-size: 16px">
					9. Ries, L.N.A., <strong>J.L. Steenwyk</strong>, P.A. de Castro, P.B.A. de Lima, F. Almeida, L.J. de Assis, A.O. Manfiolli, A. Takahashi-Nakaguchi, Y. Kusuya, D. Hagiwara, H. Takahashi, X. Wang, J. Obar, A. Rokas, & G.H. Goldman (2019). Nutritional heterogeneity among <i>Aspergillus fumigatus</i> strains has consequences for virulence in a strain- and host-dependent manner. Frontiers in Microbiology. doi: 10.3389/fmicb.2019.00854. <a href="publication_pdfs/2019_Ries_et_al_Frontiers_in_Microbiology.pdf">PDF</a>
					<div style="padding-left: 1em;">
						<div data-badge-type="4" data-doi="doi: 10.3389/fmicb.2019.00854" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>
					</div>
					</p>
					<p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px;font-size: 16px">
					8. Mead M.E., S.L. Knowles, H.A. Raja, S. R. Beattie, C.H. Kowalski, <strong>J.L. Steenwyk</strong>, L.P. Silva, J. Chiaratto, L.N.A. Ries, G.G. Goldman, R.A. Cramer, N.H. Oberlies, & A. Rokas (2019). Characterizing the pathogenic, genomic, and chemical traits of <i>Aspergillus fischeri</i>, the closest sequenced relative of the major human fungal pathogen <i>Aspergillus fumigatus</i>. mSphere. doi: 10.1128/mSphere.00018-19. <a href="publication_pdfs/2019_Mead_et_al_mSphere.pdf">PDF</a>
					<div style="padding-left: 1em;">
						<div data-badge-type="4" data-doi="doi: 10.1128/mSphere.00018-19" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>
					</div>
					</p>
					<p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px;font-size: 16px">
					7. Knowles, S.L., H.A. Raja, A.J. Wright, A.M.L. Lee, L.K. Caesar, N.B. Cech, M.E. Mead, <strong>J.L. Steenwyk</strong>, L. Ries, G.H. Goldman, A. Rokas, N.H. Oberlies (2019). Mapping the fungal battlefield: using <i>in situ</i> chemistry and deletion mutants to monitor interspecific chemical interactions between fungi. Frontiers in Microbiology. doi: 10.3389/fmicb.2019.00285. <a href="publication_pdfs/2019_Knowles_et_al_Frontiers_in_Microbiology.pdf">PDF</a>
					<div style="padding-left: 1.275em;">
						<div data-badge-type="4" data-doi="doi: 10.3389/fmicb.2019.00285" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>
					</div>
					</p>
					<p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px;font-size: 16px">
					6. Eidem, H.R., <strong>J.L. Steenwyk</strong>, J. Wisecaver, J.A. Capra, P. Abbot, & A. Rokas (2018). integRATE: a desirability-based data integration framework for the prioritization of candidate genes across heterogeneous omics and its application to preterm birth. BMC Medical Genomics. doi: 10.1186/s12920-018-0426-y. <a href="publication_pdfs/2018_Eidem_etal_BMC_Medical_Genomics.pdf">PDF</a>
					<div style="padding-left: 1.275em;">
						<div data-badge-type="4" data-doi="doi: 10.1186/s12920-018-0426-y" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>
					</div>
					</p>
					<p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px;font-size: 16px">
					5. Shen, X.-X.*, D.A. Opulente*, J. Kominek*, X. Zhou*, <strong>J.L. Steenwyk</strong>, K.V. Buh, M.A.B. Haase, J.H. Wisecaver, M. Wang, D.T. Doering, J.T. Boudouris, R.M. Schneider, Q.K. Langdon, M. Ohkuma, R. Endoh, M. Takashima, R. Manabe, N. Čadež, D. Libkind, C.A. Rosa, J. DeVirgilio, A.B. Hulfachor, M. Groenewald, C.P. Kurtzman^, C.T. Hittinger^ & A. Rokas^ (2018). Tempo and mode of genome evolution in the budding yeast subphylum. Cell. doi: 10.1016/j.cell.2018.10.023. (*Equal contributors; ^Senior authors) <a href="publication_pdfs/2018_Shen_et_al_Cell.pdf">PDF</a><br />
					<a href="https://natureecoevocommunity.nature.com/users/25228-antonis-rokas/posts/40795-mitigating-the-impact-of-rogue-genes-in-phylogenomic-studies">The story behind the paper in Nature Ecology & Evolution blog</a><br />
					<a href="https://f1000.com/prime/734380463?bd=1" target="_blank"><img src="images/F1000_logo.png" alt="Access the recommendation on F1000Prime" id="bg" /></a>
					<div style="padding-left: 1.275em;">
						<div data-badge-type="4" data-doi="doi: 10.1016/j.cell.2018.10.023" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>
					</div>
					</p>
					<p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px;font-size: 16px">
					4. Segal, E.S., V. Gritsenko, A. Levitan, B. Yadav, N. Dror, <strong>J.L. Steenwyk</strong>, Y. Silberberg, K. Mielich, A. Rokas, N.A.R. Gow, R. Kunze, R. Sharan, & J. Berman (2018). Gene Essentiality Analyzed by In Vivo Transposon Mutagenesis and Machine Learning in a Stable Haploid Isolate of <i>Candida	albicans</i>. mBio. doi: 10.1128/mBio.02048-18. <a href="publication_pdfs/2018_Segal_et_al_mBio.pdf">PDF</a><br />	
					<a href="https://mbio.asm.org/content/9/5/e02048-18">Editor's Pick</a><br />
					<a href="https://f1000.com/prime/734324385"><img src="images/F1000_logo.png" alt=""></a>
					<div style="padding-left: 1.275em;">
						<div data-badge-type="4" data-doi="doi: 10.1128/mBio.02048-18" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>
					</div>
					</p>			
					<p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px;font-size: 16px">
					3. <strong>Steenwyk, J.L.</strong> & A. Rokas (2018). Copy number variation in fungi and its implications for wine yeast genetic diversity and adaptation. Frontiers in Microbiology. doi: 10.3389/fmicb.2018.00288. <a href="publication_pdfs/2018_Steenwyk_and_Rokas_Frontiers_in_Microbiology.pdf">PDF</a>
					<div style="padding-left: 1.275em;">
						<div data-badge-type="4" data-doi="doi: https://doi.org/10.3389/fmicb.2018.00288" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>
					</div>
					</p>
					<p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px;font-size: 16px">
					2. <strong>Steenwyk, J.</strong> & A. Rokas (2017). Extensive copy number variation in fermentation-related genes among <i>Saccharomyces cerevisiae</i> wine strains. G3: Genes | Genomes | Genetics. doi: 10.1534/g3.117.040105. <a href="publication_pdfs/2017_Steenwyk_and_Rokas_G3.pdf">PDF</a>
					<div style="padding-left: 1.275em;">
						<div data-badge-type="4" data-doi="10.1534/g3.117.040105" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>
					</div>
					</p>
					<p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px;font-size: 16px">
					1. <strong>Steenwyk, J.L.</strong>, J.S. Soghigian, J.R. Perfect, & J.G. Gibbons (2016). Copy number variation contributes to cryptic genetic variation in outbreak lineages of <i>Cryptococcus gattii</i> from the North American Pacific Northwest. BMC Genomics. doi: 10.1186/s12864-016-3044-0. <a href="publication_pdfs/2016_Steenwyk_et_al_BMC_Genomics.pdf">PDF</a>
					</p>
					<div style="padding-left: 1.275em;">
						<div data-badge-type='4' data-doi="10.1186/s12864-016-3044-0" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>
					</div>	
					<br />
		</div>	


		<!-- Scripts -->
		<script src="assets/js/jquery.min.js"></script>
		<script src="assets/js/jquery.scrolly.min.js"></script>
		<script src="assets/js/skel.min.js"></script>
		<script src="assets/js/skel-viewport.min.js"></script>
		<script src="assets/js/util.js"></script>
		<!--[if lte IE 8]><script src="assets/js/ie/respond.min.js"></script><![endif]-->
		<script src="assets/js/main.js"></script>
		<!-- almetric script -->
		<script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>
		<!-- plumx metric script for indiv article-->
		<script type="text/javascript" src="//d39af2mgp1pqhg.cloudfront.net/widget-popup.js"></script>
		<!-- plumx metric script for researcher -->
		<script type="text/javascript" src="//d39af2mgp1pqhg.cloudfront.net/widget-person.js"></script>

</body>
</html>