 <!DOCTYPE HTML>
<!--
	Miniport by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>J.L. Steenwyk</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1" />
		<!--[if lte IE 8]><script src="assets/js/ie/html5shiv.js"></script><![endif]-->
		<link rel="stylesheet" href="assets/css/main.css" />
		<link rel="icon" href="images/website_flavicon_network.png">
		<!--[if lte IE 8]><link rel="stylesheet" href="assets/css/ie8.css" /><![endif]-->
		<!--[if lte IE 9]><link rel="stylesheet" href="assets/css/ie9.css" /><![endif]-->

		<!-- Google analytics -->
		<script>
		(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
		(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
		m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
		})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');

		ga('create', 'UA-104875636-1', 'auto');
		ga('send', 'pageview');
		</script>
	</head>

	<body>
		<!-- Nav -->
		<nav id="nav">
			<ul class="container">
					<li><a href="index.html">Home</a></li>
					<li><a href="about.html">About</a></li>
					<li><a href="publications.html">Publications</a></li>
					<li><a href="arts.html">Arts</a></li>
					<li><a href="resources.html">Resources</a></li>
					<li><a href="contact.html">Contact</a></li>
			</ul>
		</nav>

		<!-- Publications -->
		<div class="indent">
			<article id="Publications">
			<br /><br />
			<span class="image fit"><img src="images/website_logo_transbackground_extended1.png" alt="" /></span>

			<article id="Publications">
				<header>
					<p style="font-size: 22px"><strong>Preprint</strong></p>
					<p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px;font-size: 18px">
					2. <strong>Steenwyk, J.L.</strong>, X.-X. Shen, A.L. Lind, G.G. Goldman, & A. Rokas (2018). A robust phylogenomic timetree for biotechnologically and medically important fungi in the genera <i>Aspergillus</i> and <i>Penicillium</i>. <a href="https://www.biorxiv.org/content/early/2018/10/27/370429">bioRxiv preprint</a>
					<div style="padding-left: 1.275em;">
						<div data-badge-type="4" data-doi="doi: https://doi.org/10.1101/370429" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="30"></div>
					</div>
					</p>
					<p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px;font-size: 18px">
					1. <strong>Steenwyk, J.</strong>, J. St. Denis, J. Dresch, D. Larochelle, & R. Drewell (2017). Whole genome bisulfite sequencing reveals a sparse, but robust pattern of DNA methylation in the <i>Dictyostelium discoideum</i> genome. <a href="http://www.biorxiv.org/content/early/2017/07/20/166033">bioRxiv preprint</a>
					<div style="padding-left: 1.275em;">
						<div data-badge-type="4" data-doi="doi: 10.1101/166033" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="30"></div>
					</div>
					</p>
					<br />
				</header>
			</article>
		</div>
		<div class="indent">
			<article id="Publications">
				<header>
					<p style="font-size: 22px"><strong>Publications</strong></p>
					<p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px;font-size: 18px">
					8. Mead M.E., S.L. Knowles, H.A. Raja, S. R. Beattie, C.H. Kowalski, <strong>J.L. Steenwyk</strong>, L.P. Silva, J. Chiaratto, L.N.A. Ries, G.G. Goldman, R.A. Cramer, N.H. Oberlies, & A. Rokas (2019). Characterizing the pathogenic, genomic, and chemical traits of <i>Aspergillus fischeri</i>, the closest sequenced relative of the major human fungal pathogen <i>Aspergillus fumigatus</i>. mSphere: in press. <a href="https://www.biorxiv.org/content/early/2018/11/07/430728">bioRxiv preprint</a>
					<div style="padding-left: 1em;">
						<div data-badge-type="4" data-doi="doi: 10.1101/430728" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="30"></div>
					</div>
					</p>
					<p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px;font-size: 18px">
					7. Knowles, S.L., H.A. Raja, A.J. Wright, A.M.L. Lee, L.K. Caesar, N.B. Cech, M.E. Mead, <strong>J.L. Steenwyk</strong>, L. Ries, G.H. Goldman, A. Rokas, N.H. Oberlies (2019). Mapping the fungal battlefield: using in situ chemistry and deletion mutants to monitor interspecific chemical interactions between fungi. Frontiers in Microbiology: in press.
					</p>
					<p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px;font-size: 18px">
					6. Eidem, H.R., <strong>J.L. Steenwyk</strong>, J. Wisecaver, J.A. Capra, P. Abbot, & A. Rokas (2018). integRATE: a desirability-based data integration framework for the prioritization of candidate genes across heterogeneous omics and its application to preterm birth. BMC Medical Genomics. doi: 10.1186/s12920-018-0426-y. <a href="publication_pdfs/Eidem_et_al_2018_BMC_Medical_Genomics.pdf">Publication PDF</a>
					<div style="padding-left: 1.275em;">
						<div data-badge-type="4" data-doi="doi: 10.1186/s12920-018-0426-y" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="30"></div>
					</div>
					</p>
					<p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px;font-size: 18px">
					5. Shen, X.-X.*, D.A. Opulente*, J. Kominek*, X. Zhou*, <strong>J.L. Steenwyk</strong>, K.V. Buh, M.A.B. Haase, J.H. Wisecaver, M. Wang, D.T. Doering, J.T. Boudouris, R.M. Schneider, Q.K. Langdon, M. Ohkuma, R. Endoh, M. Takashima, R. Manabe, N. Čadež, D. Libkind, C.A. Rosa, J. DeVirgilio, A.B. Hulfachor, M. Groenewald, C.P. Kurtzman, C.T. Hittinger & A. Rokas (2018). Tempo and mode of genome evolution in the budding yeast subphylum. Cell. doi: 10.1016/j.cell.2018.10.023. (*Equal contributors). <a href="publication_pdfs/Shen_et_al_2018_Cell.pdf">Publication PDF</a><br />
					<a href="https://natureecoevocommunity.nature.com/users/25228-antonis-rokas/posts/40795-mitigating-the-impact-of-rogue-genes-in-phylogenomic-studies">The story behind the paper in Nature Ecology & Evolution blog</a><br />
					<a href="https://f1000.com/prime/734380463?bd=1" target="_blank"><img src="images/F1000_logo.png" alt="Access the recommendation on F1000Prime" id="bg" /></a>
					</p>
					<div style="padding-left: 1.275em;">
						<div data-badge-type="4" data-doi="doi: 10.1016/j.cell.2018.10.023" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="30"></div>
					</div>	
					<p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px;font-size: 18px">
					4. Segal, E.S., V. Gritsenko, A. Levitan, B. Yadav, N. Dror, <strong>J.L. Steenwyk</strong>, Y. Silberberg, K. Mielich, A. Rokas, N.A.R. Gow, R. Kunze, R. Sharan, & J. Berman (2018). Gene Essentiality Analyzed by In Vivo Transposon Mutagenesis and Machine Learning in a Stable Haploid Isolate of <i>Candida	albicans</i>. mBio. doi: 10.1128/mBio.02048-18. <a href="publication_pdfs/Segal_et_al_2018_mBio.pdf">Publication PDF</a><br />	
					<a href="https://f1000.com/prime/734324385"><img src="images/F1000_logo.png" alt=""></a>
					<div style="padding-left: 1.275em;">
						<div data-badge-type="4" data-doi="doi: 10.1128/mBio.02048-18" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="30"></div>
					</div>
					</p>			
					<p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px;font-size: 18px">
					3. <strong>Steenwyk, J.L.</strong> & A. Rokas (2018). Copy number variation in fungi and its implications for wine yeast genetic diversity and adaptation. Frontiers in Microbiology. doi: 10.3389/fmicb.2018.00288. <a href="publication_pdfs/Steenwyk_and_Rokas_2018_FrontiersMicro.pdf">Publication PDF</a>
					<div style="padding-left: 1.275em;">
						<div data-badge-type="4" data-doi="doi: https://doi.org/10.3389/fmicb.2018.00288" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="30"></div>
					</div>
					</p>
					<p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px;font-size: 18px">
					2. <strong>Steenwyk, J.</strong> & A. Rokas (2017). Extensive copy number variation in fermentation-related genes among <i>Saccharomyces cerevisiae</i> wine strains. G3: Genes | Genomes | Genetics. doi: 10.1534/g3.117.040105. <a href="publication_pdfs/Steenwyk_and_Rokas_2017_G3.pdf">Publication PDF</a>
					</p>
					<div style="padding-left: 1.275em;">
						<div data-badge-type="4" data-doi="10.1534/g3.117.040105" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="30"></div>
					</div>
					<p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px;font-size: 18px">
					1. <strong>Steenwyk, J.L.</strong>, J.S. Soghigian, J.R. Perfect, & J.G. Gibbons (2016). Copy number variation contributes to cryptic genetic variation in outbreak lineages of <i>Cryptococcus gattii</i> from the North American Pacific Northwest. BMC Genomics. doi: 10.1186/s12864-016-3044-0. <a href="publication_pdfs/Steenwyk_et_al_2016_BMC_Genomics.pdf">Publication PDF</a>
					</p>
					<div style="padding-left: 1.275em;">
						<div data-badge-type='4' data-doi="10.1186/s12864-016-3044-0" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="30"></div>
					</div>	
					<br />
		</div>	


		<!-- Scripts -->
		<script src="assets/js/jquery.min.js"></script>
		<script src="assets/js/jquery.scrolly.min.js"></script>
		<script src="assets/js/skel.min.js"></script>
		<script src="assets/js/skel-viewport.min.js"></script>
		<script src="assets/js/util.js"></script>
		<!--[if lte IE 8]><script src="assets/js/ie/respond.min.js"></script><![endif]-->
		<script src="assets/js/main.js"></script>
		<!-- almetric script -->
		<script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>
		<!-- plumx metric script for indiv article-->
		<script type="text/javascript" src="//d39af2mgp1pqhg.cloudfront.net/widget-popup.js"></script>
		<!-- plumx metric script for researcher -->
		<script type="text/javascript" src="//d39af2mgp1pqhg.cloudfront.net/widget-person.js"></script>

</body>
</html>