<!DOCTYPE HTML>
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-104875636-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-104875636-1');
</script>
<html>
	<head>
		<title>J.L. Steenwyk</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />
		<link rel="icon" href="images/website_flavicon_network.png">
		<noscript><link rel="stylesheet" href="assets/css/noscript.css" /></noscript>
	</head>
	<body class="is-preload">
		<meta name="keywords" content="steenwyk, bioinformatics, science, biology, evolution,
		evolutionary biology, genomics, transcriptomics, phylogenetics, phylogenomics, microbiology,
		fungi, microbes, art, journals, nature, cell, genetics, filamentous fungi, budding yeast,
		population biology, population genomics, software development, software, biodiversity, cells,
		genetic network, gene function, functional genomics, computational biology, 
		tutorials, science education, science-art, sciart, jacob steenwyk, steenwyck, steenwick">
		<meta name="description" content="">
		<article id="Top">
			<div id="page-wrapper">
			
			<!-- Header -->
				<header id="header">
					<nav id="nav">
						<ul>
							<li><a href="index.html#content" style="color: white">About</a></li>
							<!-- <li><a href="blog/index.html#content" style="color: white">Blog</a></li> -->
							<li><a href="publications.html" style="color: white">Publications</a></li>
							<li><a href="news.html" style="color: white">News</a></li>
							<li>
								<a href="arts.html" style="color: white">Arts</a>
								<ul>
									<li><a href="sciart.html" style="color: white">SciArt Shop</a></li>
									<li><a href="arts_visual.html" style="color: white">Visual</a></li>
									<li><a href="arts_music.html" style="color: white">Music</a></li>
									<li><a href="arts_poetry.html" style="color: white">Poetry</a></li>
								</ul>
							</li>
							<li><a href="software.html" style="color: white">Software</a></li>
							<li><a href="curriculum_vitae.html" style="color: white">CV</a></li>
							<!-- <li><a href="evo_goodreads/index.html" alt="index.html" style="color: white">Evo GoodReads</a>
								<ul>
									<li><a href="evo_goodreads/evogoodreads_archive.html" style="color: white">2021 archive</a></li>
								</ul>
							</li> -->
							<li>
								<a href="genomely.html" style="color: white">Genomely
								<ul>
									<li><a href="genomely.html" style="color: white">What is Genomely?</a></li>
									<li><a href="https://genomelybio.com" target="_blank" style="color: white">Home</a></li>
									<li><a href="https://genomely.substack.com" target="_blank" style="color: white">Blog</a></li>
									<li><a href="https://clipkit.genomelybio.com/#/" target="_blank" style="color: white">ClipKIT browser</a></li>
								</ul>
							</li>
							<li><a href="resources.html" style="color: white">Resources</a></li>
							<li><a href="contact.html" style="color: white">Contact</a></li>
						</ul>
					</nav>
				</header>


			<!-- Main -->
				<div id="main" class="wrapper style1">
					<div class="container">
						<header class="major">
							<h2>Publications</h2>
						</header>
						<center>Trainees and I are in <strong>bold</strong> font.</center>
						<div class="row gtr-150">
							<div class="col-9 col-12-medium">

								<!-- Content -->
									<section id="content">
										<br />
										<section id="Preprints">
											<details>
											<summary>
												Click here for Preprint/Submitted articles 
											</summary>
											<br />
											<u><h3>Preprints/Submitted</h3></u>
											11. <strong>Steenwyk, J.L.</strong>^ & T.J. Buida III (2025).
											ClipKIT in the Browser: Fast Online Trimming of Multiple Sequence Alignments for Phylogenetics.
											<strong><i>Submitted</i></strong>.
											<!-- <a href="https://www.biorxiv.org/content/10.1101/2024.12.07.627370v1" style="color:black"><u>OrthoHMM: Improved Inference of Ortholog Groups using Hidden Markov Models.</u></a>
											<strong><i>bioRxiv</i></strong>. DOI: 10.1101/2024.12.07.627370. <br />
											<a href="https://github.com/JLSteenwyk/orthohmm"><img alt="Github logo" src="https://badges.aleen42.com/src/github.svg"></a>
											<a href="https://jlsteenwyk.com/orthohmm/"><img alt="docs" src="https://readthedocs.org/projects/ansicolortags/badge/?version=latest"></a>
											<span class="__dimensions_badge_embed__" data-doi="10.1101/2024.12.07.627370" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1101/2024.12.07.627370" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div> -->

											<br /><br />

											10. <strong>Steenwyk, J.L.</strong>, T.J. Buida III, A. Rokas, & N. King (2024).
											<a href="https://www.biorxiv.org/content/10.1101/2024.12.07.627370v1" style="color:black"><u>OrthoHMM: Improved Inference of Ortholog Groups using Hidden Markov Models.</u></a>
											<strong><i>bioRxiv</i></strong>. DOI: 10.1101/2024.12.07.627370. <br />
											<a href="https://github.com/JLSteenwyk/orthohmm"><img alt="Github logo" src="https://badges.aleen42.com/src/github.svg"></a>
											<a href="https://jlsteenwyk.com/orthohmm/"><img alt="docs" src="https://readthedocs.org/projects/ansicolortags/badge/?version=latest"></a>
											<span class="__dimensions_badge_embed__" data-doi="10.1101/2024.12.07.627370" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1101/2024.12.07.627370" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											9. <strong>Steenwyk, J.L.</strong> & T.J. Buida III (2024). 
											<a href="https://www.biorxiv.org/content/10.1101/2024.11.28.625917v1" style="color:black"><u>RCVT: a novel diagnostic to quantify compositional biases among taxa for phylogenomics.</u></a>
											<strong><i>bioRxiv</i></strong>. DOI: 10.1101/2024.11.28.625917.
											<span class="__dimensions_badge_embed__" data-doi="10.1101/2024.11.28.625917" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1101/2024.11.28.625917" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											8. Ros-Rocher, N., J. Reyes-Rivera, U. Horo, C. Combredet, Y. Foroughijabbari, B.T. Larson, M.C. Coyle, E.A.T. Houtepen, M.J.A. Vermeij, N. King, <strong>J.L. Steenwyk</strong>, & T. Brunet (2024).
											<u>Clonal-aggregative multicellularity entrained by salinity in one of the closest relatives of animals.</u>
											<strong><i>Submitted.</i></strong>

											<br /><br />

											7. Delbaje, E., L. Pontes, J. Rhodes, <strong>J.L. Steenwyk</strong>, L. Lu, T.F. dos Reis, A. Rokas, & G.H. Goldman (2024).
											<u><i>Aspergillus fumigatus</i> mitogenomes and their influence on azole-resistant and -susceptible populations.</u>
											<strong><i>Submitted.</i></strong>

											<br /><br />

											6. Feng, B.*, Y. Li*, H. Liu*, <strong>J.L. Steenwyk</strong>, K.T. David, X. Tian, B. Xu, C. Gon√ßalves, D.A. Opulente, A.L. LaBella, M.-C. Harrison, J.F. Wolters, S. Shao, Z. Chen, K.J. Fisher, M. Groenewald, C.T. Hittinger, X.-X. Shen, A. Rokas^, X. Zhou^, & Y. Li^ (2024). *Equal contributors; ^Corresponding authors.
											<a href="https://www.biorxiv.org/content/10.1101/2024.06.05.597512v1" style="color:black"><u>Unique trajectory of gene family evolution from genomic analysis of nearly all known species in an ancient yeast lineage.</u></a>
											<strong><i>bioRxiv</i></strong>. DOI: 10.1101/2024.06.05.597512.
											<span class="__dimensions_badge_embed__" data-doi="10.1101/2024.06.05.597512" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1101/2024.06.05.597512" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											5. O. Lemke*, B.M. Heineike*, S. Viknander, N. Cohen, <strong>J.L. Steenwyk</strong>, L. Spranger, F. Li, F. Agostini, C.T. Lee, S.K. Aulakh, J. Nielsen, A. Rokas, J. Berman, A. Zelezniak, T.I. Gossmann, & M. Ralser (2024). *Equal contributors.
											<a href="https://www.biorxiv.org/content/10.1101/2024.05.27.596037v1" style="color:black"><u>The Role of Metabolism in Shaping Enzyme Structures Over 400 Million Years of Evolution.</u></a>
											<strong><i>bioRxiv</i></strong>. DOI: 10.1101/2024.05.27.596037.
											<span class="__dimensions_badge_embed__" data-doi="10.1101/2024.05.27.596037" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1101/2024.05.27.596037" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											4. Fan, Y., M. Du, W. Zhang, W. Deng, E. Yang, S. Wang, L. Yan, L. Zhang, S. Kang, <strong>J.L. Steenwyk</strong>, Z. An, X. Liu, & M. Xiang (2024).
											<a href="https://www.biorxiv.org/content/10.1101/2024.03.21.586190v1.abstract" style="color:black"><u>The Genomes of Nematode-Trapping Fungi Provide Insights into the Origin and Diversification of Fungal Carnivorism.</u></a>
											<strong><i>bioRxiv</i></strong>. DOI: 10.1101/2024.03.21.586190.
											<span class="__dimensions_badge_embed__" data-doi="10.1101/2024.03.21.586190" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1101/2024.03.21.586190" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											3. Turnbull, R.*, <strong>J.L. Steenwyk</strong>*, S. Mutch, P. Scholten, V.W. Salazar, J.L. Birch, & H. Verbruggen (2023). *Equal contributors.
											<a href="https://www.researchsquare.com/article/rs-3699210/v2" style="color:black"><u>Orthoflow: phylogenomic analysis and diagnostics with one command.</u></a>
											<strong><i>Research Square</i></strong>. DOI: 10.21203/rs.3.rs-3699210/v2.
											<br />
											<a href="https://github.com/rbturnbull/orthoflow"><img alt="Github logo" src="https://badges.aleen42.com/src/github.svg"></a>
											<a href="https://rbturnbull.github.io/orthoflow/"><img alt="docs" src="https://readthedocs.org/projects/ansicolortags/badge/?version=latest"></a>
											<span class="__dimensions_badge_embed__" data-doi="10.21203/rs.3.rs-3699210/v2" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.21203/rs.3.rs-3699210/v2" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />
										
											2. Le, H.G.B.H.^<sup>,</sup>*, <strong>J.L. Steenwyk</strong>*, N. Manske, M. Smolin, A. Abdulali, A. Kamat, R. Kanchana, K. Giffin, A. Andere, K. Workman^ (2022). *Equal contributors; ^Corresponding authors.
											<a href="https://www.biorxiv.org/content/10.1101/2022.11.10.516016v1" style="color:black"><u>Latch Verified Bulk-RNA Seq toolkit: a cloud-based suite of workflows for bulk RNA-seq quality control, analysis, and functional enrichment.</u></a>
											bioRxiv. DOI: 10.1101/2022.11.10.516016.
											<span class="__dimensions_badge_embed__" data-doi="10.1101/2022.11.10.516016" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1101/2022.11.10.516016" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>
											
											<br />
											
											1. <strong>Zheng, Q.</strong>, <strong>J.L. Steenwyk</strong>^, & A. Rokas^ (2022). ^Corresponding authors.
											<a href="https://www.biorxiv.org/content/10.1101/2022.03.29.486229v1" style="color:black"><u>Lack of universal mutational biases in a fungal phylum.</u></a>
											<strong><i>bioRxiv</i></strong>. DOI: 10.1101/2022.03.29.486229.
											<span class="__dimensions_badge_embed__" data-doi="10.1101/2022.03.29.486229" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1101/2022.03.29.486229" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>
											
											<br />
											<center><a href="#Top" class="goto-pub scrolly">Top of page</a></center>
										<hr>
										</section>
										</details>
										<br />
										<section id="PeerReviewed">
											<h3><u>Peer-reviewed</u></h3>
											77. Sarratto, T., K. Visuri, J. Lehitenen, I. Ortega-Sanz, <strong>J.L. Steenwyk</strong>, & S. Sihvonen (2025).
											<a href="publication_pdfs/2025_Saratto_etal_BMC_Bioinformatics.pdf" style="color:black"><u>Solu ‚Äì a cloud platform for real-time genomic pathogen surveillance.</u></a>
											<strong><i>BMC Bioinformatics</i></strong>. DOI: 10.1186/s12859-024-06005-z.
											<span class="__dimensions_badge_embed__" data-doi="10.1186/s12859-024-06005-z" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1186/s12859-024-06005-z" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											76. Li, N., D.M. Geiser, <strong>J.L. Steenwyk</strong>, C. Tsuchida, S. Koike, S. Slinski, & F.N. Martin (2024).
											<a href="https://apsjournals.apsnet.org/doi/10.1094/PHYTO-04-24-0142-R" style="color:black"><u>A systematic approach for identifying unique genomic sequences for <i>Fusarium oxysporum</i> f. sp. <i>lactucae</i> race 1 and development of molecular diagnostic tools.</u></a>
											<strong><i>Phytopathology</i></strong>. DOI: 10.1094/PHYTO-04-24-0142-R.
											<span class="__dimensions_badge_embed__" data-doi="10.1094/PHYTO-04-24-0142-R" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1094/PHYTO-04-24-0142-R" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											75. <strong>Steenwyk, J.L.</strong>, S. Knowles, R.W. Bastos, <strong>C. Balamurugan</strong>, D. Rinker, M.E. Mead, C.D. Roberts, H.A. Raja, Y. Li, A.C. Colabardini, P.A. de Castro, T.F. dos Reis, D. Can√≥vas, R.L. Sanchez, K. Lagrou, E. Torrado, F. Rodrigues, N.H. Oberlies, X. Zhou, G.H. Goldman^, & A. Rokas^ (2024). ^Corresponding authors.
											<a href="publication_pdfs/2024_Steenwyk_etal_Nature_Communications.pdf" style="color:black"><u>Evolutionary origin and population diversity of a cryptic hybrid pathogen.</u></a>
											<strong><i>Nature Communications</i></strong>. DOI: 10.1038/s41467-024-52639-1.
											<span class="__dimensions_badge_embed__" data-doi="10.1038/s41467-024-52639-1" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1038/s41467-024-52639-1" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											74. Gon√ßalves, C., M.-C. Harrison, <strong>J.L. Steenwyk</strong>, D.A. Opulente, A.L. LaBella, J.F. Wolters, X. Zhou, X.-X. Shen, M. Groenewald, C.T. Hittinger, & A. Rokas (2024).
											<a href="publication_pdfs/2024_Goncalves_etal_PLOS_Biology.pdf" style="color:black"><u>Diverse signatures of convergent evolution in cacti-associated yeasts.</u></a>
											<strong><i>PLOS Biology</i></strong>. DOI: 10.1371/journal.pbio.3002832.
											<span class="__dimensions_badge_embed__" data-doi="10.1371/journal.pbio.3002832" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1371/journal.pbio.3002832" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											73. <strong>Bjornson, S.</strong>, H. Verbruggen, N.S. Upham^, & <strong>J.L. Steenwyk</strong>^ (2024). ^Corresponding authors.
											<a href="publication_pdfs/2024_Bjornson_etal_Molecular_Phylogenetics_and_Evolution.pdf" style="color:black"><u>Reticulate Evolution: Detection and Utility in the Phylogenomics Era.</u></a>
											<strong><i>Molecular Phylogenetics and Evolution</i></strong>. DOI: 10.1016/j.ympev.2024.108197.
											<span class="__dimensions_badge_embed__" data-doi="10.1016/j.ympev.2024.108197" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1016/j.ympev.2024.108197" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											72. <strong>Steenwyk, J.L.</strong>, G.I. Mart√≠nez-Redondo, T.J. Buida III, E. Gluck-Thaler, X.-X. Shen, T. Gabald√≥n, A. Rokas, & R. Fern√°ndez (2024).
											<a href="publication_pdfs/2024_Steenwyk_etal_CurrentProtocols.pdf" style="color:black"><u>PhyKIT: A Multitool for Phylogenomics.</a></u>
											<strong><i>Current Protocols</i></strong>. DOI: 10.1002/cpz1.70016.
											<span class="__dimensions_badge_embed__" data-doi="10.1002/cpz1.70016" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1002/cpz1.70016" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											71. Liu, H., <strong>J.L. Steenwyk</strong>, X. Zhou, D.T. Schultz, K.M. Kocot, X.-X. Shen, A. Rokas^, & Y. Li^ (2024). ^Corresponding authors.
											<a href="publication_pdfs/2024_Liu_etal_PLOS_Biology.pdf" style="color:black"><u>A taxon-rich and genome-scale phylogeny of Opisthokonta.</u></a>
											<strong><i>PLOS Biology</i></strong>. DOI: 10.1371/journal.pbio.3002794.
											<span class="__dimensions_badge_embed__" data-doi="10.1371/journal.pbio.3002794" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1371/journal.pbio.3002794" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>
											
											<br />

											70. Pinzan, C.F., C. Valero, P.A. de Castro, J. Luiz da Silva, K. Earle, H. Liu, M.A.C. Horta, O. Kniemeyer, T. Kr√ºger, A. Pschibul, D.N. C√∂mert, T. Heinekamp, A.A. Brakhage, <strong>J.L. Steenwyk</strong>, M.E. Mead, N. Hermsdorf, S.G. Filler, N.G. da Rosa-Garzon, E. Delbaje, M.J. Bromley, H. Cabral, C. Diehl, C.B. Angeli, G. Palmisano, A.S. Ibrahim, D.C. Rinker, T.J.C. Sauters, K. Steffen, A. Gumilang, A. Rokas^, S. Gago^, & T.F. dos Reis^, G.H. Goldman^ (2024). ^Corresponding authors.
											<a href="/publication_pdfs/2024_Pinzan_etal_Nature_Microbiology.pdf" style="color:black"><u><i>Aspergillus fumigatus</i> conidial surface-associated proteome reveals factors for fungal evasion and host immunity modulation.</u></a>
											<strong><i>Nature Microbiology</i></strong>. DOI: 10.1038/s41564-024-01782-y.
											<span class="__dimensions_badge_embed__" data-doi="10.1038/s41564-024-01782-y" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1038/s41564-024-01782-y" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											69. Zhang, W., Y. Fan, W. Deng, Y. Chen, S. Wang, S. Kang, <strong>J.L. Steenwyk</strong>, M. Xiang, & X. Liu (2024).
											<a href="/publication_pdfs/2024_Zhang_etal_mBio.pdf" style="color:black"><u>Characterization of Genome-wide Phylogenetic Conflict Uncovers Evolutionary Modes of Carnivorous Fungi.</u></a>
											<strong><i>mBio</i></strong>. DOI: 10.1128/mbio.02133-24.
											<span class="__dimensions_badge_embed__" data-doi="10.1128/mbio.02133-24" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1128/mbio.02133-24" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											68. <strong>Steenwyk, J.L.</strong> (2024).
											<a href="publication_pdfs/2024_Steenwyk_Crop_Health.pdf" style="color:black"><u>The discovery of a new lifespan-extending gene in insects.</u></a>
											<strong><i>Crop Health</i></strong>. DOI: 10.1007/s44297-024-00032-1.
											<span class="__dimensions_badge_embed__" data-doi="10.1007/s44297-024-00032-1" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1007/s44297-024-00032-1" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											67. <strong>Steenwyk, J.L.</strong> & N. King (2024).
											<a href="publication_pdfs/2024_Steenwyk_and_King_PLOS_Biology.pdf" style="color:black"><u>The Promise and Pitfalls of Synteny in Phylogenomics.</u></a>
											<strong><i>PLOS Biology</i></strong>. DOI: 10.1371/journal.pbio.3002632.
											<span class="__dimensions_badge_embed__" data-doi="10.1371/journal.pbio.3002632" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1371/journal.pbio.3002632" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											66. Brown, A., <strong>J.L. Steenwyk</strong>, & A. Rokas (2024).
											<a href="publication_pdfs/2024_Brown_etal_G3.pdf" style="color:black"><u>Genome-wide patterns of non-coding sequence variation in the major fungal pathogen <i>Aspergillus fumigatus</i>.</u></a>
											<strong><i>G3 Genes|Genomes|Genetics</i></strong>. DOI: 10.1093/g3journal/jkae091.
											<span class="__dimensions_badge_embed__" data-doi="10.1093/g3journal/jkae091" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1093/g3journal/jkae091" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											65. Opulente, D.A.*, A.L. LaBella*, M.-C. Harrison<sup>#</sup>, J.F. Wolters<sup>#</sup>, C. Liu, Y. Li, J. Kominek, <strong>J.L. Steenwyk</strong>, H.R. Stoneman, J. VanDenAvond, C.R. Miller, Q.K. Langdon, M. Silva, C. Gon√ßalves, E.J. Ubbelohde, Y. Li, K.V. Buh, M. Jarzyna, M.A.B. Haase, C.A. Rosa, N. Cadez, D. Libkind, J.H. DeVirgilio, A.B. Hulfachor, C.P. Kurtzman, J.P. Sampaio, P. Gon√ßalves, X. Zhou, X.-X. Shen, M. Groenewald, A. Rokas^, & C.T. Hittinger^ (2024). *Equal co-first authors; <sup>#</sup>Equal co-second authors; ^Corresponding authors.
											<a href="publication_pdfs/2024_Opulente_and_LaBella_etal_Science.pdf" style="color:black"><u>Genomic factors shape carbon and nitrogen metabolic niche breadth across Saccharomycotina yeasts.</u></a>
											<strong><i>Science</i></strong>. DOI: 10.1126/science.adj4503.
											<span class="__dimensions_badge_embed__" data-doi="10.1126/science.adj4503" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1126/science.adj4503" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											64. <strong>Balamurugan, C.</strong>, <strong>J.L. Steenwyk</strong>^, G.H. Goldman, & A. Rokas^ (2024). ^Corresponding authors.
											<a href="publication_pdfs/2024_Balamurugan_etal_G3.pdf" style="color:black"><u>The evolution of the gliotoxin biosynthetic gene cluster in <i>Penicillium</i> fungi.</u></a>
											<strong><i>G3 Genes|Genomes|Genetics</i></strong>. DOI:10.1093/g3journal/jkae063
											<span class="__dimensions_badge_embed__" data-doi="10.1093/g3journal/jkae063" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1093/g3journal/jkae063" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />
											63. <strong>Steenwyk, J.L.</strong>, <strong>C. Balamurugan</strong>, H.A. Raja, C. Gon√ßalves, N. Li, F. Martin, J. Berman, N.H. Oberlies, J.G. Gibbons, G.H. Goldman, D.M. Geiser, D.S. Hibbett, & A. Rokas (2024).
											<a href="publication_pdfs/2024_Steenwyk_etal_mSpectrum.pdf" style="color:black"><u>Phylogenomics reveals extensive misidentification of fungal strains from the genus <i>Aspergillus</i>.</u></a>
											<strong><i>mSpectrum</i></strong>. DOI: 10.1128/spectrum.03980-23
											<span class="__dimensions_badge_embed__" data-doi="10.1128/spectrum.03980-23" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1128/spectrum.03980-23" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											62. O'Meara, M.^, J. Rapala, C.B. Nichols, C. Alexandre, B. Billmyre, <strong>J.L. Steenwyk</strong>, J.A. Alspaugh, & T. O'Meara^ (2024). ^Corresponding authors.
											<a href="publication_pdfs/2024_OMeara_etal_PLOS_Genetics.pdf" style="color:black"><u>CryptoCEN: A Co-Expression Network for <i>Cryptococcus neoformans</i> reveals novel proteins involved in DNA damage repair.</u></a>
											<strong><i>PLOS Genetics</i></strong>. DOI: 10.1371/journal.pgen.1011158.
											<span class="__dimensions_badge_embed__" data-doi="10.1371/journal.pgen.1011158" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1371/journal.pgen.1011158" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											61. Haase, M.A.B.^, <strong>J.L. Steenwyk</strong>, & J.D. Boeke (2024). ^Corresponding author.
											<a href="publication_pdfs/2024_Haase_etal_Genetics.pdf" style="color:black"><u>Gene loss and cis-regulatory novelty shaped core histone gene evolution in the apiculate yeast <i>Hanseniaspora uvarum</i>.</u></a>
											<strong><i>Genetics</i></strong>. DOI: 10.1093/genetics/iyae008
											<span class="__dimensions_badge_embed__" data-doi="10.1093/genetics/iyae008" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1093/genetics/iyae008" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											60. Wang, J.-T.J., <strong>J.L. Steenwyk</strong>, & R. Brem (2024).
											<a href="publication_pdfs/2024_Wang_etal_FEMS_Yeast_Research.pdf" style="color:black"><u>Natural trait variation across <i>Saccharomycotina</i> species.</u></a>
											<strong><i>FEMS Yeast Research</i></strong>. DOI: 10.1093/femsyr/foae002.
											<span class="__dimensions_badge_embed__" data-doi="10.1093/femsyr/foae002" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1093/femsyr/foae002" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />
											
											59. Yea, R., M. Biango-Daniels, <strong>J.L. Steenwyk</strong>, A. Rokas, N. Louwa, R. Nardellaa, & B.E. Wolfe (2024).
											<a href="publication_pdfs/2024_Yea_et_al_FGB.pdf" style="color:black"><u>Genomic, transcriptomic, and ecological diversity of <i>Penicillium</i> species in the cheese rind microbiome.</u></a>
											<strong><i>Fungal Genetics and Biology</i></strong>. DOI: 10.1016/j.fgb.2023.103862.
											<span class="__dimensions_badge_embed__" data-doi="10.1016/j.fgb.2023.103862" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1016/j.fgb.2023.103862" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											58. <strong>Steenwyk, J.L.</strong>^, A. Rokas, & G.H. Goldman (2023). ^Corresponding author.
											<a href="publication_pdfs/2023_Steenwyk_etal_PLOS_Pathogens.pdf" style="color:black"><u>Know the enemy and know yourself: addressing cryptic fungal pathogens and beyond.</u></a>
											<strong><i>PLOS Pathogens</i></strong>. DOI: 10.1371/journal.ppat.1011704.
											<span class="__dimensions_badge_embed__" data-doi="10.1371/journal.ppat.1011704" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1371/journal.ppat.1011704" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											57. Liu, C., X. Han, <strong>J.L. Steenwyk</strong>, & X.-X. Shen (2023).
											<a href="publication_pdfs/2023_Liu_etal_Crop_Health.pdf" style="color:black"><u>Temporal transcriptomics provides insight into host-pathogen interactions: a case study of <i>Didymella pinodella</i> and disease-resistant and -susceptible pea varieties.</u></a>
											<strong><i>Crop Health</i></strong>. DOI: 10.1007/s44297-023-00005-w.
											<span class="__dimensions_badge_embed__" data-doi="10.1007/s44297-023-00005-w" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1007/s44297-023-00005-w" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											56. <strong>Steenwyk, J.L.</strong>, Y. Li, X. Zhou, X.-X. Shen, & A. Rokas (2023).
											<a href="publication_pdfs/2023_Steenwyk_etal_Nature_Reviews_Genetics.pdf" style="color:black"><u>Incongruence in the phylogenomics era.</u></a>
											<strong><i>Nature Reviews Genetics</i></strong>. DOI: 10.1038/s41576-023-00620-x.
											<span class="__dimensions_badge_embed__" data-doi="10.1038/s41576-023-00620-x" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1038/s41576-023-00620-x" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											55. Drewell, R.A., T.C. Cormier, <strong>J.L. Steenwyk</strong>, J. St. Denis, J.F. Tabima, J.M. Dresch, & D.A. Larochelle (2023).
											<a href="publication_pdfs/2023_Drewell_etal_NAR_Genomics_and_Bioinformatics.pdf" style="color:black"><u>The <i>Dictyostelium discoideum</i> genome lacks significant DNA methylation and uncovers palindromic sequences as a source of false positives in bisulfite sequencing.</u></a>
											<strong><i>NAR Genomics and Bioinformatics</i></strong>. DOI: 10.1093/nargab/lqad035.
											<span class="__dimensions_badge_embed__" data-doi="10.1093/nargab/lqad035" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1093/nargab/lqad035" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											54. Mead, M.E., P.A. de Castro, <strong>J.L. Steenwyk</strong>, J. Gangeux, M. Hoenigl, J. Prattes, R. Rautemaa-Richardson, H. Guegan, C.B. Moore, C. Lass-Fl√∂rl, F. Reizine, C. Valero, N. Van Rhijn, M.J. Bromley, A. Rokas, G.H. Goldman, & S. Gago (2023).
											<a href="publication_pdfs/2023_Mead_etal_Microbiology_spectrum.pdf" style="color:black"><u>COVID-19 Associated Pulmonary Aspergillosis isolates are genomically diverse but similar to each other in their responses to infection-relevant stresses.</u></a>
											<strong><i>mSpectrum</i></strong>. DOI: 10.1128/spectrum.05128-22.
											<span class="__dimensions_badge_embed__" data-doi="10.1128/spectrum.05128-22" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1128/spectrum.05128-22" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											53. Sierra-Patev, Sean, B. Min, M. Naranjo-Ortiz, B. Looney, Z. Konkel, J.C. Slot, Y. Sakamoto, <strong>J.L. Steenwyk</strong>, A. Rokas, J. Carro, S. Camarero, P. Ferreira, G. Molpeceres, F.J. Ruiz-Due√±as, A. Serrano, B. Henrissat, E. Drula, K.W. Hughes, J.L. Mata, N.K. Ishikawa, R. Vargas-Isla, S. Ushijima, C.A. Smith, S. Ahrendt, W. Andreopoulos, G. He, K. LaButti, A. Lipzen, V. Ng, R. Riley, L. Sandor, K. Barry, A.T. Mart√≠nez, Y. Xiao, J.G. Gibbons, K. Terashima, I.V. Grigoriev, & D. Hibbett (2023).
											<a href="publication_pdfs/2023_Sierra-Patev_etal_PNAS.pdf" style="color:black"><u>A Global Phylogenomic Analysis of the Shiitake Genus <i>Lentinula</i>.</u></a>
											<strong><i>PNAS</i></strong>. DOI: 10.1073/pnas.2214076120.
											<span class="__dimensions_badge_embed__" data-doi="10.1073/pnas.2214076120" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1073/pnas.2214076120" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />
											
											52. <strong>Steenwyk, J.L.</strong>^ & A. Rokas^ (2023). ^Corresponding authors.
											<a href="publication_pdfs/2023_Steenwyk_and_Rokas_PLOS_Biology.pdf" style="color:black"><u>The dawn of relaxed phylogenetics.</u></a>
											<strong><i>PLOS Biology</i></strong>. DOI: 10.1371/journal.pbio.3001998.
											<span class="__dimensions_badge_embed__" data-doi="10.1371/journal.pbio.3001998" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1371/journal.pbio.3001998" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											51. Li, Y.^, H. Liu, <strong>J.L. Steenwyk</strong>, A.L. LaBella, M.C. Harrison, M. Groenewald, X. Zhou, X.-X. Shen, T. Zhao, C.T. Hittinger, A. Rokas^ (2022). ^Corresponding authors.
											<a href="publication_pdfs/2022_Li_etal_Current_Biology.pdf" style="color:black"><u>Contrasting modes of macro- and micro-synteny evolution in a eukaryotic subphylum.</u></a>
											<strong><i>Current Biology</i></strong>. DOI: 10.1016/j.cub.2022.10.025.
											<span class="__dimensions_badge_embed__" data-doi="10.1016/j.cub.2022.10.025" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1016/j.cub.2022.10.025" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											50. <strong>Steenwyk, J.L.</strong>^, D.C. Goltz, T.J. Buida III, Y. Li, X.-X. Shen, & A. Rokas^ (2022). ^Corresponding authors.
											<a href="publication_pdfs/2022_Steenwyk_etal_PLoS_Biology.pdf" style="color:black"><u>OrthoSNAP: a tree splitting and pruning algorithm for retrieving single-copy orthologs from gene family trees.</u></a>
											<strong><i>PLOS Biology</i></strong>. DOI: 10.1371/journal.pbio.3001827.
											<br /><a href="https://github.com/JLSteenwyk/orthosnap"><img alt="Github logo" src="https://badges.aleen42.com/src/github.svg"></a>
											<a href="https://jlsteenwyk.com/orthosnap/"><img alt="docs" src="https://readthedocs.org/projects/ansicolortags/badge/?version=latest"></a>
											<span class="__dimensions_badge_embed__" data-doi="10.1371/journal.pbio.3001827" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1371/journal.pbio.3001827" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											49. Brown, A., M.E. Mead, <strong>J.L. Steenwyk</strong>, G.H. Goldman, & A. Rokas (2022).
											<a href="publication_pdfs/2022_Brown_etal_Frontiers_in_Fungal_Biology.pdf" style="color:black"><u>Extensive sequence divergence of non-coding regions between <i>Aspergillus fumigatus</i>, a major fungal pathogen of humans, and its relatives.</u></a>
											<strong><i>Frontiers in Fungal Biology</i></strong>. DOI: 10.3389/ffunb.2022.802494.
											<span class="__dimensions_badge_embed__" data-doi="10.3389/ffunb.2022.802494" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.3389/ffunb.2022.802494" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											48. Horta, M.A., <strong>J.L. Steenwyk</strong>, M.E. Mead, L.H.B. dos Santos, S. Zhao, J.G. Gibbons, M. Marcet-Houben, T. Gabald√≥n, A. Rokas^, & G.H. Goldman^ (2022). ^Corresponding authors.
											<a href="publication_pdfs/2022_Horta_etal_mBio.pdf" style="color:black"><u>Examination of genome-wide ortholog variation in clinical and environmental isolates of the fungal pathogen <i>Aspergillus fumigatus</i>.</u></a>
											<strong><i>mBio</i></strong>. DOI: 10.1128/mbio.01519-22.
											<span class="__dimensions_badge_embed__" data-doi="10.1128/mbio.01519-22" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1128/mbio.01519-22" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											47. <strong>Steenwyk, J.L.</strong>^, T.J. Buida III, C. Gon√ßalves, D.C. Goltz, G.H. Morales, M.E. Mead, A.L. LaBella, C.M. Chavez, J.E. Schmitz, M. Hadjifrangiskou, Y. Li, & A. Rokas^ (2022). ^Corresponding authors.
											<a href="publication_pdfs/2022_Steenwyk_etal_Genetics.pdf" style="color:black"><u>BioKIT: a versatile toolkit for processing and analyzing diverse types of sequence data.</u></a>
											<strong><i>Genetics</i></strong>. DOI: 10.1093/genetics/iyac079.
											<br /><a href="https://github.com/JLSteenwyk/BioKIT"><img alt="Github logo" src="https://badges.aleen42.com/src/github.svg"></a>
											<a href="https://jlsteenwyk.com/BioKIT/"><img alt="docs" src="https://readthedocs.org/projects/ansicolortags/badge/?version=latest"></a>
											<span class="__dimensions_badge_embed__" data-doi="10.1093/genetics/iyac079" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1093/genetics/iyac079" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											46. <strong>Steenwyk, J.L.</strong>, <strong>M.A. Phillips</strong>, F. Yang, S.S. Date, T. Graham, J. Berman,  C.T. Hittinger, & A. Rokas (2022).
											<a href="publication_pdfs/2022_Steenwyk_etal_Science_Advances.pdf" style="color:black"><u>An orthologous gene coevolution network provides insight into eukaryotic cellular and genomic structure and function.</u></a>
											<strong><i>Science Advances</i></strong>. DOI: 10.1126/sciadv.abn0105.
											<br /><a href="images/2022_Steenwyk_etal_Sci_Adv_featured_image.png" style="color:black"><u>Featured Image</u></a><br />
											<a href="https://github.com/JLSteenwyk/budding_yeast_coevolution_network"><img alt="Github logo" src="https://badges.aleen42.com/src/github.svg"></a>
											<span class="__dimensions_badge_embed__" data-doi="10.1126/sciadv.abn0105" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1126/sciadv.abn0105" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>
											
											<br />

											45. Bradley, N.P.*, K.L. Wahl*, <strong>J.L. Steenwyk</strong>, A. Rokas, & B.F. Eichman (2022). *Equal contributors.
											<a href="publication_pdfs/2022_Bradley_etal_MBio.pdf" style="color:black"><u>Resistance-guided mining of bacterial genotoxins defines a family of DNA glycosylases.</u></a>
											<strong><i>mBio</i></strong>. DOI: 10.1128/mbio.03297-21.
											<span class="__dimensions_badge_embed__" data-doi="10.1128/mbio.03297-21" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1128/mbio.03297-21" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>
											
											<br />

											44. P.A. de Castro, A. Moraes, A.C. Colabardini, M.A.C. Horta, S.L. Knowles, H.A. Raja, N.H. Oberlies, Y. Koyama, M. Ogawa, K. Gomi, <strong>J.L. Steenwyk</strong>, A. Rokas, L.N.A. Ries, & G.H. Goldman (2022).
											<a href="publication_pdfs/2022_de_Castro_etal_mBio.pdf" style="color:black"><u>Regulation of gliotoxin biosynthesis and protection in <i>Aspergillus</i> species.</u></a>
											<strong><i>PLOS Genetics</i></strong>. DOI: 10.1371/journal.pgen.1009965.
											<span class="__dimensions_badge_embed__" data-doi="10.1371/journal.pgen.1009965" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1371/journal.pgen.1009965" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>
											
											<br />

											43. <strong>J.L. Steenwyk</strong>^ & A. Rokas^ (2021). ^Corresponding authors.
											<a href="publication_pdfs/2021_Steenwyk_and_Rokas_Microbiology_Resource_Announcements.pdf" style="color:black"><u>ggpubfigs: colorblind friendly color palettes and ggplot2 graphic system extensions for publication-quality scientific figures.</u></a>
											<strong><i>Microbiology Resource Announcements</i></strong>. DOI: 10.1128/MRA.00871-21.
											<br /><a href="https://github.com/JLSteenwyk/ggpubfigs"><img alt="Github logo" src="https://badges.aleen42.com/src/github.svg"></a>
											<a href="https://github.com/JLSteenwyk/ggpubfigs"><img alt="docs" src="https://readthedocs.org/projects/ansicolortags/badge/?version=latest"></a>
											<span class="__dimensions_badge_embed__" data-doi="10.1128/MRA.00871-21" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1128/MRA.00871-21" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>
											
											<br />
											
											42. <strong>Phillips, M.A.</strong>, <strong>J.L. Steenwyk</strong>^, X.-X. Shen, & A. Rokas^ (2021). ^Corresponding authors.
											<a href="publication_pdfs/2021_Phillips_etal_Genome_biology_and_evolution.pdf" style="color:black"><u>Examination of gene loss in the DNA mismatch repair pathway and its mutational consequences in a fungal phylum.</u></a>
											<strong><i>Genome Biology and Evolution</i></strong>. DOI: 10.1093/gbe/evab219.
											<span class="__dimensions_badge_embed__" data-doi="10.1093/gbe/evab219" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1093/gbe/evab219" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>
											
											<br />

											41. Santos, R.A.C., M.E. Mead, <strong>J.L. Steenwyk</strong>, O. Rivero-Men√©ndez, A. Alastruey-Izquierdo, G.H. Goldman^, & A. Rokas^ (2021). ^Corresponding authors.
											<a href="publication_pdfs/2021_dos_Santos_et_al_Frontiers_in_Fungal_Biology.pdf" style="color:black"><u>Examining signatures of natural selection in antifungal resistance genes across <i>Aspergillus</i> fungi.</u></a>
											<strong><i>Frontiers in Fungal Biology</i></strong>. DOI: 10.3389/ffunb.2021.723051.
											<span class="__dimensions_badge_embed__" data-doi="10.3389/ffunb.2021.723051" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.3389/ffunb.2021.723051" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>
											<br />

											40. <strong>Steenwyk, J.L.</strong>^ & A. Rokas^ (2021).  ^Corresponding authors.
											<a href="publication_pdfs/2021_Steenwyk_and_Rokas_G3.pdf" style="color:black"><u>orthofisher: a broadly applicable tool for automated gene identification and retrieval.</u></a>
											<strong><i>G3 Genes|Genomes|Genetics</i></strong>. DOI: 10.1093/g3journal/jkab250.
											<br /><a href="https://github.com/JLSteenwyk/orthofisher"><img alt="Github logo" src="https://badges.aleen42.com/src/github.svg"></a>
											<a href="https://jlsteenwyk.com/orthofisher/"><img alt="docs" src="https://readthedocs.org/projects/ansicolortags/badge/?version=latest"></a>
											<span class="__dimensions_badge_embed__" data-doi="10.1093/g3journal/jkab250" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1093/g3journal/jkab250" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											39. Ries, L., P. de Castro, L. Silva, C. Valero, T. dos Reis, R. Saborano, I. Duarte, G. Persinoti, <strong>J.L. Steenwyk</strong>, A. Rokas, F. Almeida, J. Costa, T. Fill, S.S.W. Wong, V. Aimanianda, F. Rodrigues, R. Gon√ßales, C. Duarte-Oliveira, A. Carvalho, & G.H. Goldman (2021).
											<a href="publication_pdfs/2021_Ries_etal_mBio.pdf" style="color:black"><u><i>Aspergillus fumigatus</i> acetate utilization impacts virulence traits and pathogenicity.</u></a>
											<strong><i>mBio</i></strong>. DOI: 10.1128/mBio.01682-21.
											<span class="__dimensions_badge_embed__" data-doi="10.1128/mBio.01682-21" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1128/mBio.01682-21" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											38. Mead, M.E.*, <strong>J.L. Steenwyk</strong>*, L.P. Silva, P.A. de Castro, N. Saeed, F. Hillmann, G.H. Goldman, & A. Rokas (2021). *Equal contributors.
											<a href="publication_pdfs/2021_Mead_and_Steenwyk_etal_Genetics.pdf" style="color:black"><u>An evolutionary genomic approach reveals both conserved and species-specific genetic elements related to human disease in closely related <i>Aspergillus</i> fungi.</u></a>
											<strong><i>Genetics</i></strong>. DOI: 10.1093/genetics/iyab066.
											<span class="__dimensions_badge_embed__" data-doi="10.1093/genetics/iyab066" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1093/genetics/iyab066" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											37. <strong>Steenwyk, J.L.</strong>, M.E. Mead, P.A. Castro, C. Valero, A. Damasio, R.A.C. Santos, A.L. LaBella, Y. Li, S.L. Knowles, H.A. Raja, N.H. Oberlies, X. Zhou, O.A. Cornely, F. Fuchs, P. Koehler^, G.H. Goldman^, & A. Rokas^ (2021). ^Corresponding authors. 
											<a href="publication_pdfs/2021_Steenwyk_etal_Microbiology_Spectrum.pdf" style="color:black"><u>Genomic and phenotypic analysis of COVID-19-associated pulmonary aspergillosis isolates of <i>Aspergillus fumigatus</i>.</u></a>
											<strong><i>mSpectrum</i></strong>. DOI: 10.1128/Spectrum.00010-21.
											<span class="__dimensions_badge_embed__" data-doi="10.1128/Spectrum.00010-21" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1128/Spectrum.00010-21" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>
											
											<br />

											36. LaBella, A.L., D. Opulente, <strong>J.L. Steenwyk</strong>, C.T. Hittinger, & A. Rokas (2021).
											<a href="publication_pdfs/2021_LaBella_etal_PLOS_Biology.pdf" style="color:black"><u>Signatures of optimal codon usage in metabolic genes inform budding yeast ecology.</u></a>
											<strong><i>PLOS Biology</i></strong>. DOI: 10.1371/journal.pbio.3001185.
											<span class="__dimensions_badge_embed__" data-doi="10.1371/journal.pbio.3001185" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1371/journal.pbio.3001185" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											35. <strong>Steenwyk, J.L.</strong> (2021).
											<a href="publication_pdfs/2021_Steenwyk_mBio.pdf" style="color:black"><u>Evolutionary divergence in the DNA damage response among fungi.</u></a>
											<strong><i>mBio</i></strong>. DOI: 10.1128/mBio.03348-20.
											<span class="__dimensions_badge_embed__" data-doi="10.1128/mBio.03348-20" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1128/mBio.03348-20" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>
											
											<br />

											34. Shen, Xing-Xing, <strong>J.L. Steenwyk</strong>, & A. Rokas (2021).
											<a href="publication_pdfs/2021_Shen_etal_Systematic_biology.pdf" style="color:black"><u>Dissecting incongruence between concatenation- and quartet-based approaches in phylogenomic data.</u></a>
											<strong><i>Systematic Biology</i></strong>. DOI: 10.1093/sysbio/syab011.
											<span class="__dimensions_badge_embed__" data-doi="10.1093/sysbio/syab011" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1093/sysbio/syab011" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											33. <strong>Steenwyk, J.L.</strong>^, T.J. Buida III, A.L. LaBella, Y. Li, X.-X. Shen, & A. Rokas^ (2021). ^Corresponding authors.
											<a href="publication_pdfs/2021_Steenwyk_etal_Bioinformatics.pdf" style="color:black"><u>PhyKIT: a broadly applicable UNIX shell toolkit for processing and analyzing phylogenomic data.</u></a>
											<strong><i>Bioinformatics</i></strong>. DOI: 10.1093/bioinformatics/btab096.
											<br /><a href="https://github.com/JLSteenwyk/PhyKIT"><img alt="Github logo" src="https://badges.aleen42.com/src/github.svg"></a>
											<a href="https://jlsteenwyk.com/PhyKIT/"><img alt="docs" src="https://readthedocs.org/projects/ansicolortags/badge/?version=latest"></a>
											<span class="__dimensions_badge_embed__" data-doi="10.1093/bioinformatics/btab096" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1093/bioinformatics/btab096" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>
											
											<br />
											
											32. Li, Y., <strong>J.L. Steenwyk</strong>, Y. Chang, Y. Wang, T.Y. James, J.E. Stajich, J.W. Spatafora, M. Groenewald, C. Dunn, C.T. Hittinger, X.-X. Shen^, A. Rokas^ (2021). ^Corresponding authors.
											<a href="publication_pdfs/2021_Li_etal_Current_Biology.pdf" style="color:black"><u>A genome-scale phylogeny of the kingdom Fungi.</u></a>
											<strong><i>Current Biology</i></strong>. DOI: 10.1016/j.cub.2021.01.074.
											<br /><a href="https://www.cell.com/current-biology/issue?pii=S0960-9822(20)X0009-1#fullCover" style="color:black"><u>Cover Article</u></a><br />
											<span class="__dimensions_badge_embed__" data-doi="10.1016/j.cub.2021.01.074" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1016/j.cub.2021.01.074" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>
											
											<br />
											
											31. <strong>Steenwyk, J.L.</strong> (2021).
											<a href="publication_pdfs/2021_Steenwyk_Yeast.pdf" style="color:black"><u>A portrait of budding yeasts: A symbol of the arts, sciences and a whole greater than the sum of its parts.</u></a>
											<strong><i>Yeast</i></strong>. DOI: 10.1002/yea.3518.
											<br /><a href="https://onlinelibrary.wiley.com/toc/10970061/2021/38/1" style="color:black"><u>Cover Article</u></a><br />
											<span class="__dimensions_badge_embed__" data-doi="10.1002/yea.3518" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1002/yea.3518" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />
											
											30. <strong>Steenwyk, J.L.</strong>^, T.J. Buida III, Y. Li, X.-X. Shen, & A. Rokas^ (2020). ^Corresponding authors.
											<a href="publication_pdfs/2020_Steenwyk_etal_PLOS_Biology.pdf" style="color:black"><u>ClipKIT: a multiple sequence alignment-trimming software for accurate phylogenomic inference.</u></a>
											<strong><i>PLOS Biology</i></strong>. DOI: 10.1371/journal.pbio.3001007.
											<br />Run <a href="https://clipkit.genomelybio.com/" style="color:black"><u>ClipKIT in the browser</u></a> and leave the computing up to <a href="https://www.genomelybio.com/" style="color:black"><u>us</u></a>!<br />
											<a href="https://github.com/JLSteenwyk/ClipKIT"><img alt="Github logo" src="https://badges.aleen42.com/src/github.svg"></a>
											<a href="https://jlsteenwyk.com/ClipKIT/"><img alt="docs" src="https://readthedocs.org/projects/ansicolortags/badge/?version=latest"></a>
											<span class="__dimensions_badge_embed__" data-doi="10.1371/journal.pbio.3001007" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1371/journal.pbio.3001007" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>
											
											<br />
											
											29. Li, Y., K.T. David, X.-X. Shen, <strong>J.L. Steenwyk</strong>, K.M. Halanych, & A. Rokas (2020).
											<a href="publication_pdfs/2020_Li_etal_PNAS.pdf" style="color:black"><u>Feature Frequency Profile-based phylogenies are inaccurate.</u></a>
											<strong><i>PNAS</i></strong>. DOI: 10.1073/pnas.2013143117.
											<span class="__dimensions_badge_embed__" data-doi="10.1073/pnas.2013143117" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1073/pnas.2013143117" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>
											
											<br />
											
											28. Shen, X.-X.^, <strong>J.L. Steenwyk</strong>, A.L. LaBella, D.A. Opulente, X. Zhou, J. Kominek, Y. Li, M. Groenewald, C.T. Hittinger, & A. Rokas^ (2020). ^Corresponding authors.
											<a href="publication_pdfs/2020_Shen_etal_Science_Advances.pdf" style="color:black"><u>Genome-scale phylogeny and contrasting modes of genome evolution in the fungal phylum Ascomycota.</u></a>
											<strong><i>Science Advances</i></strong>. DOI: 10.1126/sciadv.abd0079.
											<span class="__dimensions_badge_embed__" data-doi="10.1126/sciadv.abd0079" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1126/sciadv.abd0079" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											27. Santos, R.A.C., O. Rivero-Menendez, <strong>J.L. Steenwyk</strong>, M.E. Mead, G.H. Goldman^, A. Alastruey-Izquierdo, & A. Rokas^ (2020). ^Corresponding authors.
											<a href="publication_pdfs/2020_dos_Santos_Microbiology_Resource_Announcements.pdf" style="color:black"><u>Draft genome sequences of four <i>Aspergillus</i> section <i>Fumigati</i> clinical strains.</u></a>
											<strong><i>Microbiology Resource Announcements</i></strong>. DOI: 10.1128/MRA.00856-20.
											<span class="__dimensions_badge_embed__" data-doi="10.1101/2020.06.28.176479" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1101/2020.06.28.176479" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											26. Filho, A.P.C., G.T.P. Brancini, P.A. de Castro, J.A. Ferreira, L.P. Silva, M.C. Rocha, I. Malavazi, J.G.M. Pontes, T. Fill, R. Silva, F. Almeida, <strong>J.L. Steenwyk</strong>, A. Rokas, T.F. dos Reis, L.N.A. Ries, & G.H. Goldman (2020).
											<a href="publication_pdfs/2020_Filho_etal_mBio.pdf" style="color:black"><u><i>Aspergillus fumigatus</i> G-protein coupled receptors GprM and GprJ are important for the regulation of the cell wall integrity pathway, secondary metabolite production, and virulence.</u></a>
											<strong><i>mBio</i></strong>. DOI: 10.1128/mBio.02458-20.
											<span class="__dimensions_badge_embed__" data-doi="10.1128/mBio.02458-20" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1128/mBio.02458-20" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											25. <strong>Steenwyk, J.L.</strong>, M.E. Mead*, S.L. Knowles*, H.A. Raja, C.D. Roberts, O. Bader, J. Houbraken, G.H. Goldman, N.H. Oberlies, & A. Rokas (2020). *Equal contributors.
											<a href="publication_pdfs/2020_Steenwyk_etal_Genetics.pdf" style="color:black"><u>Variation Among Biosynthetic Gene Clusters, Secondary Metabolite Profiles, and Cards of Virulence Across <i>Aspergillus</i> Species.</u></a>
											<strong><i>Genetics</i></strong>. DOI: 10.1534/genetics.120.303549.
											<br /><a href="https://academic.oup.com/genetics/issue/217/2" style="color:black"><u>Cover Article</u></a><br />
											<a href="https://www.genetics.org/content/216/2/481" style="color:black"><u>Highlighted Article</u></a>
											<span class="__dimensions_badge_embed__" data-doi="10.1534/genetics.120.303549" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1534/genetics.120.303549" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											24. Ries, L.N.A., L. Pardeshi, Z. Dong, K. Tan, <strong>J.L. Steenwyk</strong>, A.C. Colabardini, J.A.F. Filho, P.A. de Castro, L.P. Silva, N.W. Preite, F. Almeida, L.J. de Assis, R.A.C. dos Santos, P. Bowyer, M. Bromley, R.A. Owens, S. Doyle, M. Demasi, D.C.R. Hern√°ndez, L.E.S. Netto, M.T. Pupo, A. Rokas, F.V. Loures, K.H. Wong, & G.H. Goldman (2020).
											<a href="publication_pdfs/2020_Ries_et_al_PLoS_Pathogens.pdf" style="color:black"><u>The <i>Aspergillus fumigatus</i> transcription factor RglT is important for gliotoxin biosynthesis and self-protection, and virulence.</u></a>
											<strong><i>PLOS Pathogens</i></strong>. DOI: 10.1371/journal.ppat.1008645.
											<span class="__dimensions_badge_embed__" data-doi="10.1371/journal.ppat.1008645" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1371/journal.ppat.1008645" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											23. <strong>Steenwyk, J.L.</strong>*, A.L. Lind*, L.N.A. Ries, T.F. dos Reis, L.P. Silva, F. Almeida, R.W. Bastos, T.F. de Campos Fraga da Silva, V.L.D. Bonato, A.M. Pessoni, F. Rodrigues, H.A. Raja, S.L. Knowles, N.H. Oberlies, K. Lagrou, G.H. Goldman^, A. Rokas^ (2020). *Equal contributors; ^Corresponding authors.
											<a href="publication_pdfs/2020_Steenwyk_and_Lind_et_al_Current_Biology.pdf" style="color:black"><u>Pathogenic allodiploid hybrids of <i>Aspergillus</i> fungi.</u></a>
											<strong><i>Current Biology</i></strong>. DOI: 10.1016/j.cub.2020.04.071.
											<br /><a href="https://www.cell.com/current-biology/issue?pii=S0960-9822(19)X0014-7#fullCover" style="color:black"><u>Cover Article</u></a><br />
											<a href="images/Steenwyk_and_Lind_etal_CurrBio_NatMicroReviews_Highlights.pdf"style="color:black"><u>Nature Reviews Microbiology Research Highlights</u></a><br />
											<a href="https://github.com/JLSteenwyk/Pathogenic_allodiploid_hybrids_of_Aspergillus_fungi"><img alt="Github logo" src="https://badges.aleen42.com/src/github.svg"></a>
											<span class="__dimensions_badge_embed__" data-doi="10.1016/j.cub.2020.04.071" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1016/j.cub.2020.04.071" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											22. Mead, M.E.*, A.T. Borowsky*, B. Joehnk, <strong>J.L. Steenwyk</strong>, X.-X. Shen, A. Sil, & A. Rokas (2020). *Equal contributors.
											<a href="publication_pdfs/2020_Mead_Borowsky_et_al_Genome_Biology_and_Evolution.pdf" style="color:black"><u>Recurrent loss of <i>abaA</i>, a master regulator of asexual development in filamentous fungi, correlates with changes in genomic and morphological traits.</u></a>
											<strong><i>Genome Biology and Evolution</i></strong>. DOI: 10.1093/gbe/evaa107.
											<span class="__dimensions_badge_embed__" data-doi="10.1093/gbe/evaa107" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1093/gbe/evaa107" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											21. Santos, R.A.C., <strong>J.L. Steenwyk</strong>, O. Rivero-Menendez, M.E. Mead, L.P. Silva, R.W. Bastos, A. Alastruey-Izquierdo, G.H. Goldman^, & A. Rokas^ (2020). ^Corresponding authors.
											<a href="publication_pdfs/2020_dos_Santos_et_al_Frontiers_in_Genetics.pdf" style="color:black"><u>Genomic and phenotypic heterogeneity of clinical isolates of the human pathogens <i>Aspergillus fumigatus</i>, <i>Aspergillus lentulus</i> and <i>Aspergillus fumigatiaffinis</i>.</u></a>
											<strong><i>Frontiers in Genetics</i></strong>. DOI: 10.3389/fgene.2020.00459.
											<span class="__dimensions_badge_embed__" data-doi="10.3389/fgene.2020.00459" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.3389/fgene.2020.00459" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											20. Bastos, R.W., C. Valero, L.P. Silva, T. Schoen, M. Drott, V. Brauer, R. Silva-Rocha, A. Lind, <strong>J.L. Steenwyk</strong>, A. Rokas, F. Rodrigues, A. Resendiz-Sharpe, K. Lagrou, M. Marcet-Houben, T. Gabaldon, E. McDonnell, I. Reid, A. Tsang, B.R. Oakley, F. Loures, F. Almeida, A. Huttenlocher, N.P. Keller, L. Ries, & G.H. Goldman (2020).
											<a href="publication_pdfs/2020_Bastos_et_al_mSphere.pdf" style="color:black"><u>Functional characterization of clinical isolates of the opportunistic fungal pathogen <i>Aspergillus nidulans</i>.</u></a>
											<strong><i>mSphere</i></strong>. DOI: 10.1128/mSphere.00153-20.
											<span class="__dimensions_badge_embed__" data-doi="10.1128/mSphere.00153-20" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1128/mSphere.00153-20" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											19. Rokas, A., M.E. Mead, <strong>J.L. Steenwyk</strong>, N.H. Oberlies, & G.H. Goldman (2020).
											<a href="publication_pdfs/2020_Rokas_et_al_PLoS_Pathogens.pdf" style="color:black"><u>Evolving moldy murderers: <i>Aspergillus</i> section <i>Fumigati</i> as a model for studying the repeated evolution of fungal pathogenicity.</u></a>
											<strong><i>PLOS Pathogens</i></strong>. DOI: 10.1371/journal.ppat.1008315.
											<span class="__dimensions_badge_embed__" data-doi="10.1371/journal.ppat.1008315" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1371/journal.ppat.1008315" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											18. Knowles, S.L., M.E. Mead, L.P. Silva, H.A. Raja, <strong>J.L. Steenwyk</strong>, G.H. Gustavo^, A. Rokas^, & N.H. Oberlies^ (2020).^Corresponding authors.
											<a href="publication_pdfs/2020_Knowles_et_al_mBio.pdf" style="color:black"><u>Gliotoxin, a known virulence factor in the major human pathogen <i>Aspergillus fumigatus</i>, is also biosynthesized by the non-pathogenic relative <i>A. fischeri</i>.</u></a>
											<strong><i>mBio</i></strong>. DOI: 10.1128/mBio.03361-19.
											<span class="__dimensions_badge_embed__" data-doi="10.1128/mBio.03361-19" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1128/mBio.03361-19" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											17. Libkind, D., D. Peris, F.A. Cubillos, <strong>J.L. Steenwyk</strong>, D.A. Opulente, Q.K. Langdon, N. Bellora, A. Rokas, & C.T. Hittinger (2020).
											<a href="publication_pdfs/2020_Libkind_et_al_FEMS_Yeast_Research.pdf" style="color:black"><u>Into the wild: new yeast genomes from natural environments and new tools for their analysis.</u></a>
											<strong><i>FEMS Yeast Research</i></strong>. DOI: 10.1093/femsyr/foaa008.
											<span class="__dimensions_badge_embed__" data-doi="10.1093/femsyr/foaa008" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1093/femsyr/foaa008" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											16. Rokas, A., M.E. Mead, <strong>J.L. Steenwyk</strong>, H.A. Raja, & N.H., Oberlies (2020).
											<a href="publication_pdfs/2020_Rokas_et_al_Natural_Products_Reports.pdf" style="color:black"><u>Biosynthetic gene clusters and the evolution of fungal chemodiversity.</u></a>
											<strong><i>Natural Product Reports</i></strong>. DOI: 10.1039/c9np00045c.
											<span class="__dimensions_badge_embed__" data-doi="10.1039/C9NP00045C" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1039/C9NP00045C" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											15. Bodinaku, I., J. Shaffer, A.B. Connors, <strong>J.L. Steenwyk</strong>, E. Kastman, A. Rokas, A. Robbat, & B. Wolfe (2019).
											<a href="publication_pdfs/2019_Bodinaku_etal_mBio.pdf" style="color:black"><u>Rapid phenotypic and metabolomic domestication of wild <i>Penicillium</i> molds on cheese.</u></a>
											<strong><i>mBio</i></strong>. DOI: 10.1128/mBio.02445-19.
											<br /><a href="https://mbio.asm.org/content/10/5/e02445-19" style="color:black"><u>Editor's Pick</u></a><br />
											<span class="__dimensions_badge_embed__" data-doi="10.1128/mBio.02445-19" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1128/mBio.02445-19" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											14. Mead, M.E.*, H.A. Raja*, <strong>J.L. Steenwyk</strong>, S.L. Knowles, N.H. Oberlies^, & A. Rokas^ (2019). *Equal contributors; ^Corresponding authors.
											<a href="publication_pdfs/2019_Mead_and_Raja_et_al_Microbiology_Resource_Announcements.pdf" style="color:black"><u>Draft genome sequence of the griseofulvin-producing fungus <i>Xylaria flabelliformis</i> strain G536.</u></a>
											<strong><i>Microbiology Resource Announcements</i></strong>. DOI: 10.1128/MRA.00890-19.
											<span class="__dimensions_badge_embed__" data-doi="10.1128/MRA.00890-19" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1128/MRA.00890-19" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											13. <strong>Steenwyk, J.L.</strong> & A. Rokas (2019).
											<a href="publication_pdfs/2019_Steenwyk_and_Rokas_BMC_Research_Notes.pdf" style="color:black"><u>treehouse: a user-friendly application to obtain subtrees from large phylogenies.</u></a>
											<strong><i>BMC Research Notes</i></strong>. DOI: 10.1186/s13104-019-4577-5.
											<br /><a href="https://github.com/JLSteenwyk/treehouse"><img alt="Github logo" src="https://badges.aleen42.com/src/github.svg"></a>
											<a href="https://github.com/JLSteenwyk/treehouse"><img alt="docs" src="https://readthedocs.org/projects/ansicolortags/badge/?version=latest"></a>
											<span class="__dimensions_badge_embed__" data-doi="10.1186/s13104-019-4577-5" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1186/s13104-019-4577-5" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											12. Labella, A.L., D.A. Opulente, <strong>J.L. Steenwyk</strong>, C.T. Hittinger, & A. Rokas (2019).
											<a href="publication_pdfs/2019_LaBella_et_al_PLoS_Genetics.pdf" style="color:black"><u>Variation and selection on codon usage bias across an entire subphylum.</u></a>
											<strong><i>PLOS Genetics</i></strong>. DOI: 10.1371/journal.pgen.1008304.
											<span class="__dimensions_badge_embed__" data-doi="10.1371/journal.pgen.1008304" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1371/journal.pgen.1008304" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											11. <strong>Steenwyk, J.L.</strong>, X.-X. Shen, A.L. Lind, G.H. Goldman, & A. Rokas (2019).
											<a href="publication_pdfs/2019_Steenwyk_et_al_mBio.pdf" style="color:black"><u>A robust phylogenomic timetree for biotechnologically and medically important fungi in the genera <i>Aspergillus</i> and <i>Penicillium</i>.</u></a>
											<strong><i>mBio</i></strong>. DOI: 10.1128/mBio.00925-19.
											<span class="__dimensions_badge_embed__" data-doi="10.1128/mBio.00925-19" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1128/mBio.00925-19" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											10. <strong>Steenwyk, J.L.</strong>, D. Opulente, J. Kominek, X.-X. Shen, X. Zhou, A.L. LaBella, N.P. Bradley, B.F. Eichman, N. ƒåade≈æ, D. Libkind, J. DeVirgilio, A.B. Hulfachor, C.P. Kurtzman, C.T. Hittinger^, & A. Rokas^ (2019). ^Corresponding authors.
											<a href="publication_pdfs/2019_Steenwyk_et_al_PLoS_Biology.pdf" style="color:black"><u>Extensive loss of cell cycle and DNA repair genes in an ancient lineage of bipolar budding yeasts.</u></a>
											<strong><i>PLOS Biology</i></strong>. DOI: 10.1371/journal.pbio.3000255.
											<span class="__dimensions_badge_embed__" data-doi="10.1371/journal.pbio.3000255" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1371/journal.pbio.3000255" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											9. Ries, L.N.A., <strong>J.L. Steenwyk</strong>, P.A. de Castro, P.B.A. de Lima, F. Almeida, L.J. de Assis, A.O. Manfiolli, A. Takahashi-Nakaguchi, Y. Kusuya, D. Hagiwara, H. Takahashi, X. Wang, J. Obar, A. Rokas, & G.H. Goldman (2019).
											<a href="publication_pdfs/2019_Ries_et_al_Frontiers_in_Microbiology.pdf" style="color:black"><u>Nutritional heterogeneity among <i>Aspergillus fumigatus</i> strains has consequences for virulence in a strain- and host-dependent manner.</u></a>
											<strong><i>Frontiers in Microbiology</i></strong>. DOI: 10.3389/fmicb.2019.00854.
											<span class="__dimensions_badge_embed__" data-doi="10.3389/fmicb.2019.00854" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.3389/fmicb.2019.00854" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											8. Mead M.E., S.L. Knowles, H.A. Raja, S. R. Beattie, C.H. Kowalski, <strong>J.L. Steenwyk</strong>, L.P. Silva, J. Chiaratto, L.N.A. Ries, G.H. Goldman, R.A. Cramer, N.H. Oberlies, & A. Rokas (2019).
											<a href="publication_pdfs/2019_Mead_et_al_mSphere.pdf" style="color:black"><u>Characterizing the pathogenic, genomic, and chemical traits of <i>Aspergillus fischeri</i>, the closest sequenced relative of the major human fungal pathogen <i>Aspergillus fumigatus</i>.</u></a>
											<strong><i>mSphere</i></strong>. DOI: 10.1128/mSphere.00018-19.
											<span class="__dimensions_badge_embed__" data-doi="10.3389/fmicb.2019.00285" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.3389/fmicb.2019.00285" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											7. Knowles, S.L., H.A. Raja, A.J. Wright, A.M.L. Lee, L.K. Caesar, N.B. Cech, M.E. Mead, <strong>J.L. Steenwyk</strong>, L. Ries, G.H. Goldman, A. Rokas, & N.H. Oberlies (2019).
											<a href="publication_pdfs/2019_Knowles_et_al_Frontiers_in_Microbiology.pdf" style="color:black"><u>Mapping the fungal battlefield: using <i>in situ</i> chemistry and deletion mutants to monitor interspecific chemical interactions between fungi.</u></a>
											<strong><i>Frontiers in Microbiology</i></strong>. DOI: 10.3389/fmicb.2019.00285.
											<span class="__dimensions_badge_embed__" data-doi="10.3389/fmicb.2019.00285" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.3389/fmicb.2019.00285" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											6. Eidem, H.R., <strong>J.L. Steenwyk</strong>, J. Wisecaver, J.A. Capra, P. Abbot, & A. Rokas (2018).
											<a href="publication_pdfs/2018_Eidem_etal_BMC_Medical_Genomics.pdf" style="color:black"><u>integRATE: a desirability-based data integration framework for the prioritization of candidate genes across heterogeneous omics and its application to preterm birth.</u></a>
											<strong><i>BMC Medical Genomics</i></strong>. DOI: 10.1186/s12920-018-0426-y.
											<br /><a href="https://github.com/JLSteenwyk/integRATE"><img alt="Github logo" src="https://badges.aleen42.com/src/github.svg"></a>
											<span class="__dimensions_badge_embed__" data-doi="10.1186/s12920-018-0426-y" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1186/s12920-018-0426-y" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											5. Shen, X.-X.*, D.A. Opulente*, J. Kominek*, X. Zhou*, <strong>J.L. Steenwyk</strong>, K.V. Buh, M.A.B. Haase, J.H. Wisecaver, M. Wang, D.T. Doering, J.T. Boudouris, R.M. Schneider, Q.K. Langdon, M. Ohkuma, R. Endoh, M. Takashima, R. Manabe, N. ƒåade≈æ, D. Libkind, C.A. Rosa, J. DeVirgilio, A.B. Hulfachor, M. Groenewald, C.P. Kurtzman^, C.T. Hittinger^ & A. Rokas^ (2018). *Equal contributors; ^Corresponding authors.
											<a href="publication_pdfs/2018_Shen_et_al_Cell.pdf" style="color:black"><u>Tempo and mode of genome evolution in the budding yeast subphylum.</u></a>
											<strong><i>Cell</i></strong>. DOI: 10.1016/j.cell.2018.10.023.
											<br /><a href="https://natureecoevocommunity.nature.com/users/25228-antonis-rokas/posts/40795-mitigating-the-impact-of-rogue-genes-in-phylogenomic-studies" style="color:black"><u>The story behind the paper in Nature Ecology & Evolution blog</u></a><br />
											<span class="__dimensions_badge_embed__" data-doi="10.1016/j.cell.2018.10.023" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1016/j.cell.2018.10.023" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											4. Segal, E.S., V. Gritsenko, A. Levitan, B. Yadav, N. Dror, <strong>J.L. Steenwyk</strong>, Y. Silberberg, K. Mielich, A. Rokas, N.A.R. Gow, R. Kunze, R. Sharan, & J. Berman (2018).
											<a href="publication_pdfs/2018_Segal_et_al_mBio.pdf" style="color:black"><u>Gene Essentiality Analyzed by In Vivo Transposon Mutagenesis and Machine Learning in a Stable Haploid Isolate of <i>Candida albicans</i>.</u></a>
											<strong><i>mBio</i></strong>. DOI: 10.1128/mBio.02048-18.
											<br /><a href="https://mbio.asm.org/content/9/5/e02048-18" style="color:black"><u>Editor's Pick</u></a><br />
											<span class="__dimensions_badge_embed__" data-doi="10.1128/mBio.02048-18" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.1128/mBio.02048-18" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											3. <strong>Steenwyk, J.L.</strong> & A. Rokas (2018).
											<a href="publication_pdfs/2018_Steenwyk_and_Rokas_Frontiers_in_Microbiology.pdf" style="color:black"><u>Copy number variation in fungi and its implications for wine yeast genetic diversity and adaptation.</u></a>
											<strong><i>Frontiers in Microbiology</i></strong>. DOI: 10.3389/fmicb.2018.00288.
											<span class="__dimensions_badge_embed__" data-doi="10.3389/fmicb.2018.00288" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type="4" data-doi="doi: 10.3389/fmicb.2018.00288" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											2. <strong>Steenwyk, J.</strong> & A. Rokas (2017).
											<a href="publication_pdfs/2017_Steenwyk_and_Rokas_G3.pdf" style="color:black"><u>Extensive copy number variation in fermentation-related genes among <i>Saccharomyces cerevisiae</i> wine strains.</u></a>
											<strong><i>G3 Genes|Genomes|Genetics</i></strong>. DOI: 10.1534/g3.117.040105.
											<span class="__dimensions_badge_embed__" data-doi="10.1534/g3.117.040105" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type='4' data-doi="10.1534/g3.117.040105" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="5"></div>

											<br />

											1. <strong>Steenwyk, J.L.</strong>, J.S. Soghigian, J.R. Perfect, & J.G. Gibbons (2016).
											<a href="publication_pdfs/2016_Steenwyk_et_al_BMC_Genomics.pdf" style="color:black"><u>Copy number variation contributes to cryptic genetic variation in outbreak lineages of <i>Cryptococcus gattii</i> from the North American Pacific Northwest.</u></a>
											<strong><i>BMC Genomics</i></strong>. DOI: 10.1186/s12864-016-3044-0.
											<span class="__dimensions_badge_embed__" data-doi="10.1186/s12864-016-3044-0" data-hide-zero-citations="true" data-style="small_rectangle"></span><script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
											<div data-badge-type='4' data-doi="10.1186/s12864-016-3044-0" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />
											<center><a href="#Top" class="goto-pub scrolly">Top of page</a></center>
										</section>
									</section>

							</div>
							<div class="col-3 col-12-medium">

								<!-- Sidebar -->
									<section id="sidebar">
										<section>
											<!-- <h3><u>Genomely: a sci-blog</u></h3>
											<center>
												<iframe src="https://genomely.substack.com/embed" width="480" height="320" style="border:1px solid #EEE; background:white;" frameborder="0" scrolling="no"></iframe>
											</center> -->
											<h3><u>Journal Covers</u></h3>
											<a href="publication_pdfs/2021_Li_etal_Current_Biology.pdf" class="image fit" style="color:black"><img src="journal_covers/Li_etal_2021_Current_biology_cover.jpg" alt="" /><u>Li <i>et al.</i> 2021</u></a>
											<a href="publication_pdfs/2021_Steenwyk_Yeast.pdf" class="image fit" style="color:black"><img src="journal_covers/Steenwyk_2020_Yeast_Cover.jpg" alt="" /><u>Steenwyk 2021</u></a>
											<a href="publication_pdfs/2020_Steenwyk_etal_Genetics.pdf" class="image fit" style="color:black"><img src="journal_covers/Steenwyk_etal_2021_Genetics_Cover.png" alt="" /><u>Steenwyk <i>et al.</i> 2020</u></a>
											<a href="publication_pdfs/2020_Steenwyk_and_Lind_et_al_Current_Biology.pdf" class="image fit" style="color:black"><img src="journal_covers/Steenwyk_Lind_et_al_2020_Current_biology_Cover.jpg" alt="" /><u>Steenwyk and Lind <i>et al.</i> 2020</u></a>
										</section>
										<hr>
										<section>
											<h3><u>Popular Science</u></h3>
											6. <strong>Steenwyk, J.L.</strong> & K. Giffin.
											<a href="https://genestogenomes.org/the-silver-lining-of-bioinformatics/" style="color:black"><u>The silver lining of bioinformatics.</u></a>
											Genes to Genomes. September 12, 2022.

											<br /><br />

											5. Simopoulos, M.A.C., A.F. Cisneros, A.D. Mendoza, C. Bautista, <strong>J.L. Steenwyk</strong>, & N. Ahmad.
											<a href="https://ecrlife.org/hurdles-and-advances-to-making-science-gender-neutral/" style="color:black"><u>Hurdles and advances to making science gender-neutral.</a></u>
											ecrLife. November 26, 2020.
											
											<br /><br />

											4. Mendoza, A.D., C. Bautista, E.A. Marnik, C.M.A. Simopoulos, & <strong>J.L. Steenwyk</strong>.
											<a href="https://ecrlife.org/navigating-fake-news-as-a-scientist/" style="color:black"><u>Navigating fake news as a scientist.</a></u>
											ecrLife. October 8, 2020.

											<br /><br />

											3. <strong>Steenwyk, J.L.</strong> & M. Jonika.
											<a href="https://ecrlife.org/how-to-get-started-in-science-communication-2/" style="color:black"><u>How to get started in science communication.</u></a>
											ecrLife. August 21, 2020. 

											<br /><br />

											2. <strong>Steenwyk, J.L.</strong> & A. Rokas.
											<a href="https://theconversation.com/a-new-hybrid-fungus-is-found-in-hospitals-and-linked-to-lung-disease-138783" style="color:black"><u>A new hybrid fungus is found in hospitals and linked to lung disease.</u></a>
											The Conversation. June 4, 2020.
											
											<br /><br />

											1. <strong>Steenwyk, J.L.</strong> & A. Rokas.
											<a href="https://theconversation.com/an-outlaw-yeast-thrives-with-genetic-chaos-and-could-provide-clues-for-understanding-cancer-growth-117359" style="color:black"><u>An outlaw yeast thrives with genetic chaos ‚Äì and could provide clues for understanding cancer growth.</u></a>
											The Conversation. May 21, 2019.
											<hr>
										</section>
									</section>

							</div>
						</div>
					</div>
				</div>

			<!-- Footer -->
				<footer id="footer">
					<ul class="icons">
						<li><a href="https://twitter.com/jlsteenwyk" class="icon brands alt fa-twitter"><span class="label">Twitter</span></a></li>
						<li><a href="https://scholar.google.com/citations?user=VXV2j6gAAAAJ&hl=en" class="icon brands alt fa-google"><span class="label">Google</span></a></li>
						<li><a href="https://www.researchgate.net/profile/Jacob_Steenwyk" class="icon brands alt fa-researchgate"><span class="label">ResearchGate</span></a></li>
						<li><a href="https://github.com/JLSteenwyk" class="icon brands alt fa-github"><span class="label">GitHub</span></a></li>
						<li><a href="https://www.linkedin.com/in/jlsteenwyk/" class="icon brands alt fa-linkedin-in"><span class="label">LinkedIn</span></a></li>
						<li><a href="contact.html" class="icon solid alt fa-envelope"><span class="label">Email</span></a></li>
					</ul>
					<ul class="copyright">
						<li>&copy; 2023 Jacob L. Steenwyk</li>
					</ul>
				</footer>

		</div>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/jquery.scrolly.min.js"></script>
			<script src="assets/js/jquery.dropotron.min.js"></script>
			<script src="assets/js/jquery.scrollex.min.js"></script>
			<script src="assets/js/browser.min.js"></script>
			<script src="assets/js/breakpoints.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>
		<!-- almetric script -->
		<script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

	</body>
</html>
