<!DOCTYPE HTML>
<!-- Google analytics -->
<script>
(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');

ga('create', 'UA-104875636-1', 'auto');
ga('send', 'pageview');
</script>
<html>
	<head>
		<title>J.L. Steenwyk</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />
		<link rel="icon" href="images/website_flavicon_network.png">
		<noscript><link rel="stylesheet" href="assets/css/noscript.css" /></noscript>
	</head>
	<body class="is-preload">
		<article id="Top">
			<div id="page-wrapper">
			
			<!-- Header -->
				<header id="header">
					<nav id="nav">
						<ul>
							<li><a href="index.html#content" style="color: white">About</a></li>
							<li><a href="publications.html" style="color: white">Publications</a></li>
							<li><a href="news.html" style="color: white">News</a></li>
							<li>
								<a href="arts.html" style="color: white">Arts</a>
								<ul>
									<li><a href="sciart.html" style="color: white">NEW! SciArt Shop</a></li>
									<li><a href="arts_visual.html" style="color: white">Visual</a></li>
									<li><a href="arts_music.html" style="color: white">Music</a></li>
									<li><a href="arts_poetry.html" style="color: white">Poetry</a></li>
								</ul>
							</li>
							<li><a href="software.html" style="color: white">Software</a></li>
							<li><a href="curriculum_vitae.html" style="color: white">Curriculum Vitae</a></li>
							<li><a href="resources.html" style="color: white">Resources</a></li>
							<li><a href="contact.html" style="color: white">Contact</a></li>
						</ul>
					</nav>
				</header>


			<!-- Main -->
				<div id="main" class="wrapper style1">
					<div class="container">
						<header class="major">
							<h2>Publications</h2>
						</header>
						<div class="row gtr-150">
							<div class="col-9 col-12-medium">

								<!-- Content -->
									<section id="content">
										<section id="Preprints">
											<h3><u>Preprints</u></h3>
											4. Li, Y., <strong>J.L. Steenwyk</strong>, Y. Chang, Y. Wang, T.Y. James, J.E. Stajich, J.W. Spatafora, M. Groenewald, C. Dunn, C.T. Hittinger, X.-X. Shen^, A. Rokas^ (2020). A genome-scale phylogeny of Fungi; insights into early evolution, radiations, and the relationship between taxonomy and phylogeny. (^Senior authors). bioRxiv. doi: 10.1101/2020.08.23.262857. <a href="https://www.biorxiv.org/content/10.1101/2020.08.23.262857v1">bioRxiv preprint</a> <div data-badge-type="4" data-doi="doi: 10.1101/2020.08.23.262857" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											3. LaBella, A.L., D. Opulente, <strong>J.L. Steenwyk</strong>, C.T. Hittinger, & A. Rokas (2020). Signatures of optimal codon usage predict metabolic ecology in budding yeasts. bioRxiv. doi: 10.1101/2020.07.22.214635. <a href="https://www.biorxiv.org/content/10.1101/2020.07.22.214635v1">bioRxiv preprint</a>
											<div data-badge-type="4" data-doi="doi: 10.1101/2020.07.22.214635" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />
											
											2. <strong>Steenwyk, J.L.</strong>^, T.J. Buida III, Y. Li, X.-X. Shen, & A. Rokas^ (2020). ClipKIT: a multiple sequence alignment-trimming algorithm for accurate phylogenomic inference. (^Senior authors). bioRxiv. doi: 10.1101/2020.06.08.140384.
											<a href="https://www.biorxiv.org/content/10.1101/2020.06.08.140384v1">bioRxiv preprint</a>
											<div data-badge-type="4" data-doi="doi: 10.1101/2020.06.08.140384" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											1. <strong>Steenwyk, J.</strong>, J. St. Denis, J. Dresch, D. Larochelle, & R. Drewell (2017). Whole genome bisulfite sequencing reveals a sparse, but robust pattern of DNA methylation in the <i>Dictyostelium discoideum</i> genome. bioRxiv. doi: 10.1101/166033. <a href="http://www.biorxiv.org/content/early/2017/07/20/166033">bioRxiv preprint</a> <div data-badge-type="4" data-doi="doi: 10.1101/166033" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />
											<center><a href="#Top" class="goto-pub scrolly">Top of page</a></center>
										</section>

										<hr>

										<section id="PeerReviewed">

											<h3><u>Peer-reviewed</u></h3>
											30. Li, Y., K.T. David, X.-X. Shen, J.L. Steenwyk, K.M. Halanych, & A. Rokas (2020). Feature Frequency Profile-based phylogenies are inaccurate. Proceedings of the National Academy of Sciences of the United States of America (PNAS): <i>in press</i>.
											<div data-badge-type="4" data-doi="doi: 10.1101/2020.06.28.176479" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											29. Shen, X.-X.^, <strong>J.L. Steenwyk</strong>, A.L. LaBella, D.A. Opulente, X. Zhou, J. Kominek, Y. Li, M. Groenewald, C.T. Hittinger, & A. Rokas (2020). Genome-scale phylogeny and contrasting modes of genome evolution in the fungal phylum Ascomycota. (^Senior authors). Science Advances: <i>in press</i>.
											<div data-badge-type="4" data-doi="doi: 10.1101/2020.05.11.088658" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											28. Santos, R.A.C., O. Rivero-Menendez, <strong>J.L. Steenwyk</strong>, M.E. Mead, G.H. Goldman^, A. Alastruey-Izquierdo, & A. Rokas^ (2020). Draft genome sequences of four <i>Aspergillus</i> section <i>Fumigati</i> clinical strains. (^Senior authors) Microbiology Resource Announcements. doi: 10.1128/MRA.00856-20. 
											<a href="publication_pdfs/2020_dos_Santos_Microbiology_Resource_Announcements.pdf"><u>PDF</u></a>
											<div data-badge-type="4" data-doi="doi: 10.1101/2020.06.28.176479" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											27. Filho, A.P.C., G.T.P. Brancini, P.A. de Castro, J.A. Ferreira, L.P. Silva, M.C. Rocha, I. Malavazi, J.G.M. Pontes, T. Fill, R. Silva, F. Almeida, <strong>J.L. Steenwyk</strong>, A. Rokas, T.F. dos Reis, L.N.A. Ries, & G.H. Goldman (2020). <i>Aspergillus fumigatus</i> G-protein coupled receptors GprM and GprJ are important for the regulation of the cell wall integrity pathway, secondary metabolite production, and virulence. mBio: <i>in press</i>.
											<div data-badge-type="4" data-doi="doi: 10.1101/2020.08.03.235119" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											26. <strong>Steenwyk, J.L.</strong> (2020). A portrait of budding yeasts: A symbol of the arts, sciences and a whole greater than the sum of its parts. Yeast. doi: 10.1002/yea.3518.
											<a href="publication_pdfs/2020_Steenwyk_Yeast.pdf"><u>PDF</u></a> <div data-badge-type="4" data-doi="doi: 10.1002/yea.3518" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											25. <strong>Steenwyk, J.L.</strong>, M.E. Mead*, S.L. Knowles*, H.A. Raja, C.D. Roberts, O. Bader, J. houbraken, G.H. Goldman, N.H. Oberlies, & A. Rokas (2020). Variation Among Biosynthetic Gene Clusters, Secondary Metabolite Profiles, and Cards of Virulence Across <i>Aspergillus</i> Species. (*Equal contributors). Genetics. doi: 10.1534/genetics.120.303549.
											<a href="publication_pdfs/2020_Steenwyk_etal_Genetics.pdf"><u>PDF</u></a> <br />
											<a href="https://www.genetics.org/content/216/2/481">Highlighted Article</a>
											<div data-badge-type="4" data-doi="doi: 10.1534/genetics.120.303549" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											24. Ries, L.N.A., L. Pardeshi, Z. Dong, K. Tan, <strong>J.L. Steenwyk</strong>, A.C. Colabardini, J.A.F. Filho, P.A. de Castro, L.P. Silva, N.W. Preite, F. Almeida, L.J. de Assis, R.A.C. dos Santos, P. Bowyer, M. Bromley, R.A. Owens, S. Doyle, M. Demasi, D.C.R. Hernández, L.E.S. Netto, M.T. Pupo, A. Rokas, F.V. Loures, K.H. Wong, & G.H. Goldman (2020). The <i>Aspergillus fumigatus</i> transcription factor RglT is important for gliotoxin biosynthesis and self-protection, and virulence. PLoS Pathogens. doi: 10.1371/journal.ppat.1008645. <a href="publication_pdfs/2020_Ries_et_al_PLoS_Pathogens.pdf"><u>PDF</u></a><br />
											<div data-badge-type="4" data-doi="doi: 10.1371/journal.ppat.1008645" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											23. <strong>Steenwyk, J.L.</strong>*, A.L. Lind*, L.N.A. Ries, T.F. dos Reis, L.P. Silva, F. Almeida, R.W. Bastos, T.F. de Campos Fraga da Silva, V.L.D. Bonato, A.M. Pessoni, F. Rodrigues, H.A. Raja, S.L. Knowles, N.H. Oberlies, K. Lagrou, G.H. Goldman^, A. Rokas^ (2020). Pathogenic allodiploid hybrids of <i>Aspergillus</i> fungi. (*Equal contributors; ^Senior authors). Current Biology. doi: 10.1016/j.cub.2020.04.071. <a href="publication_pdfs/2020_Steenwyk_and_Lind_et_al_Current_Biology.pdf"><u>PDF</u></a><br />
											<a href="https://www.cell.com/current-biology/issue?pii=S0960-9822(19)X0014-7#fullCover">Cover Article</a><br />
											<a href="images/Steenwyk_and_Lind_etal_CurrBio_NatMicroReviews_Highlights.pdf">Nature Reviews Microbiology Research Highlights</a><br />
											<div data-badge-type="4" data-doi="doi: 10.1016/j.cub.2020.04.071" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											22. Mead, M.E.*, A.T. Borowsky*, B. Joehnk, <strong>J.L. Steenwyk</strong>, X.-X. Shen, A. Sil, & A. Rokas (2020). Recurrent loss of <i>abaA</i>, a master regulator of asexual development in filamentous fungi, correlates with changes in genomic and morphological traits. (*Equal contributors) Genome Biology and Evolution. doi: 10.1093/gbe/evaa107. <a href="publication_pdfs/2020_Mead_Borowsky_et_al_Genome_Biology_and_Evolution.pdf"><u>PDF</u></a><br />
											<div data-badge-type="4" data-doi="doi: 10.1093/gbe/evaa107" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											21. Santos, R.A.C., <strong>J.L. Steenwyk</strong>, O. Rivero-Menendez, M.E. Mead, L.P. Silva, R.W. Bastos, A. Alastruey-Izquierdo, G.H. Goldman^, & A. Rokas^ (2020). Genomic and phenotypic heterogeneity of clinical isolates of the human pathogens <i>Aspergillus fumigatus</i>, <i>Aspergillus lentulus</i> and <i>Aspergillus fumigatiaffinis</i>. (^Senior contributors). Frontiers in Genetics. doi: 10.3389/fgene.2020.00459.
											<a href="publication_pdfs/2020_dos_Santos_et_al_Frontiers_in_Genetics.pdf"><u>PDF</u></a><br />
											<div data-badge-type="4" data-doi="doi: 10.3389/fgene.2020.00459" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											20. Bastos, R.W., C. Valero, L.P. Silva, T. Schoen, M. Drott, V. Brauer, R. Silva-Rocha, A. Lind, <strong>J.L. Steenwyk</strong>, A. Rokas, F. Rodrigues, A. Resendiz-Sharpe, K. Lagrou, M. Marcet-Houben, T. Gabaldon, E. McDonnell, I. Reid, A. Tsang, B.R. Oakley, F. Loures, F. Almeida, A. Huttenlocher, N.P. Keller, L. Ries, & G.H. Goldman (2020). Functional characterization of clinical isolates of the opportunistic fungal pathogen <i>Aspergillus nidulans</i>. mSphere. doi:10.1128/mSphere.00153-20. 
											<a href="publication_pdfs/2020_Bastos_et_al_mSphere.pdf"><u>PDF</u></a><br />
											<div data-badge-type="4" data-doi="doi: 10.1128/mSphere.00153-20" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											19. Rokas, A., M.E. Mead, <strong>J.L. Steenwyk</strong>, N.H. Oberlies, & G.H. Goldman (2020). Evolving moldy murderers: <i>Aspergillus</i> section <i>Fumigati</i> as a model for studying the repeated evolution of fungal pathogenicity. PLoS Pathogens. doi: 10.1371/journal.ppat.1008315. 
											<a href="publication_pdfs/2020_Rokas_et_al_PLoS_Pathogens.pdf"><u>PDF</u></a><br />
											<div data-badge-type="4" data-doi="doi: 10.1371/journal.ppat.1008315" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											18. Knowles, S.L., M.E. Mead, L.P. Silva, H.A. Raja, <strong>J.L. Steenwyk</strong>, G.H. Gustavo^, A. Rokas^, & N.H. Oberlies^ (2020). Gliotoxin, a known virulence factor in the major human pathogen <i>Aspergillus fumigatus</i>, is also biosynthesized by the non-pathogenic relative <i>A. fischeri</i>. (^Senior authors) mBio. doi: 10.1128/mBio.03361-19. 
											<a href="publication_pdfs/2020_Knowles_et_al_mBio.pdf"><u>PDF</u></a><br /> <div data-badge-type="4" data-doi="doi: 10.1128/mBio.03361-19" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											17. Libkind, D., D. Peris, F.A. Cubillos, <strong>J.L. Steenwyk</strong>, D.A. Opulente, Q.K. Langdon, N. Bellora, A. Rokas, & C.T. Hittinger (2020). Into the wild: new yeast genomes from natural environments and new tools for their analysis. FEMS Yeast Research. doi: 10.1093/femsyr/foaa008. <a href="publication_pdfs/2020_Libkind_et_al_FEMS_Yeast_Research.pdf"><u>PDF</u></a><br />
											<div data-badge-type="4" data-doi="doi: 10.1093/femsyr/foaa008" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											16. Rokas, A., M.E. Mead, <strong>J.L. Steenwyk</strong>, H.A. Raja, & N.H., Oberlies (2020). Biosynthetic gene clusters and the evolution of fungal chemodiversity. Natural Product Reports. doi: 10.1039/C9NP00045C. <a href="publication_pdfs/2020_Rokas_et_al_Natural_Products_Reports.pdf"><u>PDF</u></a><br />
											<div data-badge-type="4" data-doi="doi: 10.1039/C9NP00045C" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											15. Bodinaku, I., J. Shaffer, A.B. Connors, <strong>J.L. Steenwyk</strong>, E. Kastman, A. Rokas, A. Robbat, & B. Wolfe (2019). Rapid phenotypic and metabolomic domestication of wild <i>Penicillium</i> molds on cheese. mBio. doi: 10.1128/mBio.02445-19. <a href="publication_pdfs/2019_Bodinaku_etal_mBio.pdf"><u>PDF</u></a><br />
											<a href="https://mbio.asm.org/content/10/5/e02445-19">Editor's Pick</a><br />
											<div data-badge-type="4" data-doi="doi: 10.1128/mBio.02445-19" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											14. Mead, M.E.*, H.A. Raja*, <strong>J.L. Steenwyk</strong>, S.L. Knowles, N.H. Oberlies^, & A. Rokas^ (2019). Draft genome sequence of the griseofulvin-producing fungus <i>Xylaria flabelliformis</i> strain G536. (*Equal contributors; ^Senior authors) Microbiology Resource Announcements. doi: 10.1128/MRA.00890-19. <a href="publication_pdfs/2019_Mead_and_Raja_et_al_Microbiology_Resource_Announcements.pdf"><u>PDF</u></a><br />
											<div data-badge-type="4" data-doi="doi: 10.1128/MRA.00890-19" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											13. <strong>Steenwyk, J.L.</strong> & A. Rokas (2019). treehouse: a user-friendly application to obtain subtrees from large phylogenies. BMC Research Notes. doi: 10.1186/s13104-019-4577-5. <a href="publication_pdfs/2019_Steenwyk_and_Rokas_BMC_Research_Notes.pdf"><u>PDF</u></a><br />
											<a href="https://github.com/JLSteenwyk/treehouse">Download treehouse</a>
											<div data-badge-type="4" data-doi="doi: 10.1186/s13104-019-4577-5" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											12. Labella, A.L., D.A. Opulente, <strong>J.L. Steenwyk</strong>, C.T. Hittinger, & A. Rokas (2019). Variation and selection on codon usage bias across an entire subphylum. PLoS Genetics. doi: 10.1371/journal.pgen.1008304. <a href="publication_pdfs/2019_LaBella_et_al_PLoS_Genetics.pdf"><u>PDF</u></a>
											<div data-badge-type="4" data-doi="doi: 10.1371/journal.pgen.1008304" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											11. <strong>Steenwyk, J.L.</strong>, X.-X. Shen, A.L. Lind, G.H. Goldman, & A. Rokas (2019). A robust phylogenomic timetree for biotechnologically and medically important fungi in the genera <i>Aspergillus</i> and <i>Penicillium</i>. mBio. doi: 10.1128/mBio.00925-19. <a href="publication_pdfs/2019_Steenwyk_et_al_mBio.pdf"><u>PDF</u></a><br />
											<div data-badge-type="4" data-doi="doi: 10.1128/mBio.00925-19" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											10. <strong>Steenwyk, J.L.</strong>, D. Opulente, J. Kominek, X.-X. Shen, X. Zhou, A.L. LaBella, N.P. Bradley, B.F. Eichman, N. Čadež, D. Libkind, J. DeVirgilio, A.B. Hulfachor, C.P. Kurtzman, C.T. Hittinger^, & A. Rokas^ (2019). Extensive loss of cell cycle and DNA repair genes in an ancient lineage of bipolar budding yeasts. (^Senior authors) PLoS Biology. doi: 10.1371/journal.pbio.3000255. <a href="publication_pdfs/2019_Steenwyk_et_al_PLoS_Biology.pdf"><u>PDF</u></a><br />
											<div data-badge-type="4" data-doi="doi: https://doi.org/10.1371/journal.pbio.3000255" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											9. Ries, L.N.A., <strong>J.L. Steenwyk</strong>, P.A. de Castro, P.B.A. de Lima, F. Almeida, L.J. de Assis, A.O. Manfiolli, A. Takahashi-Nakaguchi, Y. Kusuya, D. Hagiwara, H. Takahashi, X. Wang, J. Obar, A. Rokas, & G.H. Goldman (2019). Nutritional heterogeneity among <i>Aspergillus fumigatus</i> strains has consequences for virulence in a strain- and host-dependent manner. Frontiers in Microbiology. doi: 10.3389/fmicb.2019.00854. <a href="publication_pdfs/2019_Ries_et_al_Frontiers_in_Microbiology.pdf"><u>PDF</u></a>
											<div data-badge-type="4" data-doi="doi: 10.3389/fmicb.2019.00854" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											8. Mead M.E., S.L. Knowles, H.A. Raja, S. R. Beattie, C.H. Kowalski, <strong>J.L. Steenwyk</strong>, L.P. Silva, J. Chiaratto, L.N.A. Ries, G.G. Goldman, R.A. Cramer, N.H. Oberlies, & A. Rokas (2019). Characterizing the pathogenic, genomic, and chemical traits of <i>Aspergillus fischeri</i>, the closest sequenced relative of the major human fungal pathogen <i>Aspergillus fumigatus</i>. mSphere. doi: 10.1128/mSphere.00018-19. <a href="publication_pdfs/2019_Mead_et_al_mSphere.pdf"><u>PDF</u></a>
											<div data-badge-type="4" data-doi="doi: 10.3389/fmicb.2019.00285" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											7. Knowles, S.L., H.A. Raja, A.J. Wright, A.M.L. Lee, L.K. Caesar, N.B. Cech, M.E. Mead, <strong>J.L. Steenwyk</strong>, L. Ries, G.H. Goldman, A. Rokas, N.H. Oberlies (2019). Mapping the fungal battlefield: using <i>in situ</i> chemistry and deletion mutants to monitor interspecific chemical interactions between fungi. Frontiers in Microbiology. doi: 10.3389/fmicb.2019.00285. <a href="publication_pdfs/2019_Knowles_et_al_Frontiers_in_Microbiology.pdf"><u>PDF</u></a>
											<div data-badge-type="4" data-doi="doi: 10.3389/fmicb.2019.00285" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											6. Eidem, H.R., <strong>J.L. Steenwyk</strong>, J. Wisecaver, J.A. Capra, P. Abbot, & A. Rokas (2018). integRATE: a desirability-based data integration framework for the prioritization of candidate genes across heterogeneous omics and its application to preterm birth. BMC Medical Genomics. doi: 10.1186/s12920-018-0426-y. <a href="publication_pdfs/2018_Eidem_etal_BMC_Medical_Genomics.pdf"><u>PDF</u></a>
											<div data-badge-type="4" data-doi="doi: 10.1186/s12920-018-0426-y" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											5. Shen, X.-X.*, D.A. Opulente*, J. Kominek*, X. Zhou*, <strong>J.L. Steenwyk</strong>, K.V. Buh, M.A.B. Haase, J.H. Wisecaver, M. Wang, D.T. Doering, J.T. Boudouris, R.M. Schneider, Q.K. Langdon, M. Ohkuma, R. Endoh, M. Takashima, R. Manabe, N. Čadež, D. Libkind, C.A. Rosa, J. DeVirgilio, A.B. Hulfachor, M. Groenewald, C.P. Kurtzman^, C.T. Hittinger^ & A. Rokas^ (2018). Tempo and mode of genome evolution in the budding yeast subphylum. Cell. doi: 10.1016/j.cell.2018.10.023. (*Equal contributors; ^Senior authors) <a href="publication_pdfs/2018_Shen_et_al_Cell.pdf"><u>PDF</u></a><br />
											<a href="https://natureecoevocommunity.nature.com/users/25228-antonis-rokas/posts/40795-mitigating-the-impact-of-rogue-genes-in-phylogenomic-studies">The story behind the paper in Nature Ecology & Evolution blog</a><br />
											<div data-badge-type="4" data-doi="doi: 10.1016/j.cell.2018.10.023" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											4. Segal, E.S., V. Gritsenko, A. Levitan, B. Yadav, N. Dror, <strong>J.L. Steenwyk</strong>, Y. Silberberg, K. Mielich, A. Rokas, N.A.R. Gow, R. Kunze, R. Sharan, & J. Berman (2018). Gene Essentiality Analyzed by In Vivo Transposon Mutagenesis and Machine Learning in a Stable Haploid Isolate of <i>Candida	albicans</i>. mBio. doi: 10.1128/mBio.02048-18. <a href="publication_pdfs/2018_Segal_et_al_mBio.pdf"><u>PDF</u></a><br />	
											<a href="https://mbio.asm.org/content/9/5/e02048-18">Editor's Pick</a><br />
											<div data-badge-type="4" data-doi="doi: 10.1128/mBio.02048-18" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											3. <strong>Steenwyk, J.L.</strong> & A. Rokas (2018). Copy number variation in fungi and its implications for wine yeast genetic diversity and adaptation. Frontiers in Microbiology. doi: 10.3389/fmicb.2018.00288. <a href="publication_pdfs/2018_Steenwyk_and_Rokas_Frontiers_in_Microbiology.pdf"><u>PDF</u></a>
											<div data-badge-type="4" data-doi="doi: https://doi.org/10.3389/fmicb.2018.00288" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />

											2. <strong>Steenwyk, J.</strong> & A. Rokas (2017). Extensive copy number variation in fermentation-related genes among <i>Saccharomyces cerevisiae</i> wine strains. G3: Genes | Genomes | Genetics. doi: 10.1534/g3.117.040105. <a href="publication_pdfs/2017_Steenwyk_and_Rokas_G3.pdf" style="color: #3097ff"><u>PDF</u></a>
											<div data-badge-type='4' data-doi="10.1534/g3.117.040105" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="5"></div>

											<br />

											1. <strong>Steenwyk, J.L.</strong>, J.S. Soghigian, J.R. Perfect, & J.G. Gibbons (2016). Copy number variation contributes to cryptic genetic variation in outbreak lineages of <i>Cryptococcus gattii</i> from the North American Pacific Northwest. BMC Genomics. doi: 10.1186/s12864-016-3044-0. <a href="publication_pdfs/2016_Steenwyk_et_al_BMC_Genomics.pdf" style="color: #3097ff"><u>PDF</u></a>
											<div data-badge-type='4' data-doi="10.1186/s12864-016-3044-0" data-hide-no-mentions="true" data-badge-popover='right' class="altmetric-embed" data-hide-less-than="20"></div>

											<br />
											<center><a href="#Top" class="goto-pub scrolly">Top of page</a></center>
										</section>
									</section>

							</div>
							<div class="col-3 col-12-medium">

								<!-- Sidebar -->
									<section id="sidebar">
										<section>
											<h3><u>Journal Covers</u></h3>
											<a href="publication_pdfs/2020_Steenwyk_and_Lind_et_al_Current_Biology.pdf" class="image fit"><img src="journal_covers/Steenwyk_Lind_Current_biology_Cover.jpg" alt="" />Steenwyk and Lind <i>et al.</i> 2020</a>
										</section>
										<hr>
										<section>
											<h3><u>Popular Science</u></h3>
											3. <strong>Steenwyk, J.L.</strong> & M. Jonika. How to get started in science communication, <i>ecrLife</i>. August 21, 2020. <a href="https://ecrlife.org/how-to-get-started-in-science-communication/">Link</a>

											<br /><br />

											2. <strong>Steenwyk, J.L.</strong> & A. Rokas. A new hybrid fungus is found in hospitals and linked to lung disease, <i>The Conversation</i>. <a href="https://theconversation.com/a-new-hybrid-fungus-is-found-in-hospitals-and-linked-to-lung-disease-138783">Link</a>
											
											<br /><br />

											1. <strong>Steenwyk, J.L.</strong> & A. Rokas. An outlaw yeast thrives with genetic chaos – and could provide clues for understanding cancer growth, <i>The Conversation</i>. <a href="https://theconversation.com/an-outlaw-yeast-thrives-with-genetic-chaos-and-could-provide-clues-for-understanding-cancer-growth-117359">Link</a>
											<hr>
										</section>
									</section>

							</div>
						</div>
					</div>
				</div>

			<!-- Footer -->
				<footer id="footer">
					<ul class="icons">
						<li><a href="https://twitter.com/jlsteenwyk" class="icon brands alt fa-twitter"><span class="label">Twitter</span></a></li>
						<li><a href="https://scholar.google.com/citations?user=VXV2j6gAAAAJ&hl=en" class="icon brands alt fa-google"><span class="label">Google</span></a></li>
						<li><a href="https://www.researchgate.net/profile/Jacob_Steenwyk" class="icon brands alt fa-researchgate"><span class="label">ResearchGate</span></a></li>
						<li><a href="https://github.com/JLSteenwyk" class="icon brands alt fa-github"><span class="label">GitHub</span></a></li>
						<li><a href="https://www.linkedin.com/in/jlsteenwyk/" class="icon brands alt fa-linkedin-in"><span class="label">LinkedIn</span></a></li>
						<li><a href="contact.html" class="icon solid alt fa-envelope"><span class="label">Email</span></a></li>
					</ul>
					<ul class="copyright">
						<li>&copy; 2020 Jacob L. Steenwyk</li>
					</ul>
				</footer>

		</div>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/jquery.scrolly.min.js"></script>
			<script src="assets/js/jquery.dropotron.min.js"></script>
			<script src="assets/js/jquery.scrollex.min.js"></script>
			<script src="assets/js/browser.min.js"></script>
			<script src="assets/js/breakpoints.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>
		<!-- almetric script -->
		<script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

	</body>
</html>