<!DOCTYPE HTML>
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-104875636-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-104875636-1');
</script>
<html>
	<style>
		.TextWrapRight{float:right; margin: 10px;}
	</style>
	<style>
		.TextWrapLeft{float:left; margin: 10px;}
	</style>
	<head>
		<title>J.L. Steenwyk</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />
		<link rel="icon" href="images/website_flavicon_network.png">
		<noscript><link rel="stylesheet" href="assets/css/noscript.css" /></noscript>
	</head>
	<body class="is-preload">
		<article id="Top">
			<div id="page-wrapper">
			<!-- Header -->
				<header id="header">
					<nav id="nav">
						<ul>
							<li><a href="index.html#content" style="color: white">About</a></li>
							<li><a href="publications.html" style="color: white">Publications</a></li>
							<li><a href="news.html" style="color: white">News</a></li>
							<li>
								<a href="arts.html" style="color: white">Arts</a>
								<ul>
									<li><a href="sciart.html" style="color: white">SciArt Shop</a></li>
									<li><a href="arts_visual.html" style="color: white">Visual</a></li>
									<li><a href="arts_music.html" style="color: white">Music</a></li>
									<li><a href="arts_poetry.html" style="color: white">Poetry</a></li>
								</ul>
							</li>
							<li><a href="software.html" style="color: white">Software</a></li>
							<li><a href="curriculum_vitae.html" style="color: white">Curriculum Vitae</a></li>
							<li><a href="microbial_goodreads/index.html" alt="index.html" style="color: white">Microbial GoodReads</a></li>
							<li><a href="resources.html" style="color: white">Resources</a></li>
							<li><a href="contact.html" style="color: white">Contact</a></li>
						</ul>
					</nav>
				</header>

			<!-- Main -->
				<div id="main" class="wrapper style1">
					<div class="container">
						<!-- Content -->
						<section id="content">
							<header class="major">
								<h2>Software</h2>
							</header>

							<section id="ClipKIT">
								<h3>ClipKIT</h3>
									<img class="TextWrapLeft" src="software_images/clipkit_logo.jpg" alt="software_images/clipkit_logo.jpg", style="width: 25%; height: 25%">Since the early 1990's, there has been one predominant guiding light for multiple sequence alignment trimming - the removal of phylogenetically uncertain sites defined as those that are highly divergent; however, the efficacy of this approach has been called into question. ClipKIT implements an alternative strategy wherein sites with phylogenetic certainty are retained and others are removed. Our benchmarking analyses show that ClipKIT is a reliable and top performing software. 
									<a href="publication_pdfs/2020_Steenwyk_etal_PLOS_Biology.pdf">Publication PDF</a>; <a href="https://jlsteenwyk.com/ClipKIT/">Documentation</a>; <a href="https://github.com/JLSteenwyk/ClipKIT">Source code</a><br />
									</center>
								</p>
							</section>

							<hr>

							<section id="PhyKIT">
								<h3>PhyKIT</h3>
									<img class="TextWrapLeft" src="software_images/phykit_logo.png" alt="software_images/phykit_logo.png", style="width: 25%; height: 25%">Diverse disciplines in biology process and analyze multiple sequence alignments (MSAs) and phylogenetic trees to evaluate their information content, infer evolutionary events and processes, and predict gene function. However, automated processing of MSAs and trees remains a challenge due to the lack of a unified toolkit. To fill this gap, we introduce PhyKIT, a toolkit for the UNIX shell environment with 35 (and counting) functions that process MSAs and trees.
									<a href="publication_pdfs/2021_Steenwyk_etal_Bioinformatics.pdf">Publication PDF</a>; <a href="https://jlsteenwyk.com/PhyKIT/">Documentation</a>; <a href="https://github.com/JLSteenwyk/PhyKIT">Source code</a><br />
									</center>
								</p>
							</section>

							<hr>

							<section id="BioKIT">
								<h3>BioKIT</h3>
									<img class="TextWrapLeft" src="software_images/biokit_logo.jpg" alt="software_images/phykit_logo.png", style="width: 25%; height: 25%">Bioinformatic workflows often rely on individual software to conduct single analyses, which makes maintaining workflows cumbersome and threatens reproducibility. To address this obstacle, we introduce BioKIT, a versatile toolkit for the UNIX shell environment with 40 functions that conduct processing and analysis such as genome assembly quality assessment, alignment summary statistics, relative synonymous codon usage, codon optimization estimation, and more.
									<a href="publication_pdfs/2021_Steenwyk_etal_Bioinformatics.pdf">Publication PDF</a>; <a href="https://jlsteenwyk.com/BioKIT/">Documentation</a>; <a href="https://github.com/JLSteenwyk/BioKIT">Source code</a><br />
									</center>
								</p>
							</section>

							<hr>

							<section id="orthosnap">
								<h3>orthoSNAP</h3>
									<img class="TextWrapLeft" src="software_images/orthosnap_full_logo.jpg" alt="software_images/orthosnap_full_logo.jpg", style="width: 25%; height: 25%">Molecular evolution studies such as phylogenomics and surveys of positive selection often strictly rely on single-copy orthologous genes (SC-OGs). To increase the number of molecular markers for use in molecular evolution studies, orthosnap identifies subgroups of SC-OGs nested within larger gene families using a phylogenetically informed framework. The resulting SC-OGs are termed SNAP-OGs because they have undergone a <u>s</u>plitti<u>n</u>g <u>a</u>nd <u>p</u>runing procedure.
									<a href="publication_pdfs/2021_Steenwyk_and_Rokas_G3.pdf">Publication PDF</a>; <a href="https://jlsteenwyk.com/orthosnap/">Documentation</a>; <a href="https://github.com/JLSteenwyk/orthosnap">Source code</a><br />
									</center>
								</p>
							</section>

							<hr>

							<section id="orthofisher">
								<h3>orthofisher</h3>
									<img class="TextWrapLeft" src="software_images/orthofisher_full_logo.jpg" alt="software_images/orthofisher.jpg", style="width: 25%; height: 25%">orthofisher conducts automated HMMsearches among a set of proteomes using a predetermined set of orthologs. Sequence similarity searches classify results as multi-copy, single-copy, or absent in a given genome. For the purposes of phylogenomics/phylogenetics, multi-fasta files are generated for all sequences as well as those that are single-copy; for gene family copy number determination, easily parsed output files contain absolute copy number of hits from the sequence similarity search. 
									<a href="publication_pdfs/2021_Steenwyk_and_Rokas_G3.pdf">Publication PDF</a>; <a href="https://jlsteenwyk.com/orthofisher/">Documentation</a>; <a href="https://github.com/JLSteenwyk/orthofisher">Source code</a><br />
									</center>
								</p>
							</section>

							<hr>

							<section id="treehouse">
								<h3>treehouse</h3>
									<img class="TextWrapLeft" src="software_images/treehouse_logo.png" alt="software_images/treehouse_logo.png", style="width: 25%; height: 25%">Sometimes phylogenies are so large it is challenging to determine the relationships among a subset of taxa. To remedy this issue, <i>treehouse</i>, a user friendly GUI app, allows users to obtain subtrees from larger phylogenies. To obtain subtrees, upload a list of tip names in the desired subtree from an inputted phylogeny or a phylogeny from the <i>treehouse</i> database. Thereafter, users can download a pdf or newick file of the subtree of interest. 
									<a href="publication_pdfs/2019_Steenwyk_and_Rokas_BMC_Research_Notes.pdf">Publication PDF</a>; <a href="https://github.com/JLSteenwyk/treehouse">Documentation & Source code</a>
									</center>
								</p>
							</section>

							<hr>

							<section id="ggpubfigs">
								<h3>ggpubfigs</h3>
									<img class="TextWrapLeft" src="software_images/ggpubfigs_logo.png" alt="software_images/ggpubfigs_logo.png", style="width: 25%; height: 25%">Creating publication ready figures can increases figure accessibility and improve science communication. Here, I present ggpubfigs, an R package with customized themes and colorblind friendly color palettes to help create publication (or presentation) ready figures. Please contact me if you would like to contribute a theme or color palette! 
									<a href="https://zenodo.org/record/4126988#.YCK_2ZNKhlc">Zenodo record</a>; <a href="https://github.com/JLSteenwyk/ggpubfigs">Documentation & Source code</a>
									</center>
								</p>
							</section>
						</section>
						</div>
						<br />
						<center><a href="#Top" class="goto-pub scrolly">Top of page</a></center>
				</div>
			</article>
			<br />
			<!-- Footer -->
				<footer id="footer">
					<ul class="icons">
						<li><a href="https://twitter.com/jlsteenwyk" class="icon brands alt fa-twitter"><span class="label">Twitter</span></a></li>
						<li><a href="https://scholar.google.com/citations?user=VXV2j6gAAAAJ&hl=en" class="icon brands alt fa-google"><span class="label">Google</span></a></li>
						<li><a href="https://www.researchgate.net/profile/Jacob_Steenwyk" class="icon brands alt fa-researchgate"><span class="label">ResearchGate</span></a></li>
						<li><a href="https://github.com/JLSteenwyk" class="icon brands alt fa-github"><span class="label">GitHub</span></a></li>
						<li><a href="https://www.linkedin.com/in/jlsteenwyk/" class="icon brands alt fa-linkedin-in"><span class="label">LinkedIn</span></a></li>
						<li><a href="contact.html" class="icon solid alt fa-envelope"><span class="label">Email</span></a></li>
					</ul>
					<ul class="copyright">
						<li>&copy; 2021 Jacob L. Steenwyk</li>
					</ul>
				</footer>

		</div>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/jquery.scrolly.min.js"></script>
			<script src="assets/js/jquery.dropotron.min.js"></script>
			<script src="assets/js/jquery.scrollex.min.js"></script>
			<script src="assets/js/browser.min.js"></script>
			<script src="assets/js/breakpoints.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>

	</body>
</html>