<!DOCTYPE HTML>
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-104875636-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-104875636-1');
</script>
<html>
	<style>
		.TextWrapRight{float:right; margin: 10px;}
	</style>
	<style>
		.TextWrapLeft{float:left; margin: 10px;}
	</style>
	<head>
		<title>J.L. Steenwyk</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />
		<link rel="icon" href="images/website_flavicon_network.png">
		<noscript><link rel="stylesheet" href="assets/css/noscript.css" /></noscript>
	</head>
	<body class="is-preload">
		<article id="Top">
			<div id="page-wrapper">
			<!-- Header -->
				<header id="header">
					<nav id="nav">
						<ul>
							<li><a href="index.html#content" style="color: white">About</a></li>
							<!-- <li><a href="blog/index.html#content" style="color: white">Blog</a></li> -->
							<li><a href="publications.html" style="color: white">Publications</a></li>
							<li><a href="news.html" style="color: white">News</a></li>
							<li>
								<a href="arts.html" style="color: white">Arts</a>
								<ul>
									<li><a href="sciart.html" style="color: white">SciArt Shop</a></li>
									<li><a href="arts_visual.html" style="color: white">Visual</a></li>
									<li><a href="arts_music.html" style="color: white">Music</a></li>
									<li><a href="arts_poetry.html" style="color: white">Poetry</a></li>
								</ul>
							</li>
							<li><a href="software.html" style="color: white">Software</a></li>
							<li><a href="curriculum_vitae.html" style="color: white">CV</a></li>
							<!-- <li><a href="evo_goodreads/index.html" alt="index.html" style="color: white">Evo GoodReads</a>
								<ul>
									<li><a href="evo_goodreads/evogoodreads_archive.html" style="color: white">2021 archive</a></li>
								</ul>
							</li> -->
							<li>
								<a href="genomely.html" target="_blank" style="color: white">Genomely
								<ul>
									<li><a href="genomely.html" target="_blank" style="color: white">What is Genomely?</a></li>
									<li><a href="https://genomelybio.com" target="_blank" style="color: white">Home</a></li>
									<li><a href="https://genomely.substack.com" target="_blank" style="color: white">Blog</a></li>
									<li><a href="https://clipkit.genomelybio.com/#/" target="_blank" style="color: white">ClipKIT browser</a></li>
								</ul>
							</li>
							<li><a href="resources.html" style="color: white">Resources</a></li>
							<li><a href="contact.html" style="color: white">Contact</a></li>
						</ul>
					</nav>
				</header>

			<!-- Main -->
				<div id="main" class="wrapper style1">
					<div class="container">
						<!-- Content -->
						<section id="content">
							<header class="major">
								<h2>Software</h2>
							</header>

							<!-- <p>
								<strong>
									Software maintenance is a labor of love. If you wish
									to support my work by buying me a coffee as a
									<a href="https://www.patreon.com/jlsteenwyk">Patreon supporter</a>, I would be very
									grateful! User's can subscribe for different tiers of support!
								</strong>
								<center>
									<a href="https://patreon.com/jlsteenwyk"><img src="https://img.shields.io/endpoint.svg?url=https%3A%2F%2Fshieldsio-patreon.vercel.app%2Fapi%3Fusername%3Djlsteenwyk%26type%3Dpatrons&style=flat" alt="Support me on Patreon" /></a>
								</center>
							</p>

							<hr> -->
							
							<!-- <section id="Header">
								<div class="row gtr-50 gtr-uniform">
									<div class="col-4 col-6-xsmall"><span class="image fit"><img src="software_images_cols/clipkit_logo.png" alt="" />
										<li><a href="software.html">About</a></li>
										<li><a href="publication_pdfs/2020_Steenwyk_etal_PLOS_Biology.pdf">Publication PDF</a></li>
										<li><a href="https://jlsteenwyk.com/ClipKIT/">Documentation</a></li>
										<li><a href="https://github.com/JLSteenwyk/ClipKIT">Source code</a></li>
									</span>
									</div>
									
									<div class="col-4 col-6-xsmall"><span class="image fit"><img src="software_images_cols/phykit_logo.png" alt="" />
										<li><a href="#PhyKIT">About</a></li>
										<li><a href="publication_pdfs/2021_Steenwyk_etal_Bioinformatics.pdf">Publication PDF</a></li>
										<li><a href="https://jlsteenwyk.com/PhyKIT/">Documentation</a></li>
										<li><a href="https://github.com/JLSteenwyk/PhyKIT">Source code</a></li>
									</span>
									</div>

									<div class="col-4 col-6-xsmall"><span class="image fit"><img src="software_images_cols/biokit_logo.png" alt="" />
										<li><a href="#BioKIT">About</a></li>
										<li><a href="publication_pdfs/2022_Steenwyk_etal_Genetics.pdf">Publication PDF</a></li>
										<li><a href="https://jlsteenwyk.com/BioKIT/">Documentation</a></li>
										<li><a href="https://github.com/JLSteenwyk/BioKIT">Source code</a></li>
									</span>
									</div>
								</div>

								<br />

								<div class="row gtr-50 gtr-uniform">
									<div class="col-4 col-6-xsmall"><span class="image fit"><img src="software_images_cols/orthosnap_logo.png" alt="" />
										<li><a href="#orthosnap">About</a></li>
										<li><a href="publication_pdfs/2022_Steenwyk_etal_PLoS_Biology.pdf">Publication PDF</a></li>
										<li><a href="https://jlsteenwyk.com/orthosnap/">Documentation</a></li>
										<li><a href="https://github.com/JLSteenwyk/orthosnap">Source code</a></li>

									</div>
									
									<div class="col-4 col-6-xsmall"><span class="image fit"><img src="software_images_cols/orthofisher_logo.png" alt="" />
										<li><a href="#orthofisher">About</a></li>
										<li><a href="publication_pdfs/2021_Steenwyk_and_Rokas_G3.pdf">Publication PDF</a></li>
										<li><a href="https://jlsteenwyk.com/orthofisher/">Documentation</a></li>
										<li><a href="https://github.com/JLSteenwyk/orthofisher">Source code</a></li>
										</span>
									</div>
								
									<div class="col-4 col-6-xsmall"><span class="image fit"><img src="software_images_cols/treehouse_logo.png" alt="" />
										<li><a href="#treehouse">About</a></li>
										<li><a href="publication_pdfs/2019_Steenwyk_and_Rokas_BMC_Research_Notes.pdf">Publication PDF</a></li>
										<li><a href="https://github.com/JLSteenwyk/treehouse">Documentation</a></li>
										<li><a href="https://github.com/JLSteenwyk/treehouse">Source code</a></li>
										</span>
									</div>
								</div>

								<br />

								<div class="row gtr-50 gtr-uniform">
									<div class="col-4 col-6-xsmall"><span class="image fit"><img src="software_images_cols/ggpubfigs_logo.png" alt="" />
										<li><a href="#ggpubfigs">About</a></li>
										<li><a href="publication_pdfs/2021_Steenwyk_and_Rokas_Microbiology_Resource_Announcements.pdf">Publication PDF</a></li>
										<li><a href="https://jlsteenwyk.com/ggpubfigs/">Documentation</a></li>
										<li><a href="https://github.com/JLSteenwyk/ggpubfigs">Source code</a></li>

									</div>
									
									<div class="col-4 col-6-xsmall"><span class="image fit"><img src="software_images_cols/lvbrs_logo.png" alt="" />
										<li><a href="#lvbrs">About</a></li>
										<li><a href="publication_pdfs/2021_Steenwyk_and_Rokas_G3.pdf">Publication PDF</a></li>
										<li><a href="https://jlsteenwyk.com/orthofisher/">Documentation</a></li>
										<li><a href="https://github.com/JLSteenwyk/orthofisher">Source code</a></li>
										</span>
									</div>
								</div>

								<hr>

							</section> -->

							<section id="ClipKIT">
								<h3><a id="ClipKIT">ClipKIT</a></h3>
								<img class="TextWrapLeft" src="software_images/clipkit_logo.jpg" alt="software_images/clipkit_logo.jpg", style="width: 25%; height: 25%">
								Since the early 1990's, there has been one predominant guiding light for multiple sequence alignment trimming -
								the removal of phylogenetically uncertain sites defined as those that are highly divergent; however, the
								efficacy of this approach has been called into question. ClipKIT implements an alternative strategy wherein
								sites with phylogenetic certainty are retained and others are removed. Our benchmarking analyses show that
								ClipKIT is a reliable and top performing software. 
								<a href="publication_pdfs/2020_Steenwyk_etal_PLOS_Biology.pdf">Publication PDF</a>;
								<a href="https://jlsteenwyk.com/ClipKIT/">Documentation</a>;
								<a href="https://github.com/JLSteenwyk/ClipKIT">Source code</a>
								<br /><br />
								<center>Run <a href="https://clipkit.genomelybio.com/">ClipKIT in the browser</a> and leave the computing up to <a href="https://www.genomelybio.com/">us</a>!</center>
								<br />
								<center>
								<a href="https://github.com/JLSteenwyk/ClipKIT/actions" alt="Build">
									<img src="https://img.shields.io/github/actions/workflow/status/JLSteenwyk/ClipKIT/ci.yml?branch=master">
								</a>
								<a href="https://codecov.io/gh/jlsteenwyk/clipkit" alt="Coverage">
									<img src="https://codecov.io/gh/jlsteenwyk/clipkit/branch/master/graph/badge.svg?token=0J49I6441V">
								</a>
								<a href="https://github.com/jlsteenwyk/clipkit/graphs/contributors" alt="Contributors">
									<img src="https://img.shields.io/github/contributors/jlsteenwyk/clipkit">
								</a>
								<a href="https://twitter.com/intent/follow?screen_name=jlsteenwyk" alt="Author Twitter">
									<a class="twitter-follow-button" data-size="small" href="https://twitter.com/jlsteenwyk">Follow @JLSteenwyk</a><script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
								</a>
								<br />
								<a href="https://pepy.tech/badge/clipkit">
									<img src="https://static.pepy.tech/personalized-badge/clipkit?period=total&units=international_system&left_color=grey&right_color=blue&left_text=PyPi%20Downloads">
								</a>
								<a href="https://lbesson.mit-license.org/" alt="License">
									<img src="https://img.shields.io/badge/License-MIT-blue.svg">
								</a>
								<a href="https://pypi.org/project/clipkit/" alt="PyPI - Python Version">
									<img src="https://img.shields.io/pypi/pyversions/clipkit">
								</a>
								<a href="https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.3001007">
									<img src="https://zenodo.org/badge/DOI/10.1371/journal.pbio.3001007.svg">
								</a>								
								</center>
							</section>

							<hr>

							<section id="PhyKIT">
								<h3>PhyKIT</h3>
								<img class="TextWrapLeft" src="software_images/phykit_logo.png" alt="software_images/phykit_logo.png", style="width: 25%; height: 25%">
								Diverse disciplines in biology process and analyze multiple sequence alignments (MSAs) and phylogenetic trees
								to evaluate their information content, infer evolutionary events and processes, and predict gene function.
								However, automated processing of MSAs and trees remains a challenge due to the lack of a unified toolkit.
								To fill this gap, we introduce PhyKIT, a Swiss-army knife-like toolkit for processing and analyzing
								multiple sequence alignments and phylogenetic trees.
								<a href="publication_pdfs/2021_Steenwyk_etal_Bioinformatics.pdf">Publication PDF</a>;
								<a href="https://www.preprints.org/manuscript/202404.1514/v1">Protocols manuscript</a>;
								<a href="https://jlsteenwyk.com/PhyKIT/">Documentation</a>;
								<a href="https://github.com/JLSteenwyk/PhyKIT">Source code</a><br />
								<br /><br />
								<center>
								<a href="https://github.com/JLSteenwyk/PhyKIT/actions" alt="Build">
									<img src="https://img.shields.io/github/actions/workflow/status/JLSteenwyk/PhyKIT/ci.yml?branch=master">
								</a>
								<a href="https://codecov.io/gh/jlsteenwyk/phykit" alt="Coverage">
									<img src="https://codecov.io/gh/jlsteenwyk/phykit/branch/master/graph/badge.svg?token=0J49I6441V">
								</a>
								<a href="https://github.com/jlsteenwyk/phykit/graphs/contributors" alt="Contributors">
									<img src="https://img.shields.io/github/contributors/jlsteenwyk/phykit">
								</a>
								<a href="https://twitter.com/intent/follow?screen_name=jlsteenwyk" alt="Author Twitter">
									<a class="twitter-follow-button" data-size="small" href="https://twitter.com/jlsteenwyk">Follow @JLSteenwyk</a><script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
								</a>
								<br />
								<a href="https://pepy.tech/badge/phykit">
									<img src="https://static.pepy.tech/personalized-badge/phykit?period=total&units=international_system&left_color=grey&right_color=blue&left_text=PyPi%20Downloads">
								</a>
								<a href="https://lbesson.mit-license.org/" alt="License">
									<img src="https://img.shields.io/badge/License-MIT-blue.svg">
								</a>
								<a href="https://pypi.org/project/phykit/" alt="PyPI - Python Version">
									<img src="https://img.shields.io/pypi/pyversions/phykit">
								</a>
								<a href="https://academic.oup.com/bioinformatics/article-abstract/37/16/2325/6131675?redirectedFrom=fulltext">
									<img src="https://zenodo.org/badge/DOI/10.1093/bioinformatics/btab096.svg">
								</a>
								</center>
							</section>

							<hr>

							<section id="BioKIT">
								<h3>BioKIT</h3>
								<img class="TextWrapLeft" src="software_images/biokit_logo.jpg" alt="software_images/biokit_logo.jpg", style="width: 25%; height: 25%">
								Bioinformatic workflows often rely on individual software to conduct single analyses, which makes maintaining
								workflows cumbersome and threatens reproducibility. To address this obstacle, we introduce BioKIT, a versatile
								toolkit that conducts diverse processing and analysis functions such as genome assembly quality assessment,
								alignment summary statistics, relative synonymous codon usage, codon optimization estimation, and more.
								<a href="publication_pdfs/2022_Steenwyk_etal_Genetics.pdf">Publication PDF</a>;
								<a href="https://jlsteenwyk.com/BioKIT/">Documentation</a>;
								<a href="https://github.com/JLSteenwyk/BioKIT">Source code</a><br />
								<br /><br />
								<center>
								<a href="https://github.com/JLSteenwyk/BioKIT/actions" alt="Build">
									<img src="https://img.shields.io/github/actions/workflow/status/JLSteenwyk/BioKIT/ci.yml?branch=main">
								</a>
								<a href="https://codecov.io/gh/JLSteenwyk/BioKIT" alt="Coverage">
									<img src="https://codecov.io/gh/JLSteenwyk/BioKIT/branch/main/graph/badge.svg?token=5X9C6YAVIG">
								</a>
								<a href="https://github.com/jlsteenwyk/biokit/graphs/contributors" alt="Contributors">
									<img src="https://img.shields.io/github/contributors/jlsteenwyk/biokit">
								</a>
								<a href="https://twitter.com/intent/follow?screen_name=jlsteenwyk" alt="Author Twitter">
									<a class="twitter-follow-button" data-size="small" href="https://twitter.com/jlsteenwyk">Follow @JLSteenwyk</a><script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
								</a>
								<br />
								<a href="https://pepy.tech/badge/jlsteenwyk-biokit">
									<img src="https://static.pepy.tech/personalized-badge/jlsteenwyk-biokit?period=total&units=international_system&left_color=grey&right_color=blue&left_text=PyPi%20Downloads">
								</a>
								<a href="https://lbesson.mit-license.org/" alt="License">
									<img src="https://img.shields.io/badge/License-MIT-blue.svg">
								</a>
								<a href="https://pypi.org/project/jlsteenwyk-biokit/" alt="PyPI - Python Version">
									<img src="https://img.shields.io/pypi/pyversions/jlsteenwyk-biokit">
								</a>
								<a href="https://academic.oup.com/genetics/advance-article/doi/10.1093/genetics/iyac079/6583183?login=true">
									<img src="https://zenodo.org/badge/DOI/10.1093/genetics/iyac079.svg">
								</a>
								</center>
							</section>

							<hr>

							<section id="orthosnap">
								<h3>OrthoSNAP</h3>
								<img class="TextWrapLeft" src="software_images/orthosnap_full_logo.jpg" alt="software_images/orthosnap_full_logo.jpg", style="width: 25%; height: 25%">
								Molecular evolution studies such as phylogenomics and surveys of positive selection often strictly rely on
								single-copy orthologous genes (SC-OGs). To increase the number of molecular markers for use in molecular
								evolution studies, OrthoSNAP identifies subgroups of SC-OGs nested within larger gene families using a
								phylogenetically informed framework. The resulting SC-OGs are termed SNAP-OGs because they have been identified
								using a <u>s</u>plitti<u>n</u>g <u>a</u>nd <u>p</u>runing procedure.
								<a href="publication_pdfs/2022_Steenwyk_etal_PLoS_Biology.pdf">Publication PDF</a>;
								<a href="https://jlsteenwyk.com/orthosnap/">Documentation</a>;
								<a href="https://github.com/JLSteenwyk/orthosnap">Source code</a><br />
								<br /><br />
								<center>
								<a href="https://github.com/JLSteenwyk/orthosnap/actions" alt="Build">
									<img src="https://img.shields.io/github/actions/workflow/status/JLSteenwyk/orthosnap/ci.yml?branch=master">
								</a>
								<a href="https://codecov.io/gh/JLSteenwyk/orthosnap">
								  <img src="https://codecov.io/gh/JLSteenwyk/orthosnap/branch/master/graph/badge.svg?token=FX66FUET0L"/>
								</a>
								<a href="https://github.com/jlsteenwyk/orthosnap/graphs/contributors" alt="Contributors">
									<img src="https://img.shields.io/github/contributors/jlsteenwyk/orthosnap">
								</a>
								<a href="https://twitter.com/intent/follow?screen_name=jlsteenwyk" alt="Author Twitter">
									<a class="twitter-follow-button" data-size="small" href="https://twitter.com/jlsteenwyk">Follow @JLSteenwyk</a><script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
								</a>
								<br />
								<a href="https://pepy.tech/badge/orthosnap">
									<img src="https://static.pepy.tech/personalized-badge/orthosnap?period=total&units=international_system&left_color=grey&right_color=blue&left_text=PyPi%20Downloads">
								</a>
								<a href="https://lbesson.mit-license.org/" alt="License">
									<img src="https://img.shields.io/badge/License-MIT-blue.svg">
								</a>
								<a href="https://pypi.org/project/orthosnap/" alt="PyPI - Python Version">
									<img src="https://img.shields.io/pypi/pyversions/orthosnap">
								</a>
								<a href="publication_pdfs/2022_Steenwyk_etal_PLoS_Biology.pdf">
								  <img src="https://zenodo.org/badge/DOI/10.1371/journal.pbio.3001827.svg">
								</a>
								</center>
							</section>

							<hr>

							<section id="orthofisher">
								<h3>orthofisher</h3>
								<img class="TextWrapLeft" src="software_images/orthofisher_full_logo.jpg" alt="software_images/orthofisher.jpg", style="width: 25%; height: 25%">
								orthofisher conducts automated HMMsearches among a set of proteomes using a predetermined set of orthologs.
								Sequence similarity searches classify results as multi-copy, single-copy, or absent in a given genome. For
								the purposes of phylogenomics/phylogenetics, multi-fasta files are generated for all sequences as well as
								those that are single-copy; for gene family copy number determination, easily parsed output files contain
								absolute copy number of hits from the sequence similarity search. 
								<a href="publication_pdfs/2021_Steenwyk_and_Rokas_G3.pdf">Publication PDF</a>;
								<a href="https://jlsteenwyk.com/orthofisher/">Documentation</a>;
								<a href="https://github.com/JLSteenwyk/orthofisher">Source code</a><br />
								<br /><br />
								<center>
								<a href="https://github.com/JLSteenwyk/orthofisher/actions" alt="Build">
									<img src="https://img.shields.io/github/actions/workflow/status/JLSteenwyk/orthofisher/ci.yml?branch=main">
								</a>
								<a href="https://codecov.io/gh/jlsteenwyk/orthofisher" alt="Coverage">
								  <img src="https://codecov.io/gh/JLSteenwyk/orthofisher/branch/main/graph/badge.svg?token=7LYMLHDL9D"/>
								</a>
								<a href="https://github.com/jlsteenwyk/orthofisher/graphs/contributors" alt="Contributors">
									<img src="https://img.shields.io/github/contributors/jlsteenwyk/orthofisher">
								</a>
								<a href="https://twitter.com/intent/follow?screen_name=jlsteenwyk" alt="Author Twitter">
									<a class="twitter-follow-button" data-size="small" href="https://twitter.com/jlsteenwyk">Follow @JLSteenwyk</a><script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
								</a>
								<br />
								<a href="https://pepy.tech/badge/orthofisher">
									<img src="https://static.pepy.tech/personalized-badge/orthofisher?period=total&units=international_system&left_color=grey&right_color=blue&left_text=PyPi%20Downloads">
								</a>
								<a href="https://lbesson.mit-license.org/" alt="License">
									<img src="https://img.shields.io/badge/License-MIT-blue.svg">
								</a>
								<a href="https://pypi.org/project/orthofisher/" alt="PyPI - Python Version">
									<img src="https://img.shields.io/pypi/pyversions/orthofisher">
								</a>
								<a href="https://academic.oup.com/g3journal/article/11/9/jkab250/6321954">
								  <img src="https://zenodo.org/badge/DOI/10.1093/g3journal/jkab250.svg">
								</a>
								</center>
							</section>

							<hr>

							<section id="treehouse">
								<h3>treehouse</h3>
								<img class="TextWrapLeft" src="software_images/treehouse_logo.png" alt="software_images/treehouse_logo.png", style="width: 25%; height: 25%">
								Sometimes phylogenies are so large it is challenging to determine the relationships among a subset of taxa. To
								remedy this issue, <i>treehouse</i>, a user friendly GUI app, allows users to obtain subtrees from larger
								phylogenies. To obtain subtrees, upload a list of tip names in the desired subtree from an inputted phylogeny
								or a phylogeny from the <i>treehouse</i> database. Thereafter, users can download a pdf or newick file of the
								subtree of interest. 
								<a href="publication_pdfs/2019_Steenwyk_and_Rokas_BMC_Research_Notes.pdf">Publication PDF</a>;
								<a href="https://github.com/JLSteenwyk/treehouse">Documentation & Source code</a><br />
								<br />
								<center>
								<a href="https://github.com/jlsteenwyk/ggpubfigs/graphs/contributors" alt="Contributors">
									<img src="https://img.shields.io/github/contributors/jlsteenwyk/treehouse">
								</a>
								<a href="https://twitter.com/intent/follow?screen_name=jlsteenwyk" alt="Author Twitter">
									<a class="twitter-follow-button" data-size="small" href="https://twitter.com/jlsteenwyk">Follow @JLSteenwyk</a><script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
								</a>
								<br />
								<a href="https://lbesson.mit-license.org/" alt="License">
									<img src="https://img.shields.io/badge/License-MIT-blue.svg">
								</a>
								<a href="https://bmcresnotes.biomedcentral.com/articles/10.1186/s13104-019-4577-5">
									<img src="https://zenodo.org/badge/DOI/10.1186/s13104-019-4577-5.svg">
								</a>
								</center>
							</section>

							<hr>

							<section id="ggpubfigs">
								<h3>ggpubfigs</h3>
								<img class="TextWrapLeft" src="software_images/ggpubfigs_logo.png" alt="software_images/ggpubfigs_logo.png", style="width: 25%; height: 25%">
								Creating publication ready figures can increases figure accessibility and improve science communication. Here,
								I present ggpubfigs, an R package with customized themes and colorblind friendly color palettes to help create
								publication (or presentation) ready figures. Please contact me if you would like to contribute a theme or color
								palette!
								<a href="publication_pdfs/2021_Steenwyk_and_Rokas_Microbiology_Resource_Announcements.pdf">Publication PDF</a>;
								<a href="https://github.com/JLSteenwyk/ggpubfigs">Documentation & Source code</a><br />
								<br />
								<center>
								<a href="https://github.com/jlsteenwyk/ggpubfigs/graphs/contributors" alt="Contributors">
									<img src="https://img.shields.io/github/contributors/jlsteenwyk/ggpubfigs">
								</a>
								<a href="https://twitter.com/intent/follow?screen_name=jlsteenwyk" alt="Author Twitter">
									<a class="twitter-follow-button" data-size="small" href="https://twitter.com/jlsteenwyk">Follow @JLSteenwyk</a><script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
								</a>
								<br />
								<a href="https://lbesson.mit-license.org/" alt="License">
									<img src="https://img.shields.io/badge/License-MIT-blue.svg">
								</a>
								<a href="https://journals.asm.org/doi/10.1128/MRA.00871-21">
								  <img src="https://zenodo.org/badge/DOI/10.1128/MRA.00871-21.svg">
								</a>
								</center>
							</section>

							<hr>

							<section id="orthoflow">
								<h3>Orthoflow</h3>
								<img class="TextWrapLeft" src="software_images/orthoflow_logo.jpg" alt="software_images/orthoflow_logo.jpg", style="width: 25%; height: 25%">
								The development, maintenance, and execution of phylogenomic workflows is challenging, requiring
								programming, data management skills, and familiarity with changing best practices. We introduce Orthoflow, a software
								wherein a single command automatically conducts end-to-end phylogenomic analysis using supermatrix
								and supertree methods from multiple input data formats. Orthoflow increases the accessibility of
								researchers to conduct rigorous phylogenomic analysis flexibly.
								<a href="https://www.researchsquare.com/article/rs-3699210/v1">Publication PDF</a>;
								<a href="https://rbturnbull.github.io/orthoflow/">Documentation</a>;
								<a href="https://github.com/rbturnbull/orthoflow">Source code</a><br />
								<br /><br />
								<center>
								<a href="https://github.com/rbturnbull/orthoflow/actions/workflows/docs.yml/badge.svg" alt="Docs">
									<img src="https://github.com/rbturnbull/orthoflow/actions/workflows/docs.yml/badge.svg">
								</a>
								<a href="https://github.com/rbturnbull/orthoflow/graphs/contributors" alt="Contributors">
									<img src="https://img.shields.io/github/contributors/rbturnbull/orthoflow">
								</a>
								<a href="https://twitter.com/intent/follow?screen_name=jlsteenwyk" alt="Author Twitter">
									<a class="twitter-follow-button" data-size="small" href="https://twitter.com/jlsteenwyk">Follow @JLSteenwyk</a><script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
								</a>
								<br />
								<a href="https://pepy.tech/badge/orthoflow">
									<img src="https://static.pepy.tech/personalized-badge/orthoflow?period=total&units=international_system&left_color=grey&right_color=blue&left_text=PyPi%20Downloads">
								</a>
								<a href="https://www.apache.org/licenses/LICENSE-2.0" alt="License">
									<img src="https://img.shields.io/badge/License-Apache%202.0-blue.svg">
								</a>
								<a href="https://pypi.org/project/orthoflow/" alt="PyPI - Python Version">
									<img src="https://img.shields.io/pypi/pyversions/orthoflow">
								</a>
								<a href="https://www.researchsquare.com/article/rs-3699210/v1">
								  <img src="https://zenodo.org/badge/DOI/10.21203/rs.3.rs-3699210.svg">
								</a>
								</center>
							</section>

							<hr>

							<!-- <section id="lvbrs">
								<h3>LVBRS</h3>
								<img class="TextWrapLeft" src="software_images/lvbrs_logo.jpg" alt="software_images/lvbrs_logo.jpg", style="width: 25%; height: 25%">
								Analysis of Bulk RNA-Seq data, differential expression analysis, and functional enrichment requires processing
								and handling diverse data types. The LVBRS toolkit (the Latch Verified Bulk RNA-Seq toolkit) conducts end-to-end
								analysis, differential expression, and functional enrichment analysis from raw reads from Bulk RNA-Sequencing 
								experiments using a cloud-based framework. LVBRS enables researchers to focus on interpretation of biological
								data, not processing, file handling, data management, and resource allocation. 
								<a href="https://www.biorxiv.org/content/10.1101/2022.11.10.516016v1">Publication PDF</a>;
								<a href="https://jlsteenwyk.com/orthofisher/">Documentation</a>;
								<a href="https://github.com/JLSteenwyk/orthofisher">Source code</a><br />
								<br />
								<center>
									<a href="https://lbesson.mit-license.org/" alt="License">
										<img src="https://img.shields.io/badge/License-MIT-blue.svg">
									</a>
									<a href="https://www.biorxiv.org/content/10.1101/2022.11.10.516016v1">
										<img src="https://zenodo.org/badge/DOI/10.1101/2022.11.10.516016.svg">
									</a>
									<a href="https://twitter.com/intent/follow?screen_name=jlsteenwyk" alt="Author Twitter">
										<a class="twitter-follow-button" data-size="small" href="https://twitter.com/jlsteenwyk">Follow @JLSteenwyk</a><script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
									</a>
								</center>
							</section>

							<hr> -->
						</section>
						</div>
						<br />
						<center><a href="#Top" class="goto-pub scrolly">Top of page</a></center>
				</div>
			</article>
			<br />
			<!-- Footer -->
				<footer id="footer">
					<ul class="icons">
						<li><a href="https://twitter.com/jlsteenwyk" class="icon brands alt fa-twitter"><span class="label">Twitter</span></a></li>
						<li><a href="https://scholar.google.com/citations?user=VXV2j6gAAAAJ&hl=en" class="icon brands alt fa-google"><span class="label">Google</span></a></li>
						<li><a href="https://www.researchgate.net/profile/Jacob_Steenwyk" class="icon brands alt fa-researchgate"><span class="label">ResearchGate</span></a></li>
						<li><a href="https://github.com/JLSteenwyk" class="icon brands alt fa-github"><span class="label">GitHub</span></a></li>
						<li><a href="https://www.linkedin.com/in/jlsteenwyk/" class="icon brands alt fa-linkedin-in"><span class="label">LinkedIn</span></a></li>
						<li><a href="contact.html" class="icon solid alt fa-envelope"><span class="label">Email</span></a></li>
					</ul>
					<ul class="copyright">
						<li>&copy; 2023 Jacob L. Steenwyk</li>
					</ul>
				</footer>

		</div>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/jquery.scrolly.min.js"></script>
			<script src="assets/js/jquery.dropotron.min.js"></script>
			<script src="assets/js/jquery.scrollex.min.js"></script>
			<script src="assets/js/browser.min.js"></script>
			<script src="assets/js/breakpoints.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>

	</body>
</html>
