<!DOCTYPE HTML>
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-104875636-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-104875636-1');
</script>
<html>
	<style>
		.TextWrapRight{float:right; margin: 10px;}
	</style>
	<style>
		.TextWrapLeft{float:left; margin: 10px;}
	</style>
	<head>
		<title>J.L. Steenwyk</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="../assets/css/main.css" />
		<link rel="icon" href="../images/website_flavicon_network.png">
		<noscript><link rel="stylesheet" href="../assets/css/noscript.css" /></noscript>
	</head>
	<body class="is-preload">
		<article id="Top">
			<div id="page-wrapper">
			<!-- Header -->
				<header id="header">
					<nav id="nav">
						<ul>
							<li><a href="../index.html#content" style="color: white">About</a></li>
							<li><a href="../publications.html" style="color: white">Publications</a></li>
							<li><a href="../news.html" style="color: white">News</a></li>
							<li><a href="../software.html" style="color: white">Software</a></li>
							<li><a href="../curriculum_vitae.html" style="color: white">Curriculum Vitae</a></li>
							<li><a href="index.html" alt="index.html" style="color: white">Evo GoodReads</a>
								<ul>
									<li><a href="evogoodreads_archive.html" style="color: white">2021 archive</a></li>
								</ul>
							</li>
							<li><a href="../resources.html" style="color: white">Resources</a></li>
							<li><a href="../contact.html" style="color: white">Contact</a></li>
						</ul>
					</nav>
				</header>

			<!-- Main -->
				<div id="main" class="wrapper style1">
					<div class="container">
						<!-- Content -->
							<section id="content">
								<section id="evolution_of_cellular_diversity"></section>
								<center>
									<h3><u>LatchBio launches the first of many evolutionary bioinformatics workflows to come</u></h3>
								</center>

									<p>
									<img class="TextWrapLeft" src="../images/latchbio_logo.png"
									alt="../images/latchbio_logo.png", style="width: 32.5%; height: 32.5%">
									<a href="https://latch.bio/">LatchBio</a> is implementing innovative approaches to streamline bioinformatic
									workflows through an easy-to-use browser-based platform. The bioinformatics workflows available on the
									<a href="https://latch.bio/">LatchBio</a> platform facilitate diverse analyses across many  disciplines
									in the biological sciences including <a href="https://latch.bio/solutions/gene-therapy">Gene Therapy and
									Editing</a>, <a href="https://latch.bio/solutions/cell-therapy">Cell Therapy</a>,
									<a href="https://latch.bio/solutions/sars-cov-2">SARS-CoV-2 research</a>, and
									<a href="https://latch.bio/solutions/ngs">Next Generation Sequence</a> analysis. “The
									<a href="https://latch.bio/">LatchBio</a> platform makes trusted academic
									tools accessible directly to biologists with best-in-class user interfaces and a flexible cloud-native data
									infrastructure accessible from any browser,” says Kenny Workman co-founder and CTO.
									
									<br /><br />

									<img class="TextWrapRight" src="../software_images/clipkit_logo.jpg"
									alt="../software_images/clipkit_logo.jpg", style="width: 35%; height: 35%">
									Recently, <a href="https://latch.bio/">LatchBio</a> has expanded the workflows available on our platform to
									include analyses for the field of evolutionary biology through a collaboration with 
									<a href="https://jlsteenwyk.com/">Dr. Jacob L. Steenwyk</a>,
									a <a href="https://www.hhmi.org/">Howard Hughes Medical Institute</a>
									<a href="https://www.hhmi.org/science-education/programs/gilliam-fellowships-advanced-study">Gilliam Fellow</a>
									working in the laboratory of <a href="https://as.vanderbilt.edu/rokaslab/">Dr. Antonis Rokas</a> at 
									<a href="https://www.vanderbilt.edu/">Vanderbilt University</a>. As the debut workflow,
									<a href="https://latch.bio/">LatchBio</a> and <a href="https://jlsteenwyk.com/">Steenwyk</a> release 
									<a href="https://jlsteenwyk.com/ClipKIT/">ClipKIT</a>,
									an algorithm for trimming multiple sequence alignments prior to
									phylogenetic inference
									(<a href="https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.3001007">Steenwyk et al., 2020</a>).
									See and use the workflow <a href="https://console.latch.bio/explore/59878/info">here</a>! Across 140,000
									multiple sequence alignments that span a broad diversity of evolutionary histories, phylogenies inferred from
									multiple sequence alignments trimmed with ClipKIT were shown to be accurate and robust. Multiple sequence alignments
									trimmed using ClipKIT may help address a major goal in evolutionary biology — elucidating the evolutionary history
									of genes, genomes, and species.

									<br /><br />

									<a href="https://latch.bio/">LatchBio</a> and <a href="https://jlsteenwyk.com/">Steenwyk</a> plan to continue
									collaborating and build more evolutionary genomic bioinformatic
									workflows on the <a href="https://latch.bio/">LatchBio</a> platform. “Working with the team at LatchBio is
									incredibly exciting,” Steenwyk
									said. <a href="https://latch.bio/">LatchBio</a> is solving real problems in the world of bioinformatics and
									simultaneously democratizing
									analyses. Now, with the click of a few buttons, scientists with varying degrees of bioinformatic experience
									can easily conduct high throughput and reproducible bioinformatic analyses — that is <i>really</i> powerful.”
									
									<br /><br />

									<a href="https://jlsteenwyk.com/">Steenwyk</a> was one of the first user's of <a href="https://latch.bio/">LatchBio</a>'s
									recently released Software Development Kit (<a href="https://latchbio.substack.com/p/the-latch-sdk">SDK</a>), an
									open-source bioinformatics workflow development kit. The <a href="https://latch.bio/">LatchBio</a>
									<a href="https://latchbio.substack.com/p/the-latch-sdk">SDK</a> improves
									upon predecessors such as WDL and nextflow by allowing Python-native
									development, serverless cloud resource definition (e.g., memory or GPU
									requirements), and dynamically generated and customizable front-end interfaces.
									Using the <a href="https://latch.bio/">LatchBio</a> <a href="https://latchbio.substack.com/p/the-latch-sdk">SDK</a>,
									academic python scripts become versioned, containerized,
									and accessible pieces of software that can be cited and shared within research
									communities.

									<br /><br />

									<strong>Reference</strong>
									<br />
									Shafer, M. E. R., Sawh, A. N., and Schier, A. F. (2022).
									Steenwyk, J. L., Buida, T. J., Li, Y., Shen, X.-X., and Rokas, A. (2020).
									<a href="https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.3001007">ClipKIT: A multiple sequence
									alignment trimming software for accurate phylogenomic inference.</a> PLOS Biol. 18, e3001007. doi:10.1371/journal.pbio.3001007.
									</p>
									<br /><br />

									<center>
										Conflict of Interest Statement: J.L.S. is a scientific consultant for Latch AI Inc.
									</center>

									<br /><br />
									<center><a href="#Top" class="goto-pub scrolly">Top of page</a></center>
								</section>
							</section>
					</div>
				</div>
			</article>

			<!-- Footer -->
				<footer id="footer">
					<ul class="icons">
						<li><a href="https://twitter.com/jlsteenwyk" class="icon brands alt fa-twitter"><span class="label">Twitter</span></a></li>
						<li><a href="https://scholar.google.com/citations?user=VXV2j6gAAAAJ&hl=en" class="icon brands alt fa-google"><span class="label">Google</span></a></li>
						<li><a href="https://www.researchgate.net/profile/Jacob_Steenwyk" class="icon brands alt fa-researchgate"><span class="label">ResearchGate</span></a></li>
						<li><a href="https://github.com/JLSteenwyk" class="icon brands alt fa-github"><span class="label">GitHub</span></a></li>
						<li><a href="https://www.linkedin.com/in/jlsteenwyk/" class="icon brands alt fa-linkedin-in"><span class="label">LinkedIn</span></a></li>
						<li><a href="../contact.html" class="icon solid alt fa-envelope"><span class="label">Email</span></a></li>
					</ul>
					<ul class="copyright">
						<li>&copy; <span class="current-year"></span> Jacob L. Steenwyk</li>
					</ul>
				</footer>

		</div>

		<!-- Scripts -->
			<script src="../assets/js/jquery.min.js"></script>
			<script src="../assets/js/jquery.scrolly.min.js"></script>
			<script src="../assets/js/jquery.dropotron.min.js"></script>
			<script src="../assets/js/jquery.scrollex.min.js"></script>
			<script src="../assets/js/browser.min.js"></script>
			<script src="../assets/js/breakpoints.min.js"></script>
			<script src="../assets/js/util.js"></script>
			<script src="../assets/js/main.js"></script>

	</body>
</html>