<!DOCTYPE HTML>
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-104875636-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-104875636-1');
</script>
<html>
	<style>
		.TextWrapRight{float:right; margin: 10px;}
	</style>
	<style>
		.TextWrapLeft{float:left; margin: 10px;}
	</style>
	<head>
		<title>J.L. Steenwyk</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="../assets/css/main.css" />
		<link rel="icon" href="../images/website_flavicon_network.png">
		<noscript><link rel="stylesheet" href="../assets/css/noscript.css" /></noscript>
	</head>
	<body class="is-preload">
		<article id="Top">
			<div id="page-wrapper">
			<!-- Header -->
				<header id="header">
					<nav id="nav">
						<ul>
							<li><a href="../index.html#content" style="color: white">About</a></li>
							<li><a href="../publications.html" style="color: white">Publications</a></li>
							<li><a href="../news.html" style="color: white">News</a></li>
							<li>
								<a href="../arts.html" style="color: white">Arts</a>
								<ul>
									<li><a href="../sciart.html" style="color: white">SciArt Shop</a></li>
									<li><a href="../arts_visual.html" style="color: white">Visual</a></li>
									<li><a href="../arts_music.html" style="color: white">Music</a></li>
									<li><a href="../arts_poetry.html" style="color: white">Poetry</a></li>
								</ul>
							</li>
							<li><a href="../software.html" style="color: white">Software</a></li>
							<li><a href="../curriculum_vitae.html" style="color: white">Curriculum Vitae</a></li>
							<li><a href="index.html" alt="index.html" style="color: white">Evo GoodReads</a></li>
							<li><a href="../resources.html" style="color: white">Resources</a></li>
							<li><a href="../contact.html" style="color: white">Contact</a></li>
						</ul>
					</nav>
				</header>

			<!-- Main -->
				<div id="main" class="wrapper style1">
					<div class="container">
						<!-- Content -->
							<section id="content">
								<header class="major">
									<h2>Evo GoodReads</h2>
								</header>
								<section id="EuropeanBatrachochytriumsalamandrivoransepidemic">
									<center>
										<h3><u>Population insights into the European <i>Batrachochytrium salamandrivorans</i> epidemic</u></h3>
									</center>
									<p>
									<img class="TextWrapLeft" src="2021_images/Kelly_etal_2021_Nat_Comm.png"
									alt="2021_images/Kelly_etal_2021_Nat_Comm.png", style="width: 30%; height: 30%">
									Fungi pose a serious threat to global biodiversity. The lack of fungal genomes impedes
									our ability to characterize epidemics and develop strategies to stop them. To address this,
									researchers sequenced nine isolates of <i>Batrachochytrium salamandrivorans</i> that were associated
									with fire salamander die-offs that also span the geographic and temporal range of the outbreak.
									Molecular evolution analyses reveal lineage-specific
									gene family acquisitions, losses, and expansions. Analyses also revealed horizontal
									gene transfer events that occurred after the divergence of major lineages. Together, these analyses
									reveal a multitude of evolutionary mechanisms contribute to genome diversity among <i>B. salamandrivorans</i>
									isolates. Examination of
									gene content and functions therein suggests <i>B. salamandrivorans</i> has a saprotrophic
									lifestyle, which was previously not observed. This finding suggests that a natural reservoir
									of pathogens can continually infect fire salamanders even after population decline, which can 
									lead to extinction. These findings shed light on pathogen success and provide key insights
									to inform the design of mitigation strategies.
									<br /><br />
									Kelly, M., Pasmans, F., Muñoz, J. F., Shea, T. P., Carranza, S., Cuomo, C. A., et al. (2021).
									<a href="https://www.nature.com/articles/s41467-021-26758-y">Diversity, multifaceted evolution,
									and facultative saprotrophism in the European <i>Batrachochytrium salamandrivorans</i> epidemic.</a>
									Nat. Commun. doi:10.1038/s41467-021-27005-0.
									</p>
								</section>
								<hr />
								<section id="wildanddomesticducks">
									<center>
										<h3><u>Comparative genomics of wild and domestic ducks provides insight into avian domestication</u></h3>
									</center>
									<p>
									<img class="TextWrapRight" src="2021_images/Zhu_etal_2021_Nature_comm.png"
									alt="2021_images/Zhu_etal_2021_Nature_comm.png", style="width: 35%; height: 35%">
									Ducks are a source of food (i.e., meat and eggs) and feather down. Similar to other
									domesticated organisms, human intervention has imposed a strong selective pressure
									on ducks used for human affairs resulting in diversity of body size, plumage, and
									reproduction and the rise of two major breeds: one used for egg laying (the Shaoxing
									breed) and another used for meat consumption (the Pekin breed). Researchers generated
									high-quality genome assemblies for the Mallard, Pekin, and Shaoxing ducks to illuminate
									the genetic underpinnings of duck domestication. Over 35 million variants such as single
									nucleotide polymorphisms, insertions and deletions, and large structural variants were
									identified among wild and domesticated ducks. Comparative transcriptomics revealed
									differential gene expression among several genes including <i>NR2F2</i>, which is 
									involved in adipocyte differentiation in mice. After introducing mutations in 
									<i>NR2F2</i> in an immortalized chicken preadipocyte cell line revealed variation in
									mRNA expression lovels and adipogenic potential. These results implicate <i>NR2F2</i>
									in adipocyte variation between wild and domestic ducks, but further analysis is required.
									Taken together, this study uncovers several mutations associated with duck domestication
									and identifies candidate genes that may be associated with phenotypic differences among
									wild and domestic ducks.
									<br /><br />
									Zhu, F., Yin, Z.-T., Wang, Z., Smith, J., Zhang, F., Martin, F., et al. (2021).
									<a href="https://www.nature.com/articles/s41467-021-26758-y">Three chromosome-level duck genome
									assemblies provide insights into genomic variation during domestication.</a>
									Nat. Commun. doi:10.1038/s41467-021-26272-1.
									</p>
								</section>
								<hr />
								<section id="arebacterialchromosomesmobilegeneticelements">
									<center>
										<h3><u>Are bacterial chromosomes mobile genetic elements?</u></h3>
									</center>
									<p>
									<img class="TextWrapLeft" src="2021_images/Hall_etal_Nat_Comm.png"
									alt="2021_images/Hall_etal_Nat_Comm.png", style="width: 35%; height: 35%">
									Genome sequencing of bacteria has revealed that horizontal gene transfer—wherein individuals
									acquire genes from non-vertical processes such as phage-related bioprocesses—occurs frequently.
									Horizontally acquired genes can have functional consequences related to organismal ecology. These processes
									suggest that the bacterial tree of life is more like a network rather than a bifurcating tree.
									Furthermore, the rate of horizontal gene transfer among chromosomal genes can exceed
									the rate of transfer among genetic elements classified as "mobile genetic elements" such as
									plasmids. As a result, these observations call into question whether bacterial chromosomes 
									should also be considered mobile genetic elements. This question is complicated by asymmetric
									rates of horizontal gene transfer across bacterial chromosomes; thus, should genomic islands
									in the bacterial chromosome be considered mobile genetic elements or the entire chromosome?
									Another question to be posed is whether microbiomes should
									be viewed through the lens of taxonomic composition or the functional genetic toolkit available
									to the community?Although answers are not readily available, this thought provoking question
									can inform diverse studies including organismal ecology and the microbiome.  
									<br /><br />
									Hall (2021).
									<a href="https://www.nature.com/articles/s41467-021-26758-y">Is the bacterial chromosome a mobile genetic
										element?</a>
									Nat. Commun. doi:10.1038/s41467-021-26758-y.
									</p>
								</section>
								<hr />
								<section id="Recodingandgenespecificmixturemodelssuggestsspongesaretherootoftheanimaltree">
									<center>
										<h3><u>Recoding and gene-specific mixture models suggests sponges are the root of the animal tree</u></h3>
									</center>
									<p>
									<img class="TextWrapRight" src="2021_images/Redmond_and_McLysaght_Nat_Comm.png"
									alt="2021_images/Redmond_and_McLysaght_Nat_Comm.png", style="width: 45%; height: 45%">
									Understanding the evolution of animal complexity requires resolved evolutionary relationships among extant taxa.
									Sponges, which have a simple body plan, were thought to be sister to all other animals thereby suggesting animal
									evolution is marked by a gradual increase in body plan and cell type complexity. This view has been challenged
									by phylogenomics, which initially supported comb jellies as sister to all other animals. This observation
									suggests secondary loss or independent origins of complex traits. Researchers demonstrate that comb jellies as
									sister to all other animals likely stems from model misspecification due to the use of overly simplistic models.
									Instead, they utilize site-heterogeneous mixture models, which frequently had better model fit than site-homogeneous
									models, and character recoding into a partitioned phylogenomic workflow. This strategy diminished the impact of
									long branch attraction artifacts and provided support for sponges as sister to all animals. The efficacy of using
									mixture models in partitioned phylogenomics provides a mechanism for future studies to combat analytical artifacts
									in phylogenomic analyses.
									Text adapted from <a href="https://www.nature.com/ncomms/">Nature Communications</a>.
									<br /><br />
									Redmond and McLysaght (2021).
									<a href="https://www.nature.com/articles/s41467-021-22074-7">Evidence for sponges as sister to all other
										animals from partitioned phylogenomics with mixture models and recoding.</a>
									Nat. Commun. doi:10.1038/s41467-021-22074-7.
									</p>
								</section>
								<hr />
								<section id="Epigeneticsandenvironmentimpactmutationrateinapathogenicfungus">
									<center>
										<h3><u>Epigenetics and environment impact mutation rate in a pathogenic fungus</u></h3>
									</center>
									<p>
									<img class="TextWrapLeft" src="2021_images/Habig_etal_Nat_Comm.png"
									alt="2021_images/Habig_etal_Nat_Comm.png", style="width: 45%; height: 45%">
									Mutation rates may vary across the genome and be impacted by selection. Intra-genomic mutation rate variation
									may be linked to epigenetic modifications but studies that directly link the two are sparse. Here, researchers
									unravel the impact of epigenetic modifications and temperature stress on mutation rates in a fungal pathogen.
									Deletion mutants that lack epigenetic modifications reveal that histone mark H3K27me3 increases mutation rates;
									in contrast, H3K9me3 decreases the mutation rate. Researchers also reveal two ways that intra-genomic mutation
									rate can vary: one, cytosine methylation in transposable elements (TE) increases mutation rate, which results
									in significantly less TE mobilization, and, two, accessory chromosomes have a significantly higher mutation rates.
									Environmental stressors, namely temperature stress, were found to increase mutation rates. These results reveal
									that epigenetic and environmental parameters can significantly impact genome-wide mutation rate thereby impacting
									the evolutionary trajectory of a species.
									Text adapted from <a href="https://www.nature.com/ncomms/">Nature Communications</a>.
									<br /><br />
									Habig, M., Lorrain, C., Feurtey, A., Komluski, J., and Stukenbrock, E. H. (2021).
									<a href="https://www.nature.com/articles/s41467-021-26108-y">Epigenetic
									modifications affect the rate of spontaneous mutations in a pathogenic fungus.</a>
									Nat. Commun. doi:10.1038/s41467-021-26108-y.
									</p>
								</section>
								<hr />
								<section id="Microevolutionleadstoantifungaldrugresistanceinafungalpathogen">
									<center>
										<h3><u>Microevolution leads to antifungal drug resistance in a fungal pathogen</u></h3>
									</center>
									<p>
									<img class="TextWrapRight" src="2021_images/Ksiezopolska_etal_2021_Current_Biology.jpeg"
									alt="2021_images/Ksiezopolska_etal_2021_Current_Biology.jpeg", style="width: 45%; height: 45%">
									Increased resistance to antifungal drugs among fungal pathogens is of growing medical concern.
									However, the processes leading to resistance are poorly understood. Researchers used experimental
									evolution to study the mutational signatures associated with the evolution of fluconazole and
									anidulafungin resistance in the major fungal pathogen <i>Candida glabrata</i>. Resistance to one
									or both drugs occurred rapidly and was associated with repeatedly observed mutations in a few genes as
									well as a moderate fitness costs. Mutations in <i>ERG3</i> were associated with
									resistance to anidulafungin and cross-resistance to fluconazole. These results shed light 
									on the evolution of resistance and cross-resistance to
									antifungal drugs.
									Text adapted from <a href="https://www.sciencedirect.com/journal/current-biology">Current Biology</a>.
									<br /><br />
									Ksiezopolska, E., Schikora-Tamarit, M. À., Beyer, R., Nunez-Rodriguez, J. C., Schüller, C., and
									Gabaldón, T. (2021).
									<a href="https://www.sciencedirect.com/science/article/abs/pii/S0960982221001391">Narrow mutational
									signatures drive acquisition of multidrug resistance in the fungal pathogen Candida glabrata.</a>
									Curr. Biol. doi:10.1016/j.cub.2021.09.084.
									</p>
								</section>
								<hr />
								<section id="Theevolutionaryhistoryoffungi">
									<center>
										<h3><u>The evolutionary history of fungi</u></h3>
									</center>
									<p>
									<img class="TextWrapLeft" src="2021_images/Li_etal_2021_graphical_abstract_Current_Biology.jpeg"
									alt="2021_images/Li_etal_2021_graphical_abstract_Current_Biology.jpeg", style="width: 25%; height: 25%">
									Phylogenomic studies have improved our understanding of the tree of life.
									Despite the technologic and medical significance of fungi, the evolutionary
									relationships among major clades are poorly resolved. To evaluate poorly resolved relationships,
									researchers constructed a phylogenomic data matrix of 290 genes from 
									over 1,600 species, which includes representatives from most major lineages. Researchers
									also implemented a rigorous subsampling strategy for identifying incongruence bipartitions.
									Analyses of these the full and subsampled data matrices using concatenation- and coalescent-based
									approaches yielded a robust phylogeny of the fungal kingdom. Support was observed for several poorly
									resolved relationships and episodes of ancient diversification. These results provide a
									framework to studying fungal evolution.
									Text adapted from <a href="https://www.sciencedirect.com/journal/current-biology">Current Biology</a>.
									<br /><br />
									Li, Y., Steenwyk, J. L., Chang, Y., Wang, Y., James, T. Y., Stajich, J. E., et al. (2021).
									<a href="https://www.sciencedirect.com/science/article/abs/pii/S0960982221001391">A
									genome-scale phylogeny of the kingdom Fungi.</a> Curr. Biol.
									doi:10.1016/j.cub.2021.01.074.
									</p>
								</section>
								<hr />
								<section id="Insightsintotheemergenceofthemycorrhizalsymbiosis">
									<center>
										<h3><u>Insights into the emergence of the mycorrhizal symbiosis</u></h3>
									</center>
									<img class="TextWrapRight" src="2021_images/Malar_etal_2021_Current_biology_graphical_abstract.jpg"
									alt="2021_images/Malar_etal_2021_Current_biology_graphical_abstract.jpg", style="width: 30%; height: 30%">
									<p>
									Arbuscular mycorrhizal fungi (AMF) (subphylum Glomeromycotina) form a
									symbiotic relationship with over 70% of known land plants.
									This symbiosis enables plants to acquire poorly soluble soil nutrients and AMF to
									receive photosynthetically fixed carbohydrates. This plant-fungus symbiosis dates back more
									than 400 million years and is thought to have contributed to the
									colonization of land by plants. Genomic analyses of plant species has begun to unravel
									the mechanisms that facilitated the evolution of this symbiosis,
									however, how and when the plant-fungus symbiosis emerged in AMF remains unknown and research
									into this topic has been stymied by the dearth of genomic data among key AMF fungi.
									<i>Geosiphon pyriformis</i> (Archaeosporales) is a basal AMF fungus and the only one known
									to produce endosymbiosis with nitrogen-fixing cyanobacteria (<i>Nostoc punctiforme</i>), which is
									thought to be the ancestral AMF-state. Thus, <i>G.
									pyriformis</i> is ideal to glean insights into the evolutionary origins of AMS and the emergence
									of AMF symbiosis. To begin to address these questions by sequencing, researchers sequenced the
									genome of <i>G. pyriformis</i>.
									Text adapted from <a href="https://www.sciencedirect.com/journal/current-biology">Current Biology</a>.
									<br /><br />
									Malar C, M., Krüger, M., Krüger, C., Wang, Y., Stajich, J. E., Keller, J., et al. (2021).
									<a href="https://www.sciencedirect.com/science/article/abs/pii/S0960982221001391">The genome of
									Geosiphon pyriformis reveals ancestral traits linked to the emergence of the arbuscular
									mycorrhizal symbiosis.</a> Curr. Biol. doi:10.1016/j.cub.2021.01.058.
									</p>
								</section>

								<br />
								<center><a href="#Top" class="goto-pub scrolly">Top of page</a></center>
							</section>
					</div>
				</div>
			</article>

			<!-- Footer -->
				<footer id="footer">
					<ul class="icons">
						<li><a href="https://twitter.com/jlsteenwyk" class="icon brands alt fa-twitter"><span class="label">Twitter</span></a></li>
						<li><a href="https://scholar.google.com/citations?user=VXV2j6gAAAAJ&hl=en" class="icon brands alt fa-google"><span class="label">Google</span></a></li>
						<li><a href="https://www.researchgate.net/profile/Jacob_Steenwyk" class="icon brands alt fa-researchgate"><span class="label">ResearchGate</span></a></li>
						<li><a href="https://github.com/JLSteenwyk" class="icon brands alt fa-github"><span class="label">GitHub</span></a></li>
						<li><a href="https://www.linkedin.com/in/jlsteenwyk/" class="icon brands alt fa-linkedin-in"><span class="label">LinkedIn</span></a></li>
						<li><a href="../contact.html" class="icon solid alt fa-envelope"><span class="label">Email</span></a></li>
					</ul>
					<ul class="copyright">
						<li>&copy; 2021 Jacob L. Steenwyk</li>
					</ul>
				</footer>

		</div>

		<!-- Scripts -->
			<script src="../assets/js/jquery.min.js"></script>
			<script src="../assets/js/jquery.scrolly.min.js"></script>
			<script src="../assets/js/jquery.dropotron.min.js"></script>
			<script src="../assets/js/jquery.scrollex.min.js"></script>
			<script src="../assets/js/browser.min.js"></script>
			<script src="../assets/js/breakpoints.min.js"></script>
			<script src="../assets/js/util.js"></script>
			<script src="../assets/js/main.js"></script>

	</body>
</html>